<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title></title>
<!--Generated on Tue Apr 21 17:16:15 2015 by LaTeXML (version 0.8.1) http://dlmf.nist.gov/LaTeXML/.-->

<meta http-equiv="Content-Type" content="application/xhtml+xml; charset=UTF-8"></meta>
<link rel="stylesheet" href="LaTeXML.css" type="text/css"></link>
<link rel="stylesheet" href="ltx-article.css" type="text/css"></link>
<link rel="stylesheet" href="ltx-ulem.css" type="text/css"></link>
<link rel="stylesheet" href="resources/plr-style.css" type="text/css"></link>
<script src="resources/LaTeXML-maybeMathjax.js" type="text/javascript"></script>
<script src="resources/adjust-svg.js" type="text/javascript"></script>
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<div class="ltx_document">
  
  
  
  
<div id="p1" class="ltx_para">
    
<p class="ltx_p">Dear Michael,
<br class="ltx_break"></br>Thanks to Flo, Sam, and you for all of the useful comments, they were very helpful. We have revised the manuscript in light of those comments, and additional comments that we received from others. We have responded point by point to the reviews below (our responses are in blue). In the main text changes made in response to the reviewers are in blue, changes made by us in response to other comments are marked in red.</p>
  
</div>
  
<div id="p2" class="ltx_para">
    
<p class="ltx_p">Best wishes
<br class="ltx_break"></br>Peter Ralph and Graham Coop
<br class="ltx_break"></br>4/21/15</p>
  
</div>
  
  
  
<div id="p3" class="ltx_para">
    
<p class="ltx_p"><span class="ltx_text ltx_font_bold">Mike’s comments:</span></p>
  
</div>
  
<div id="p4" class="ltx_para">
    
<p class="ltx_p">I’ll add my comments in order that they appear in the presentation of the paper, rather than trying to rank them by importance.</p>
  
</div>
  
<div id="p5" class="ltx_para">
    
<p class="ltx_p">64 –ish: Be clear about whether you are assuming that this variation is allelic</p>
  
</div>
  
<div id="p6" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Good point. We have extended our discussion there. </span></em></p>
  
</div>
  
<div id="p7" class="ltx_para">
    
<p class="ltx_p">68: Especially as this is in an issue about local adaption, you probably want to be very clear there that the alleles have the same fitness everywhere.</p>
  
</div>
  
<div id="p8" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We have expanded on this.</span></em></p>
  
</div>
  
<div id="p9" class="ltx_para">
    
<p class="ltx_p">77: “below”? &amp; 77: “we assume to be equal to one” – re-write please, it isn’t clear at first reading what you are assuming to be one.</p>
  
</div>
  
<div id="p10" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We have edited this in response to reviewers’ comments.</span></em></p>
  
</div>
  
<div id="p11" class="ltx_para">
    
<p class="ltx_p">After 90: “somewhat larger probability of local establishment” – it is not at all clear to me what local establishment means in the case of a deleterious allele. Can you unpack this please?</p>
  
</div>
  
<div id="p12" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We meant establishment following the environmental shift, we have altered the text to reflect that.</span></em></p>
  
</div>
  
  
<div id="p13" class="ltx_para">
    
<p class="ltx_p">Equation 2: If the mutation selection balance is <math xmlns="http://www.w3.org/1998/Math/MathML" id="p13.m1" class="ltx_Math" alttext="\mu/s_{d}" display="inline"><mrow><mi mathsize="83%" stretchy="false">μ</mi><mo mathsize="83%" stretchy="false">/</mo><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">d</mi></msub></mrow></math>, then the number of copies of the allele on average per unit area is <math xmlns="http://www.w3.org/1998/Math/MathML" id="p13.m2" class="ltx_Math" alttext="\rho mu/s_{d}" display="inline"><mrow><mrow><mi mathsize="83%" stretchy="false">ρ</mi><mo>⁢</mo><mi mathsize="83%" stretchy="false">m</mi><mo>⁢</mo><mi mathsize="83%" stretchy="false">u</mi></mrow><mo mathsize="83%" stretchy="false">/</mo><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">d</mi></msub></mrow></math>. The probability that this leads to a successfully establishment is <math xmlns="http://www.w3.org/1998/Math/MathML" id="p13.m3" class="ltx_Math" alttext="2s_{b}" display="inline"><mrow><mn mathsize="83%" stretchy="false">2</mn><mo>⁢</mo><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">b</mi></msub></mrow></math> per copy approximately, which gives <math xmlns="http://www.w3.org/1998/Math/MathML" id="p13.m4" class="ltx_Math" alttext="2s_{b}\mu\rho/s_{d}" display="inline"><mrow><mrow><mn mathsize="83%" stretchy="false">2</mn><mo>⁢</mo><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">b</mi></msub><mo>⁢</mo><mi mathsize="83%" stretchy="false">μ</mi><mo>⁢</mo><mi mathsize="83%" stretchy="false">ρ</mi></mrow><mo mathsize="83%" stretchy="false">/</mo><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">d</mi></msub></mrow></math> for <math xmlns="http://www.w3.org/1998/Math/MathML" id="p13.m5" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi mathsize="83%" stretchy="false">λ</mi><mn mathsize="83%" stretchy="false">0</mn></msub></math>. This seems like an easier derivation to me. (For large <math xmlns="http://www.w3.org/1998/Math/MathML" id="p13.m6" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">d</mi></msub></math>, this diverges from your result – which is more correct?)</p>
  
</div>
  
<div id="p14" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> This difference arises from taking a continuous time approach. We have extended the description here to more clearly give the “classic” mutation-selection balance interpretation. We think it is important to work through this in the simple spatial case (via branching processes) rather than just move straight from the result people are familiar with deriving under the panmictic model. This allows folks to see that the result is valid in our spatial setting. </span></em></p>
  
</div>
  
  
<div id="p15" class="ltx_para">
    
<p class="ltx_p">P3, somewhere. It would be worth explicitly giving the formula for lambda somewhere.</p>
  
</div>
  
<div id="p16" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Done.</span></em></p>
  
</div>
  
<div id="p17" class="ltx_para">
    
<p class="ltx_p">Figure 1: The grey cone for (x,t) is too faint to see easily. I didn’t notice it until trying to find what the caption was referring to. And in the caption, it would be worth explaining why you are looking at this cone before describing it.</p>
  
</div>
  
<div id="p18" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Good points. We have edited the caption and darkened the cone. </span></em></p>
  
</div>
  
<div id="p19" class="ltx_para">
    
<p class="ltx_p">192: You need to be more specific about where these two formulas for the means come from. It is not clear at all.</p>
  
</div>
  
<div id="p20" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We have expanded on this section to explain how these calculations are performed.</span></em></p>
  
</div>
  
<div id="p21" class="ltx_para">
    
<p class="ltx_p">210: It may be confusing to refer to standing variation as mutational input. If they already exist they are not mutations anymore, but alleles.</p>
  
</div>
  
<div id="p22" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Changed</span></em></p>
  
</div>
  
<div id="p23" class="ltx_para">
    
<p class="ltx_p">225-229 – Cool!</p>
  
</div>
  
  
<div id="p24" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Thanks! We like this point.</span></em></p>
  
</div>
  
<div id="p25" class="ltx_para">
    
<p class="ltx_p">231-232 – I thought the first person here “ourselves” was a little jarring.</p>
  
</div>
  
<div id="p26" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Changed.</span></em></p>
  
</div>
  
<div id="p27" class="ltx_para">
    
<p class="ltx_p">246: Can you use something other than nu? It looks too much like the <math xmlns="http://www.w3.org/1998/Math/MathML" id="p27.m1" class="ltx_Math" alttext="v" display="inline"><mi mathsize="83%" stretchy="false">v</mi></math> that you have already used for something else.
<em class="ltx_emph"> <span class="ltx_text" style="color:#0000FF;"> Changed to <math xmlns="http://www.w3.org/1998/Math/MathML" id="p27.m2" class="ltx_Math" alttext="\gamma" display="inline"><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">γ</mi></math></span></em></p>
  
</div>
  
<div id="p28" class="ltx_para">
    
<p class="ltx_p">Figure 5: The caption was missing words or symbols in my copy. Also remove the last comma.</p>
  
</div>
  
<div id="p29" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Fixed.</span></em></p>
  
</div>
  
<div id="p30" class="ltx_para">
    
<p class="ltx_p">250: Perhaps this would be better as “the mean proportion of adapted patches”</p>
  
</div>
  
<div id="p31" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Done.</span></em>
251: Please give a reference for the Palm measure.</p>
  
</div>
  
<div id="p32" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Added more description and a reference.</span></em></p>
  
</div>
  
<div id="p33" class="ltx_para">
    
<p class="ltx_p">Figure 6, and many other places, especially in the discussion. You assume a quite large mutation rate here. This may be appropriate for a loss of function mutation ( is this the case for this example?), but it is very high for many types of mutations per locus. Many of your conclusions would be very different with lower mutation rates. It would be nice to know ( at least qualitatively ) what would happen if the mutation rates were lower. You say that convergent evolution may be quite high (e.g. start of discussion), but this is not true if the mutation rate were an order of magnitude or two lower than you assume here, I think.</p>
  
</div>
  
<div id="p34" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We have expanded our discussion at the start of the discussion to make this clearer. The G6PD alleles are not knockouts but do correspond to reduced activity. While a 100bp target may be high for some traits, it perhaps is not when you consider all of the possible genes in which a change could occur. Also if you move to considering smaller target sizes the time to adaptation goes up considerably, both as there is a longer waiting time for new mutations to arise and standing variants are much rarer. </span></em></p>
  
</div>
  
  
<div id="p35" class="ltx_para">
    
<p class="ltx_p">284 plus a couple of lines: The sentence fragment beginning with “Or, …” is a sentence fragment.</p>
  
</div>
  
<div id="p36" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Modified.</span></em></p>
  
</div>
  
<div id="p37" class="ltx_para">
    
<p class="ltx_p">Discussion, somewhere: Can you say anything about the pattern of differentiation of these convergent selected genes relative to neutral differentiation?</p>
  
</div>
  
<div id="p38" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Added a sentence to the “The confusing signal of geographic convergent evolution” section pointing out that they will stick out against neutral variation. In Ralph and Coop 2010 we gave a more complete version of this argument. However, in practice it will depend on the true per generation rate of migration among demes, so it’s hard to say anything to definite when that is rarely known.</span></em></p>
  
</div>
  
  
<div id="p39" class="ltx_para">
    
<p class="ltx_p">
      <span class="ltx_text ltx_font_bold">Reviewer 1 (Flo)
<br class="ltx_break"></br></span>
    </p>
  
</div>
  
  
<div id="p40" class="ltx_para">
    
<p class="ltx_p">The manuscript is well written, interesting, insightful, and dense (in a good way!). It is also sometimes a bit hard to read. The main issue is that it is rather likely that the average AmNat reader will not have read Ralph &amp; Coop 2010, or, even if they have, they will not remember some specific details. I had to read Ralph &amp; Coop 2010 (hereafter RC10) to better understand this manuscript, so I will try to highlight the points that could be clarified, in the hope that this will help make the manuscript more accessible to less mathematically-inclined readers.</p>
  
</div>
  
<div id="p41" class="ltx_para">
    
<p class="ltx_p"><span class="ltx_text ltx_font_bold">Comments on the model and the analysis (clarifications needed)</span></p>
  
</div>
  
<div id="p42" class="ltx_para">
    
<p class="ltx_p">- The introduction is written as if it were obvious that there is a switch in selection pressures (e.g. line 41), while other scenarios are conceivable (e.g. a species encountering a novel habitat type while expanding its range; or, other types of temporal variation (not just a switch)). Maybe clarify near the beginning of the introduction the type of environmental variation you are considering.</p>
  
</div>
  
<div id="p43" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Good Points. We had originally wanted to avoid too much overlap with the previous paper, but agree that makes it hard to read as a stand-alone paper. We have extended our introduction to more fully describe our previous work on the topic, the homogeneous nature of the selection pressure, and adding a simulation figure to illustrate the previous intuition. </span></em></p>
  
</div>
  
<div id="p44" class="ltx_para">
    
<p class="ltx_p">- line 64: it is mainly about vocabulary here: the word ”class” is used to refer to the ”types”, while in section 1.6 ”class” is used to refer to the diversity of mutations (i.e., within type variability).</p>
  
</div>
  
  
<div id="p45" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Good point, although it is consistent in calling a “class” a category of mutations with distinct properties? We’ve modified section 1.6 a bit to set the usage up like this.</span></em></p>
  
</div>
  
<div id="p46" class="ltx_para">
    
<p class="ltx_p">- line 65: add ”neutral” before ”variation” (if appropriate)</p>
  
</div>
  
<div id="p47" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> added neutral, thanks.</span></em></p>
  
</div>
  
<div id="p48" class="ltx_para">
    
<p class="ltx_p">- somewhere in the model description, it would be good to mention that time is measured in generations (and clarify whether it is continuous/discrete).</p>
  
</div>
  
<div id="p49" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Good point, added a sentence to “Rates of origination of standing and new mutations”.</span></em></p>
  
</div>
  
<div id="p50" class="ltx_para">
    
<p class="ltx_p">- line 76-77: this sentence was opaque to me until I read RC10 and the formula <math xmlns="http://www.w3.org/1998/Math/MathML" id="p50.m1" class="ltx_Math" alttext="2s/\xi^{2}" display="inline"><mrow><mrow><mn mathsize="83%" stretchy="false">2</mn><mo>⁢</mo><mi mathsize="83%" stretchy="false">s</mi></mrow><mo mathsize="83%" stretchy="false">/</mo><msup><mi mathsize="83%" stretchy="false">ξ</mi><mn mathsize="83%" stretchy="false">2</mn></msup></mrow></math>.</p>
  
</div>
  
<div id="p51" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> In hindsight that is pretty cryptic. We changed this to “We assume that the variance in offspring number is one, see
Ralph &amp; Coop 2010 for how this assumption can be relaxed.”</span></em></p>
  
</div>
  
<div id="p52" class="ltx_para">
    
<p class="ltx_p">- line 86: the way mutation works is not clear yet. If I understand correctly, a neutral type’s offspring is mutated with probability <math xmlns="http://www.w3.org/1998/Math/MathML" id="p52.m1" class="ltx_Math" alttext="\mu" display="inline"><mi mathsize="83%" stretchy="false">μ</mi></math>, but there are no further mutations among the mutated types (i.e. there cannot be a new cone growing within a cone). So ”each new offspring” is the offspring of neutral type individuals, right? It could be also useful to recap somewhere the main assumptions of the analysis, e.g. the decomposition of time scales between local establishment and then spatial spread (which is discussed in 1.7), or why there are not further mutants emerging within mutant cones.</p>
  
</div>
  
<div id="p53" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We have now added “In geographic areas already
occupied by the mutant type (i.e. in regions it has spread to already), new mutations are selectively excluded
(are effectively neutral), and so unlikely to establish over the
short-time scales considered here.”. Hopefully that clarifies that we are not banning subsequent mutations, only assuming that they are not selectively favoured.</span></em></p>
  
</div>
  
<div id="p54" class="ltx_para">
    
<p class="ltx_p">- line 88: It would be great to add a few lines to explain what it is a Poisson process in space and time (the explanation in RC10, p651, is very clear).
</p>
  
</div>
  
<div id="p55" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Added text to the section “Rates of origination of standing and new mutations” to help with this.</span></em></p>
  
</div>
  
<div id="p56" class="ltx_para">
    
<p class="ltx_p">- [unnumbered line that would otherwise be 94] ”*Since* the times and locations …” has it been proven? &amp; [unnumbered line that would otherwise be 96] ”Poisson Mapping Theorem” maybe add reference?</p>
  
</div>
  
<div id="p57" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We have edited this part for clarity.</span></em></p>
  
</div>
  
<div id="p58" class="ltx_para">
    
<p class="ltx_p">- [unnumbered line that would otherwise be 103] Using the letter <math xmlns="http://www.w3.org/1998/Math/MathML" id="p58.m1" class="ltx_Math" alttext="s" display="inline"><mi mathsize="83%" stretchy="false">s</mi></math> to refer to a time is a bit confusing!!
(same line 334, where <math xmlns="http://www.w3.org/1998/Math/MathML" id="p58.m2" class="ltx_Math" alttext="s" display="inline"><mi mathsize="83%" stretchy="false">s</mi></math> is now a rate)</p>
  
</div>
  
<div id="p59" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Good point, changed this around.</span></em></p>
  
</div>
  
<div id="p60" class="ltx_para">
    
<p class="ltx_p">- [unnumbered line that would otherwise be 106] Do you actually need to use <math xmlns="http://www.w3.org/1998/Math/MathML" id="p60.m1" class="ltx_Math" alttext="\zeta" display="inline"><mi mathsize="83%" stretchy="false">ζ</mi></math>? Could not the equation that follows simply be written with <math xmlns="http://www.w3.org/1998/Math/MathML" id="p60.m2" class="ltx_Math" alttext="E[(1-p_{s})^{Z_{T}}" display="inline"><mrow><mi mathsize="83%" stretchy="false">E</mi><mrow><mo maxsize="83%" minsize="83%">[</mo><msup><mrow><mo maxsize="83%" minsize="83%">(</mo><mn mathsize="83%" stretchy="false">1</mn><mo mathsize="83%" stretchy="false">-</mo><msub><mi mathsize="83%" stretchy="false">p</mi><mi mathsize="83%" stretchy="false">s</mi></msub><mo maxsize="83%" minsize="83%">)</mo></mrow><msub><mi mathsize="83%" stretchy="false">Z</mi><mi mathsize="83%" stretchy="false">T</mi></msub></msup></mrow></mrow></math>]? Also, isn’t <math xmlns="http://www.w3.org/1998/Math/MathML" id="p60.m3" class="ltx_Math" alttext="\zeta(u,T)" display="inline"><mrow><mi mathsize="83%" stretchy="false">ζ</mi><mo>⁢</mo><mrow><mo maxsize="83%" minsize="83%">(</mo><mi mathsize="83%" stretchy="false">u</mi><mo mathsize="83%" stretchy="false">,</mo><mi mathsize="83%" stretchy="false">T</mi><mo maxsize="83%" minsize="83%">)</mo></mrow></mrow></math> the generating function of <math xmlns="http://www.w3.org/1998/Math/MathML" id="p60.m4" class="ltx_Math" alttext="Z_{T}" display="inline"><msub><mi mathsize="83%" stretchy="false">Z</mi><mi mathsize="83%" stretchy="false">T</mi></msub></math>?</p>
  
</div>
  
<div id="p61" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Sure, good suggestion. We’ve removed <math xmlns="http://www.w3.org/1998/Math/MathML" id="p61.m1" class="ltx_Math" alttext="\zeta" display="inline"><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">ζ</mi></math>.</span></em></p>
  
</div>
  
<div id="p62" class="ltx_para">
    
<p class="ltx_p">- equation (2), since this will be used later on, maybe add a line <math xmlns="http://www.w3.org/1998/Math/MathML" id="p62.m1" class="ltx_Math" alttext="\lambda_{0}=\lambda/(-\log(1-s_{d}))" display="inline"><mrow><msub><mi mathsize="83%" stretchy="false">λ</mi><mn mathsize="83%" stretchy="false">0</mn></msub><mo mathsize="83%" stretchy="false">=</mo><mrow><mi mathsize="83%" stretchy="false">λ</mi><mo mathsize="83%" stretchy="false">/</mo><mrow><mo maxsize="83%" minsize="83%">(</mo><mrow><mo mathsize="83%" stretchy="false">-</mo><mrow><mi mathsize="83%" stretchy="false">log</mi><mo>⁡</mo><mrow><mo maxsize="83%" minsize="83%">(</mo><mrow><mn mathsize="83%" stretchy="false">1</mn><mo mathsize="83%" stretchy="false">-</mo><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">d</mi></msub></mrow><mo maxsize="83%" minsize="83%">)</mo></mrow></mrow></mrow><mo maxsize="83%" minsize="83%">)</mo></mrow></mrow></mrow></math>.</p>
  
</div>
  
<div id="p63" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Added.</span></em></p>
  
</div>
  
<div id="p64" class="ltx_para">
    
<p class="ltx_p">- Figure 1: I think that it would really help the reader if the figure were in 3D (space x space x time) instead of 2D (space x time), especially to visualize the ”cones” carved out in space-time. Also, the lightening bolts give the false impression that the mutants appear right at <math xmlns="http://www.w3.org/1998/Math/MathML" id="p64.m1" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi mathsize="83%" stretchy="false">t</mi><mo mathsize="83%" stretchy="false">=</mo><mn mathsize="83%" stretchy="false">0</mn></mrow></math>.</p>
  
</div>
  
<div id="p65" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We have kept this figure relatively unchanged, although we agree that it’s not perfect. We are not convinced that showing the picture in 3D would help, as other details would have to be omitted. The lightening bolts do show the standing variants at time <math xmlns="http://www.w3.org/1998/Math/MathML" id="p65.m1" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">t</mi><mo mathcolor="#0000FF" mathsize="83%" stretchy="false">=</mo><mn mathcolor="#0000FF" mathsize="83%" stretchy="false">0</mn></mrow></math>, rather than the original mutations, so we have modified the caption to indicate this.</span></em></p>
  
</div>
  
  
<div id="p66" class="ltx_para">
    
<p class="ltx_p">- line 208: higher population densities (adjective missing)</p>
  
</div>
  
<div id="p67" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Added higher.</span></em></p>
  
</div>
  
<div id="p68" class="ltx_para">
    
<p class="ltx_p">- equation (9) I do not understand where it comes from! (i.e. how using the proportionality between <math xmlns="http://www.w3.org/1998/Math/MathML" id="p68.m1" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi mathsize="83%" stretchy="false">λ</mi><mn mathsize="83%" stretchy="false">0</mn></msub></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="p68.m2" class="ltx_Math" alttext="\lambda" display="inline"><mi mathsize="83%" stretchy="false">λ</mi></math> give this result). Do we actually need (9) anyway to obtain expression (10)?</p>
  
</div>
  
<div id="p69" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Thanks for spotting this slip, we had accidentally merged together two different results and a typo. We have corrected this now.</span></em></p>
  
</div>
  
<div id="p70" class="ltx_para">
    
<p class="ltx_p">- line 251: I do not know what is the ”Palm measure”, so a reference would be welcome here (or consider removing?)</p>
  
</div>
  
<div id="p71" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Added more description and a reference.</span></em></p>
  
</div>
  
<div id="p72" class="ltx_para">
    
<p class="ltx_p">- line 343: I do not understand why there is a factor 4 here (and not 2).</p>
  
</div>
  
<div id="p73" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Sorry, we had accidentally reverted to <math xmlns="http://www.w3.org/1998/Math/MathML" id="p73.m1" class="ltx_Math" alttext="\rho" display="inline"><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">ρ</mi></math> being the density of diploids, we’ve changed these to a factor or 2 instead of 4. Thanks! </span></em></p>
  
</div>
  
<div id="p74" class="ltx_para">
    
<p class="ltx_p">- equation (16) I am not sure about the exponents: I find that the exponent for <math xmlns="http://www.w3.org/1998/Math/MathML" id="p74.m1" class="ltx_Math" alttext="\beta" display="inline"><mi mathsize="83%" stretchy="false">β</mi></math> is <math xmlns="http://www.w3.org/1998/Math/MathML" id="p74.m2" class="ltx_Math" alttext="-(1+c)/(d+1)" display="inline"><mrow><mo mathsize="83%" stretchy="false">-</mo><mrow><mrow><mo maxsize="83%" minsize="83%">(</mo><mrow><mn mathsize="83%" stretchy="false">1</mn><mo mathsize="83%" stretchy="false">+</mo><mi mathsize="83%" stretchy="false">c</mi></mrow><mo maxsize="83%" minsize="83%">)</mo></mrow><mo mathsize="83%" stretchy="false">/</mo><mrow><mo maxsize="83%" minsize="83%">(</mo><mrow><mi mathsize="83%" stretchy="false">d</mi><mo mathsize="83%" stretchy="false">+</mo><mn mathsize="83%" stretchy="false">1</mn></mrow><mo maxsize="83%" minsize="83%">)</mo></mrow></mrow></mrow></math> and that the exponent for u (outside of exp) is <math xmlns="http://www.w3.org/1998/Math/MathML" id="p74.m3" class="ltx_Math" alttext="(c-d)/(d+1)" display="inline"><mrow><mrow><mo maxsize="83%" minsize="83%">(</mo><mrow><mi mathsize="83%" stretchy="false">c</mi><mo mathsize="83%" stretchy="false">-</mo><mi mathsize="83%" stretchy="false">d</mi></mrow><mo maxsize="83%" minsize="83%">)</mo></mrow><mo mathsize="83%" stretchy="false">/</mo><mrow><mo maxsize="83%" minsize="83%">(</mo><mrow><mi mathsize="83%" stretchy="false">d</mi><mo mathsize="83%" stretchy="false">+</mo><mn mathsize="83%" stretchy="false">1</mn></mrow><mo maxsize="83%" minsize="83%">)</mo></mrow></mrow></math>, but I may be wrong?</p>
  
</div>
  
<div id="p75" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> In the end we evaluate all of the integrals numerically, without the aid of this trick, so we have now dropped the entire appendix section.</span></em></p>
  
</div>
  
<div id="p76" class="ltx_para">
    
<p class="ltx_p"><span class="ltx_text ltx_font_bold">Minor remarks</span></p>
  
</div>
  
<div id="p77" class="ltx_para">
    
<p class="ltx_p">- line 58: ”the genetics of adaptation may often be geographically local” the meaning of this phrases was not obvious at first.</p>
  
</div>
  
<div id="p78" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> edited this sentence.</span></em>
</p>
  
</div>
  
<div id="p79" class="ltx_para">
    
<p class="ltx_p">- line 61: A more natural order seems to be first history, then implications.</p>
  
</div>
  
<div id="p80" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> done</span></em></p>
  
</div>
  
<div id="p81" class="ltx_para">
    
<p class="ltx_p">- line 80-81: If there are more than 1 chromosome per haploid individual, this is not exactly the same?</p>
  
</div>
  
<div id="p82" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We meant that diploids carried twice the number of alleles as haploids. Edited in response.</span></em></p>
  
</div>
  
<div id="p83" class="ltx_para">
    
<p class="ltx_p">- line 91: that’s probably personal, but I usually prefer when the terms are grouped depending on where they come from, and so I would find it clearer to write <math xmlns="http://www.w3.org/1998/Math/MathML" id="p83.m1" class="ltx_Math" alttext="2s_{b}\mu\rho" display="inline"><mrow><mn mathsize="83%" stretchy="false">2</mn><mo>⁢</mo><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">b</mi></msub><mo>⁢</mo><mi mathsize="83%" stretchy="false">μ</mi><mo>⁢</mo><mi mathsize="83%" stretchy="false">ρ</mi></mrow></math>.</p>
  
</div>
  
<div id="p84" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> changed.</span></em></p>
  
</div>
  
<div id="p85" class="ltx_para">
    
<p class="ltx_p">[- bottom of page 4, the lines are unnumbered, but you can avoid this bug of the lineno package by leaving an empty space before begin equation]</p>
  
</div>
  
<div id="p86" class="ltx_para">
    
<p class="ltx_p">- line 126: I do not understand which the sides curve, but this is probably not a crucial point!</p>
  
</div>
  
<div id="p87" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> When there is just standing variation, we simple need to bisect up the area into regions closer to one origin or another, that leads to polygons. However, when the mutations are arising through time we have to consider whether a point lands in the cone of one mutation or another, this leads to curved edges. We’ve added a reference to Ralph and Coop 2010, which graphically depicted this point more clearly, and the teaser “because the radii of the colliding circles differ” (as it is the special case of equal radii circles that leads to the special property of Voronoi cells). A longer explanation: when two expanding circles meet, the direction their intersections move in is the average of their tangents; for non-equally sized circles (as with different ages), this direction changes as they expand.</span></em></p>
  
</div>
  
<div id="p88" class="ltx_para">
    
<p class="ltx_p">- line 200 ”close to the value obtained just from standing variation” obtained by whom/how? &amp; line 202: ”close to the value from” RC10: how different are the values and why?</p>
  
</div>
  
<div id="p89" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Sorry, we meant obtain by us, and by “close to” we meant that they approach these limits at these extremes. Edited accordingly.</span></em></p>
  
</div>
  
  
<div id="p90" class="ltx_para">
    
<p class="ltx_p">- line 216: add ”locally” in front of ”established”, if appropriate.</p>
  
</div>
  
<div id="p91" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> added.</span></em></p>
  
</div>
  
<div id="p92" class="ltx_para">
    
<p class="ltx_p">- line 219: add ”of order” or an equivalent in front of <math xmlns="http://www.w3.org/1998/Math/MathML" id="p92.m1" class="ltx_Math" alttext="s_{b}^{1/2}" display="inline"><msubsup><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">b</mi><mrow><mn mathsize="83%" stretchy="false">1</mn><mo mathsize="83%" stretchy="false">/</mo><mn mathsize="83%" stretchy="false">2</mn></mrow></msubsup></math>, if appropriate.</p>
  
</div>
  
<div id="p93" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> added.</span></em></p>
  
</div>
  
<div id="p94" class="ltx_para">
    
<p class="ltx_p">- Figure 2: Not sure why ”Both” is in black in the legend with the two other line types are in gre/ay.</p>
  
</div>
  
<div id="p95" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Changed this over to being grey for all three.</span></em></p>
  
</div>
  
<div id="p96" class="ltx_para">
    
<p class="ltx_p">- line 258: the expression for the ”sliver” is actually not used in (11), but will be useful in (13).</p>
  
</div>
  
<div id="p97" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Agreed, but we think it is useful to introduce all of the components first.</span></em></p>
  
</div>
  
<div id="p98" class="ltx_para">
    
<p class="ltx_p">- the paragraph lines 304–323 seems to rather belong to the Discussion section.</p>
  
</div>
  
<div id="p99" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We agree that they are discussion flavo(u)red but they naturally go here, having to move this into the discussion would complicate the (already long) discussion more. So we think this is better left here.</span></em></p>
  
</div>
  
<div id="p100" class="ltx_para">
    
<p class="ltx_p">- line 336: ”size” could refer to population size or the area of a patch, maybe add ”population” to clarify?</p>
  
</div>
  
<div id="p101" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We meant the size (i.e. count) of the branching process. Added to text.</span></em></p>
  
</div>
  
  
<div id="p102" class="ltx_para">
    
<p class="ltx_p">- line 354: ”total amount of area” isn’t the environment effectively infinite? Or should one rather understand ”sampled area”?</p>
  
</div>
  
<div id="p103" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> In practice we mean the area where the selection pressure is present.</span></em></p>
  
</div>
  
<div id="p104" class="ltx_para">
    
<p class="ltx_p">- line 419–420: I am not sure I understand here; do you mean that the distant populations could share the same allele? But with the work you have done, including standing variance, shouldn’t we expect them to be of independent origin, even if they are neutral?</p>
  
</div>
  
<div id="p105" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We think is correct as written. We are referring to a model where a common allele (of single origin) is wide-spread before selection pressures switch. Let us know if we need to clarify further.</span></em></p>
  
</div>
  
<div id="p106" class="ltx_para">
    
<p class="ltx_p">- line 459: distance between the lineages is in time<math xmlns="http://www.w3.org/1998/Math/MathML" id="p106.m1" class="ltx_Math" alttext="{}^{-1}" display="inline"><msup><mi></mi><mrow><mo mathsize="83%" stretchy="false">-</mo><mn mathsize="83%" stretchy="false">1</mn></mrow></msup></math>, but the displacement is in distance?</p>
  
</div>
  
<div id="p107" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Clarified this sentence to make clear that <math xmlns="http://www.w3.org/1998/Math/MathML" id="p107.m1" class="ltx_Math" alttext="v/\chi" display="inline"><mrow><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">v</mi><mo mathcolor="#0000FF" mathsize="83%" stretchy="false">/</mo><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">χ</mi></mrow></math> Morgans refers to how closely linked lineages must be to hitchhike (spatially) with a selected allele. </span></em></p>
  
</div>
  
<div id="p108" class="ltx_para">
    
<p class="ltx_p">- line 467: another answer is that it is unlikely that all the alleles will strictly have the same beneficial effect <math xmlns="http://www.w3.org/1998/Math/MathML" id="p108.m1" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">b</mi></msub></math>!</p>
  
</div>
  
<div id="p109" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We have extended the text to reflect this.</span></em></p>
  
</div>
  
<div id="p110" class="ltx_para">
    
<p class="ltx_p"><span class="ltx_text ltx_font_bold">Typos that will probably be anyway corrected when the article is typeset</span></p>
  
</div>
  
<div id="p111" class="ltx_para">
    
<p class="ltx_p">- wide-spread or widespread (lines 57, 59)</p>
  
</div>
  
<div id="p112" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> corrected</span></em></p>
  
</div>
  
<div id="p113" class="ltx_para">
    
<p class="ltx_p">- line 225: that (thar)
<em class="ltx_emph"> <span class="ltx_text" style="color:#0000FF;"> Fixed.</span></em>
</p>
  
</div>
  
<div id="p114" class="ltx_para">
    
<p class="ltx_p">- line 495: citet instead of citep</p>
  
</div>
  
<div id="p115" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Fixed.</span></em></p>
  
</div>
  
<div id="p116" class="ltx_para">
    
<p class="ltx_p">- the abstract on the title page contains a few typos.</p>
  
</div>
  
<div id="p117" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Thanks for all your useful comments Flo!</span></em></p>
  
</div>
  
  
<div id="p118" class="ltx_para">
    
<p class="ltx_p"><span class="ltx_text ltx_font_bold">Reviewer 2 (Sam)</span></p>
  
</div>
  
<div id="p119" class="ltx_para">
    
<p class="ltx_p">The authors have presented an intriguing approach to the problem of convergent genetic evolution, showing the relative importance of standing variation vs. new mutations and how spatial structure interacts with this. Generally, the manuscript is clearly written and covers a lot of new and interesting ground using very nice analytical models. I very much appreciate the presentation and discussion of the results and the big-picture issues covered at the end on speciation and extinction. As I am not primarily a mathematician, there are certain places where I struggled, and I have identified a few spots that could be clarified to make it easier to understand for a general audience. The derivations appear to be correct and to have made reasonable assumptions, but I did not re-derive them exhaustively.</p>
  
</div>
  
<div id="p120" class="ltx_para">
    
<p class="ltx_p">I have a few comments about the discussion and the general implications of this paper, that might be worth considering. I would like to see more discussion of when this kind of evolution will matter, and if/when we can ”safely ignore” it. The examples used in the paper are human-focused, but many species have much smaller total population sizes and ranges, and might never experience this kind of evolution. Is there some way we can get a general prediction about when this might matter?</p>
  
</div>
  
<div id="p121" class="ltx_para">
    
<p class="ltx_p">Consider the following interpretation of Eq. 4: if we want to know whether convergent genetic evolution will occur at all in a given species, then the scale we are interested in is the entire species range, and rho = N. If convergent genetic evolution is unimportant, then the characteristic length has to be larger than the total species range size, and we can find out where this happens by setting chi = 1 and asking what parameters result in <math xmlns="http://www.w3.org/1998/Math/MathML" id="p121.m1" class="ltx_Math" alttext="\chi&lt;1" display="inline"><mrow><mi mathsize="83%" stretchy="false">χ</mi><mo mathsize="83%" stretchy="false">&lt;</mo><mn mathsize="83%" stretchy="false">1</mn></mrow></math>. This simplifies things quite a bit, as simplifies the <math xmlns="http://www.w3.org/1998/Math/MathML" id="p121.m2" class="ltx_Math" alttext="\chi^{2}" display="inline"><msup><mi mathsize="83%" stretchy="false">χ</mi><mn mathsize="83%" stretchy="false">2</mn></msup></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="p121.m3" class="ltx_Math" alttext="\chi^{3}" display="inline"><msup><mi mathsize="83%" stretchy="false">χ</mi><mn mathsize="83%" stretchy="false">3</mn></msup></math> terms, so that we can rearrange the equation to solve for rho (which is now N):</p>
  
</div>
  
<div id="p122" class="ltx_para">
    
<table id="S0.Ex1" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S0.Ex1.m1" class="ltx_Math" alttext="Ncrit==(\sigma*\log[1-s])/(\sqrt{2}\mu\pi*(\sqrt{s}\log[1-s]-\sqrt{2}\sigma))" display="block"><mrow><mi mathsize="83%" stretchy="false">N</mi><mi mathsize="83%" stretchy="false">c</mi><mi mathsize="83%" stretchy="false">r</mi><mi mathsize="83%" stretchy="false">i</mi><mi mathsize="83%" stretchy="false">t</mi><mo mathsize="83%" stretchy="false">=</mo><mo mathsize="83%" stretchy="false">=</mo><mrow><mo maxsize="83%" minsize="83%">(</mo><mi mathsize="83%" stretchy="false">σ</mi><mo mathsize="83%" stretchy="false">*</mo><mi mathsize="83%" stretchy="false">log</mi><mrow><mo maxsize="83%" minsize="83%">[</mo><mn mathsize="83%" stretchy="false">1</mn><mo mathsize="83%" stretchy="false">-</mo><mi mathsize="83%" stretchy="false">s</mi><mo maxsize="83%" minsize="83%">]</mo></mrow><mo maxsize="83%" minsize="83%">)</mo></mrow><mo mathsize="83%" stretchy="false">/</mo><mrow><mo maxsize="83%" minsize="83%">(</mo><msqrt><mn mathsize="83%" stretchy="false">2</mn></msqrt><mi mathsize="83%" stretchy="false">μ</mi><mi mathsize="83%" stretchy="false">π</mi><mo mathsize="83%" stretchy="false">*</mo><mrow><mo maxsize="83%" minsize="83%">(</mo><msqrt><mi mathsize="83%" stretchy="false">s</mi></msqrt><mi mathsize="83%" stretchy="false">log</mi><mrow><mo maxsize="83%" minsize="83%">[</mo><mn mathsize="83%" stretchy="false">1</mn><mo mathsize="83%" stretchy="false">-</mo><mi mathsize="83%" stretchy="false">s</mi><mo maxsize="83%" minsize="83%">]</mo></mrow><mo mathsize="83%" stretchy="false">-</mo><msqrt><mn mathsize="83%" stretchy="false">2</mn></msqrt><mi mathsize="83%" stretchy="false">σ</mi><mo maxsize="83%" minsize="83%">)</mo></mrow><mo maxsize="83%" minsize="83%">)</mo></mrow></mrow></math></td>
<td class="ltx_eqn_pad"></td></tr>
</table>
  
</div>
  
<div id="p123" class="ltx_para">
    
<p class="ltx_p">(see attached mathematica file for a more clear version). If N is less than this value, then the characteristic length would be greater than 1, and convergent genetic evolution would not be expected. I’m sure there are some nicer ways that this could be expressed to show various features of importance when certain parameters are small (for example, when sigma is small, it drops out of the denominator, and then Ncrit is a simple function of sigma in the numerator). The scale of sigma is now expressed as a proportion of the total species range, so sigma can realistically be quite small for some species. Of course, this doesn’t help us know how fast the characteristic length decreases once <math xmlns="http://www.w3.org/1998/Math/MathML" id="p123.m1" class="ltx_Math" alttext="N&gt;Ncrit" display="inline"><mrow><mi mathsize="83%" stretchy="false">N</mi><mo mathsize="83%" stretchy="false">&gt;</mo><mrow><mi mathsize="83%" stretchy="false">N</mi><mo>⁢</mo><mi mathsize="83%" stretchy="false">c</mi><mo>⁢</mo><mi mathsize="83%" stretchy="false">r</mi><mo>⁢</mo><mi mathsize="83%" stretchy="false">i</mi><mo>⁢</mo><mi mathsize="83%" stretchy="false">t</mi></mrow></mrow></math>, but Eq. 4 does that. I’m not sure if I’ve made an obvious blunder in here somewhere, but perhaps this is a useful approach? In any case, it seems worth acknowledging in the discussion that some species are quite unlikely to have convergent genetic evolution, due to their small range size, total population size, or high dispersal distance. The discussion currently says that convergent genetic evolution may be quite common (and line 466 even suggests it may be rare for gene flow to spread a selected allele across the range of a species). Delineating when this is important is especially interesting because the signal of convergent evolution can resemble local adaptation (e.g., line 400-408). If this approach makes sense, it could be discussed along with the section from lines 225-229, which presents a very important conclusion arising from this work.</p>
  
</div>
  
<div id="p124" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Thanks, this is a nice way of interpreting the result! We have added a paragraph giving this intepretation in roughly the locaiton in the text suggested.</span></em></p>
  
</div>
  
  
<div id="p125" class="ltx_para">
    
<p class="ltx_p">Another general consideration is that the trait needs to be non-genetically-redundant for the kind of genetic convergent evolution discussed here, in that there needs to be few genes that could evolve to give the phenotype of interest (ideally, just one, given the model formulation). In both skin pigmentation and malaria resistance, there is only one (or a few) gene(s) that can be mutated to give the phenotype, but many quantitative traits are much more genetically redundant. In a highly genetically redundant case, a very different but interesting prediction can be made: if the characteristic length is small, then we might expect different genetic solutions to the same evolutionary problem in populations separated by distance <math xmlns="http://www.w3.org/1998/Math/MathML" id="p125.m1" class="ltx_Math" alttext="&gt;\chi" display="inline"><mrow><mi></mi><mo mathsize="83%" stretchy="false">&gt;</mo><mi mathsize="83%" stretchy="false">χ</mi></mrow></math> i.e., genetically non-convergent, but phenotypically convergent evolution). This is an interesting corollary of the work that could be explored in the future. Then, the ratio of mutational target size within a single gene vs. mutation target size in all other genes that affect the trait with similar s becomes important for whether evolution is genetically convergent or not. While redundant mutations in quantitative traits would likely have small s, they would also have a higher total target mutation rate across many genes, so species with relatively smaller total size might have small chi and high probability of evolving different genetic solutions for the same phenotype. The section on lines 418-426 touches on this a bit. This suggests that the model could be generalized by considering three classes of mutation with the same (redundant) value of s: 1) same nucleotide, 2) same gene, 3) all other genes. The characteristic length just gives the likelihood of there being multiple independent evolutionary lineages that arise and fix locally, making a tessellation (I think?), and the probability of the alleles being from class 1, 2, or 3 is then just determined by their relative mutational target sizes/rates. The type of convergence/parallelism is then defined by whether it is at the level of QTN (class 1), QTL (class 2), or phenotypic with a different genetic basis (class 3). If the approach I suggested above is sound, then this type of evolution would be common for most quantitative traits in most species (because the high mutation rate at the whole-trait level would compensate for the smaller total population size).</p>
  
</div>
  
<div id="p126" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We are interested in this area, and agree with the reviewer, but we don’t want to expand on it given that the paper is already heavy on discussion. In general we think that highly polygenic traits may often result in very different sets of alleles/loci being used in different populations. We are currently thinking on expanding on this point to form another paper. </span></em>
</p>
  
</div>
  
<div id="p127" class="ltx_para">
    
<p class="ltx_p">Perhaps I am carrying this too far, but it suggests part of the reason why we might sometimes see so many outliers when we use approaches like Bayenv to look at putatively locally adapted quantitative traits. If different populations are evolving in response to a spatially homogenous temporal change in selection pressure (e.g., climate change) via highly redundant traits, then as long as the characteristic length is small, they would evolve different alleles at different loci in different populations. Because climatic variables are correlated with space, it is natural that they would often match up with the tesselations, resulting in associations between snps and local environment. As long as the environment changes faster than the period required for drift to homogenize these responses and wipe out the structure, this signature would be prevalent, but spurious. With suitably rapid changes in environment, alleles could have quite different values of s without enough time to homogenize them across the range of the species (e.g., as discussed around line 316). Obviously, I like how the paper discusses how this kind of evolution would generate signatures resembling local adaptation! I guess I’m arguing that this problem could be much more general, and extend to non-genetically convergent evolution.</p>
  
</div>
  
<div id="p128" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Agreed. A somewhat related argument is being made in the Bierne et al. 2011 we cite in the discussion.</span></em></p>
  
</div>
  
<div id="p129" class="ltx_para">
    
<p class="ltx_p">Some more specific comments:</p>
  
</div>
  
<div id="p130" class="ltx_para">
    
<p class="ltx_p">- It might be helpful to discuss convergence in terms of ”identical by descent” versus ”identical in state”, at least for the case where the same nucleotide is mutating. The phrase ”different or shared genetic routes” (line 31) is a bit ambiguous in this regard, because it’s not clear whether ”genetic routes” is referring to the evolutionary lineage history of the gene or the actual genetic changes being compared. This becomes clear when you read it a few times. There is a similar problem with evolution producing ”different phenotypes” (line 25) because this seems contrary to the definition of convergent evolution (evolution produces similar phenotypes, independently).</p>
  
</div>
  
<div id="p131" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We’ve actually left this ambiguous on purpose, as the results don’t depend on whether the mutations are actually the same,
at the same genes, or in totally different places (as you note). We’ve added clarification of this point in the first paragraph of “Model description”.</span></em></p>
  
</div>
  
  
<div id="p132" class="ltx_para">
    
<p class="ltx_p">- I couldn’t reproduce the exact values for the lines in figures 3 &amp; 4. I’ve attached the approach I used in the same mathematica file as above; the main text says <math xmlns="http://www.w3.org/1998/Math/MathML" id="p132.m1" class="ltx_Math" alttext="\mu=10^{-6}" display="inline"><mrow><mi mathsize="83%" stretchy="false">μ</mi><mo mathsize="83%" stretchy="false">=</mo><msup><mn mathsize="83%" stretchy="false">10</mn><mrow><mo mathsize="83%" stretchy="false">-</mo><mn mathsize="83%" stretchy="false">6</mn></mrow></msup></mrow></math>, while the legend says <math xmlns="http://www.w3.org/1998/Math/MathML" id="p132.m2" class="ltx_Math" alttext="\mu=10^{-5}" display="inline"><mrow><mi mathsize="83%" stretchy="false">μ</mi><mo mathsize="83%" stretchy="false">=</mo><msup><mn mathsize="83%" stretchy="false">10</mn><mrow><mo mathsize="83%" stretchy="false">-</mo><mn mathsize="83%" stretchy="false">5</mn></mrow></msup></mrow></math>, which seems to match better, but Eq. 5 still doesn’t seem to match that way. I assume I did something wrong there but I wanted to mention it just in case.</p>
  
</div>
  
<div id="p133" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Sorry there was a typo in the caption, we are using a mutation rate of <math xmlns="http://www.w3.org/1998/Math/MathML" id="p133.m1" class="ltx_Math" alttext="10^{-6}" display="inline"><msup><mn mathcolor="#0000FF" mathsize="83%" stretchy="false">10</mn><mrow><mo mathcolor="#0000FF" mathsize="83%" stretchy="false">-</mo><mn mathcolor="#0000FF" mathsize="83%" stretchy="false">6</mn></mrow></msup></math> to match the 100 different changes seen at G6PD. We think our calculation is correct, the R code is available (function charLength in standing-var-fns.R). It also gives the correct answers in the limiting cases (no standing variation and only standing variation), so we think it is correctly implemented.</span></em></p>
  
</div>
  
<div id="p134" class="ltx_para">
    
<p class="ltx_p">- line 124-126: I don’t understand why introducing new mutations results in curvature of the tessellations. Wouldn’t it just result in a bunch of smaller-sized cells being intermingled with the larger cells that arose from standing variation, without changing the tesselation shape? I guess I can’t intuitively see why a wave front arising from new mutation would behave differently than a wave front arising from standing variation, but maybe I missed the point here?</p>
  
</div>
  
<div id="p135" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> See response to reviewer 1 above, we now give a reference to a figure in Ralph and Coop 2010 that makes this point clearer.</span></em></p>
  
</div>
  
<div id="p136" class="ltx_para">
    
<p class="ltx_p">- Conte et al. (2012); Proceedings of the Royal Society B) reviewed cases of gene reuse and parallel and convergent evolution, and found a high probability of gene reuse, further suggesting the importance of this kind of evolution.</p>
  
</div>
  
<div id="p137" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Added this reference to the introduction, thanks.</span></em></p>
  
</div>
  
<div id="p138" class="ltx_para">
    
<p class="ltx_p">- line 46: patchy geographically <math xmlns="http://www.w3.org/1998/Math/MathML" id="p138.m1" class="ltx_Math" alttext="\rightarrow" display="inline"><mo mathsize="83%" stretchy="false">→</mo></math> geographically patchy</p>
  
</div>
  
<div id="p139" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Fixed.</span></em></p>
  
</div>
  
<div id="p140" class="ltx_para">
    
<p class="ltx_p">- bracket at the bottom of page 4, ”this approximation will …”: this statement seemed unsubstantiated. Why the log of family size? Perhaps add more justification or move it to somewhere where it fits better?</p>
  
</div>
  
<div id="p141" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> The log follows from the fact that it takes <math xmlns="http://www.w3.org/1998/Math/MathML" id="p141.m1" class="ltx_Math" alttext="\sim log(k)/s" display="inline"><mrow><mi></mi><mo mathcolor="#0000FF" mathsize="83%" stretchy="false">∼</mo><mrow><mrow><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">l</mi><mo mathcolor="#0000FF">⁢</mo><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">o</mi><mo mathcolor="#0000FF">⁢</mo><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">g</mi><mo mathcolor="#0000FF">⁢</mo><mrow><mo mathcolor="#0000FF" maxsize="83%" minsize="83%">(</mo><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">k</mi><mo mathcolor="#0000FF" maxsize="83%" minsize="83%">)</mo></mrow></mrow><mo mathcolor="#0000FF" mathsize="83%" stretchy="false">/</mo><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">s</mi></mrow></mrow></math> generations for a selected mutation with selection coefficient <math xmlns="http://www.w3.org/1998/Math/MathML" id="p141.m2" class="ltx_Math" alttext="s" display="inline"><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">s</mi></math> to have <math xmlns="http://www.w3.org/1998/Math/MathML" id="p141.m3" class="ltx_Math" alttext="k" display="inline"><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">k</mi></math> descendents; we’ve added a note explaining this further.</span></em></p>
  
</div>
  
<div id="p142" class="ltx_para">
    
<p class="ltx_p">- line 168: I’m not clear why <math xmlns="http://www.w3.org/1998/Math/MathML" id="p142.m1" class="ltx_Math" alttext="s_{d}=0.05" display="inline"><mrow><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">d</mi></msub><mo mathsize="83%" stretchy="false">=</mo><mn mathsize="83%" stretchy="false">0.05</mn></mrow></math>, just because <math xmlns="http://www.w3.org/1998/Math/MathML" id="p142.m2" class="ltx_Math" alttext="s_{b}\geq 0.05" display="inline"><mrow><msub><mi mathsize="83%" stretchy="false">s</mi><mi mathsize="83%" stretchy="false">b</mi></msub><mo mathsize="83%" stretchy="false">≥</mo><mn mathsize="83%" stretchy="false">0.05</mn></mrow></math>? Perhaps more description here of why this is the case.</p>
  
</div>
  
  
<div id="p143" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> This is based on heterozygote advantage calculations for the X chromosome given the observed frequency and <math xmlns="http://www.w3.org/1998/Math/MathML" id="p143.m1" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">s</mi><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">b</mi></msub></math>. We have now explained that more fully. However, we have not shown the calculations as similar ones were presented in Ruwende et al..</span></em></p>
  
</div>
  
<div id="p144" class="ltx_para">
    
<p class="ltx_p">- line 190-196: the alternation between the one-dimensional and two-dimensional equations here is a little confusing and could be explained more clearly. When you say you omit the explicit formula, it takes some looking back and forth to figure out that the explicit formula for 2D is still there in Eq. 4, it’s just not rearranged so that there is a single chi term on the left hand side.</p>
  
</div>
  
<div id="p145" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> We have worked to unpack this a little more for clarity. Thanks for pointing this out.</span></em></p>
  
</div>
  
<div id="p146" class="ltx_para">
    
<p class="ltx_p">- line 201: could add ”and standing variation is unimportant” after ”is highly deleterious” to clarify.</p>
  
</div>
  
<div id="p147" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Done.</span></em></p>
  
</div>
  
<div id="p148" class="ltx_para">
    
<p class="ltx_p">- line 316: have an <math xmlns="http://www.w3.org/1998/Math/MathML" id="p148.m1" class="ltx_Math" alttext="\rightarrow" display="inline"><mo mathsize="83%" stretchy="false">→</mo></math> have a</p>
  
</div>
  
<div id="p149" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> fixed.</span></em></p>
  
</div>
  
<div id="p150" class="ltx_para">
    
<p class="ltx_p">- Figure 5: legend is missing a word (”benefit”?)</p>
  
</div>
  
<div id="p151" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> Fixed the caption.</span></em></p>
  
</div>
  
<div id="p152" class="ltx_para">
    
<p class="ltx_p">- line 333-335: If it’s ”feeling” spatial structure, doesn’t that mean that it’s interfering with neighbours? Doesn’t it grow deterministically until it ”feels” structure?</p>
  
</div>
  
<div id="p153" class="ltx_para">
    
<p class="ltx_p"><em class="ltx_emph"><span class="ltx_text" style="color:#0000FF;"> The initial growth is generated by a branching process, where each lineage branch independently at a rate <math xmlns="http://www.w3.org/1998/Math/MathML" id="p153.m1" class="ltx_Math" alttext="1+s_{b}" display="inline"><mrow><mn mathcolor="#0000FF" mathsize="83%" stretchy="false">1</mn><mo mathcolor="#0000FF" mathsize="83%" stretchy="false">+</mo><msub><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">s</mi><mi mathcolor="#0000FF" mathsize="83%" stretchy="false">b</mi></msub></mrow></math>, so its initial growth is not deterministic.</span></em></p>
  
</div>

</div>
</div>
<div class="ltx_page_footer">
<div class="ltx_page_logo">Generated  on Tue Apr 21 17:16:15 2015 by <a href="http://dlmf.nist.gov/LaTeXML/">LaTeXML <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="[LOGO]"></img></a></div></div>
</div>
</body>
</html>
