\documentclass[12pt,letterpaper]{article}
\bibliographystyle{evolution}
\usepackage{epsfig}                 
\usepackage[authoryear]{natbib}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage{psfrag}
\usepackage{mathabx}
\renewcommand{\baselinestretch}{1.6}
\large
\pagenumbering{arabic}
\usepackage[usenames,dvipsnames]{color}
\usepackage{fullpage}
\usepackage{multirow}
\newcommand{\ye}{\hat{y}}
\newcommand{\xe}{\hat{x}}
\usepackage{color}
\usepackage[normalem]{ulem}  
	\newcommand{\gc}[1]{{ \color{red} #1}}
	\newcommand{\plr}[1]{{ \color{green} #1}}
	\newcommand{\yb}[1]{{ \color{blue} #1}}

	\newcommand{\response}[1]{\emph{ \color{blue} #1}}

\usepackage{lineno}
%\linenumbers 
\begin{document}

Dear Michael,\\
Thanks to Flo, Sam, and you for all of the useful comments, they were very helpful. We have revised the manuscript in light of those comments, and additional comments that we received from others. We have responded point by point to the reviews below (our responses are in blue). In the main text changes made in response to the reviewers are in blue, changes made by us in response to other comments are marked in red.

 
Best wishes\\
Peter Ralph and Graham Coop\\
4/7/15
\newpage 


%Dear Graham and Peter-

%Thanks for your nice paper on the geographic patterns of convergent evolution. I really like the ideas in this paper, and it is going to be a nice match to the special issue.

%I have two reviews that also both like the paper (see below). Also I have read it carefully myself, and I'll add my comments below. I think the general consensus is that the paper covers very interesting and novel territory, but perhaps it could be easier to read with some revision.

%Please use the comments to make the paper better in the ways that you see best. I think that you will be able to use some of these comments in ways that you will agree are improvements. I look forward to the revision. This already is a nice paper.

%Thanks for contributing to this issue. I think it is going to turn out to be pretty nice.


%All my best,
%Mike

{\bf Mike's comments:}

I'll add my comments in order that they appear in the presentation of the paper, rather than trying to rank them by importance.

64 --ish: Be clear about whether you are assuming that this variation is allelic

\response{Good point. We have extended our discussion there. }


68: Especially as this is in an issue about local adaption, you probably want to be very clear there that the alleles have the same fitness everywhere.

\response{We have expanded on this.}

77: ``below''? \& 77: ``we assume to be equal to one'' -- re-write please, it isn't clear at first reading what you are assuming to be one.

\response{We have edited this in response to reviewers' comments.}


After 90: ``somewhat larger probability of local establishment'' -- it is not at all clear to me what local establishment means in the case of a deleterious allele. Can you unpack this please?

\response{We meant establishment following the environmental shift, we have altered the text to reflect that.}

Equation 2: If the mutation selection balance is $\mu/s_d$, then the number of copies of the allele on average per unit area is $\rho mu/ s_d$. The probability that this leads to a successfully establishment is $2 s_b$ per copy approximately, which gives $2 s_b \mu \rho / s_d$ for $\lambda_0$. This seems like an easier derivation to me. (For large $s_d$, this diverges from your result -- which is more correct?)

\response{This difference arises from taking a continuous time approach. We have extended the description here to more clearly give the ``classic'' mutation-selection balance interpretation. We think it is important to work through this in the simple spatial case (via branching processes) rather than just move straight from the result people are familiar with deriving under the panmictic model. This allows folks to see that the result is valid in our spatial setting. }

%\gc{Peter is the difference coming about in part because the deterministic result is an approximation? Or because we are working in continuous time (or another reason?)?}
%\plr{Continuous time: in a discrete-time model, the mean density of individuals resulting from a rate-1 Poisson rain of branching processes would be $1/s_d$; in continuous time, it would be $1/\log(1/(1-s_d))$.  So, I wouldn't say that one is more valid that the other.  Hmph, confusing.}

P3, somewhere. It would be worth explicitly giving the formula for lambda somewhere.

\response{Done.}

Figure 1: The grey cone for (x,t) is too faint to see easily. I didn't notice it until trying to find what the caption was referring to. And in the caption, it would be worth explaining why you are looking at this cone before describing it. 

\response{Good points. We have edited the caption and darkened the cone. }

192: You need to be more specific about where these two formulas for the means come from. It is not clear at all.

\response{We have expanded on this section to explain how these calculations are performed.}

210: It may be confusing to refer to standing variation as mutational input. If they already exist they are not mutations anymore, but alleles.

\response{Changed}

225-229 -- Cool!

\response{Thanks! We like this point.}

231-232 -- I thought the first person here ``ourselves'' was a little jarring.

\response{Changed.}

246: Can you use something other than nu? It looks too much like the $v$ that you have already used for something else.
\response{Changed to $\gamma$}

Figure 5: The caption was missing words or symbols in my copy. Also remove the last comma.

\response{Fixed.}

250: Perhaps this would be better as ``the mean proportion of adapted patches''

\response{Done.}
251: Please give a reference for the Palm measure.

\response{Added more description and a reference.}

Figure 6, and many other places, especially in the discussion. You assume a quite large mutation rate here. This may be appropriate for a loss of function mutation ( is this the case for this example?), but it is very high for many types of mutations per locus. Many of your conclusions would be very different with lower mutation rates. It would be nice to know ( at least qualitatively ) what would happen if the mutation rates were lower. You say that convergent evolution may be quite high (e.g. start of discussion), but this is not true if the mutation rate were an order of magnitude or two lower than you assume here, I think. 

\response{We have expanded our discussion at the start of the discussion to make this clearer. The G6PD alleles are not knockouts but do correspond to reduced activity. While a 100bp target may be high for some traits, it perhaps is not when you consider all of the possible genes in which a change could occur. Also if you move to considering smaller target sizes the time to adaptation goes up considerably, both as there is a longer waiting time for new mutations to arise and standing variants are much rarer. }


284 plus a couple of lines: The sentence fragment beginning with ``Or, \ldots'' is a sentence fragment.

\response{Modified.}

Discussion, somewhere: Can you say anything about the pattern of differentiation of these convergent selected genes relative to neutral differentiation? 

\response{Added a sentence to the ``The confusing signal of geographic convergent evolution'' section pointing out that they will stick out against neutral variation. In Ralph and Coop 2010 we gave a more complete version of this argument. However, in practice it will depend on the true per generation rate of migration among demes, so it's hard to say anything to definite when that is rarely known.}

\newpage


{\bf Reviewer 1 (Flo)}\\

%  Extending their previous work on the topic, Ralph \& Coop investigate the role of standing variation in geographic convergent adaptation, or, in other words, how already existing mutations contribute to convergent adaptation (the independent evolution and coexistence of multiple beneficial alleles) in a geographically spread population subjected to a switch in selection pressures. The authors identify a characteristic length $\chi$, which gives the spatial scale at which different adapted mutants will establish. If the scale of the environment is larger than $\chi$, we can expect geographic convergent evolution to occur or have occurred. Importantly, and particularly relevant for the themed issue, the authors warn that the resulting pattern of geographic convergent adaptation can be mistaken for a pattern local adaptation.

The manuscript is well written, interesting, insightful, and dense (in a good way!). It is also sometimes a bit hard to read. The main issue is that it is rather likely that the average AmNat reader will not have read Ralph \& Coop 2010, or, even if they have, they will not remember some specific details. I had to read Ralph \& Coop 2010 (hereafter RC10) to better understand this manuscript, so I will try to highlight the points that could be clarified, in the hope that this will help make the manuscript more accessible to less mathematically-inclined readers.

\textbf{Comments on the model and the analysis (clarifications needed)}

- The introduction is written as if it were obvious that there is a switch in selection pressures (e.g. line 41), while other scenarios are conceivable (e.g. a species encountering a novel habitat type while expanding its range; or, other types of temporal variation (not just a switch)). Maybe clarify near the beginning of the introduction the type of environmental variation you are considering.

\response{Good Points. We had originally wanted to avoid too much overlap with the previous paper, but agree that makes it hard to read as a stand-alone paper. We have extended our introduction to more fully describe our previous work on the topic, the homogeneous nature of the selection pressure, and adding a simulation figure to illustrate the previous intuition. }

- line 64: it is mainly about vocabulary here: the word "class" is used to refer to the "types", while in section 1.6 "class" is used to refer to the diversity of mutations (i.e., within type variability).

\response{Good point, although it is consistent in calling a ``class'' a category of mutations with distinct properties? We've modified section 1.6 a bit to set the usage up like this.}


- line 65: add "neutral" before "variation" (if appropriate)

\response{added neutral, thanks.}

- somewhere in the model description, it would be good to mention that time is measured in generations (and clarify whether it is continuous/discrete).


\response{Good point, added a sentence to ``Rates of origination of standing and new mutations''.}

- line 76-77: this sentence was opaque to me until I read RC10 and the formula $2s/\xi^2$.

\response{In hindsight that is pretty cryptic. We changed this to ``We assume that the variance in offspring number is one, see
  Ralph \& Coop 2010 for how this assumption can be relaxed.''}

- line 86: the way mutation works is not clear yet. If I understand correctly, a neutral type's offspring is mutated with probability $\mu$, but there are no further mutations among the mutated types (i.e. there cannot be a new cone growing within a cone). So "each new offspring" is the offspring of neutral type individuals, right? It could be also useful to recap somewhere the main assumptions of the analysis, e.g. the decomposition of time scales between local establishment and then spatial spread (which is discussed in 1.7), or why there are not further mutants emerging within mutant cones.

\response{We have now added ``In geographic areas already
 occupied by the mutant type (i.e. in regions it has spread to already), new mutations are selectively excluded
 (are effectively neutral), and so unlikely to establish over the
 short-time scales considered here.''. Hopefully that clarifies that we are not banning subsequent mutations, only assuming that they are not selectively favoured.}

- line 88: It would be great to add a few lines to explain what it is a Poisson process in space and time (the explanation in RC10, p651, is very clear).

\response{Added text to the section ``Rates of origination of standing and new mutations'' to help with this.} 

- [unnumbered line that would otherwise be 94] "*Since* the times and locations \ldots" has it been proven? \& [unnumbered line that would otherwise be 96] "Poisson Mapping Theorem" maybe add reference?

\response{We have edited this part for clarity.}


- [unnumbered line that would otherwise be 103] Using the letter $s$ to refer to a time is a bit confusing!!
(same line 334, where $s$ is now a rate)

\response{Good point, changed this around.}

- [unnumbered line that would otherwise be 106] Do you actually need to use $\zeta$? Could not the equation that follows simply be written with $E[(1-p_s)^{Z_T}$]? Also, isn't $\zeta(u,T)$ the generating function of $Z_T$?

\response{Sure, good suggestion.  We've removed $\zeta$.}

- equation (2), since this will be used later on, maybe add a line $\lambda_0 = \lambda / (-\log(1-s_d))$.

\response{Added.}

- Figure 1: I think that it would really help the reader if the figure were in 3D (space x space x time) instead of 2D (space x time), especially to visualize the "cones" carved out in space-time. Also, the lightening bolts give the false impression that the mutants appear right at $t=0$.

\response{We have kept this figure relatively unchanged, although we agree that it's not perfect. We are not convinced that showing the picture in 3D would help, as other details would have to be omitted. The lightening bolts do show the standing variants at time $t=0$, rather than the original mutations, so we have modified the caption to indicate this.}

- line 208: higher population densities (adjective missing)

\response{Added higher.}

- equation (9) I do not understand where it comes from! (i.e. how using the proportionality between $\lambda_0$ and $\lambda$ give this result). Do we actually need (9) anyway to obtain expression (10)?

\response{Thanks for spotting this slip, we had accidentally merged together two different results and a typo. We have corrected this now.}

- line 251: I do not know what is the "Palm measure", so a reference would be welcome here (or consider removing?)

\response{Added more description and a reference.}

- line 343: I do not understand why there is a factor 4 here (and not 2).

\response{Sorry, we had accidentally reverted to $\rho$ being the density of diploids, we've changed these to a factor or 2 instead of 4. Thanks! }

- equation (16) I am not sure about the exponents: I find that the exponent for $\beta$ is $-(1+c)/(d+1)$ and that the exponent for u (outside of exp) is $(c-d)/(d+1)$, but I may be wrong?

\response{In the end we evaluate all of the integrals numerically, without the aid of this trick, so we have now dropped the entire appendix section.}

\textbf{Minor remarks}

- line 58: "the genetics of adaptation may often be geographically local" the meaning of this phrases was not obvious at first.

\response{edited this sentence.}

- line 61: A more natural order seems to be first history, then implications.

\response{done}

- line 80-81: If there are more than 1 chromosome per haploid individual, this is not exactly the same?

\response{We meant that diploids carried twice the number of alleles as haploids. Edited in response.}

- line 91: that's probably personal, but I usually prefer when the terms are grouped depending on where they come from, and so I would find it clearer to write $2 s_b \mu \rho$.

\response{changed.}

[- bottom of page 4, the lines are unnumbered, but you can avoid this bug of the lineno package by leaving an empty space before begin equation]

- line 126: I do not understand which the sides curve, but this is probably not a crucial point!

\response{When there is just standing variation, we simple need to bisect up the area into regions closer to one origin or another, that leads to polygons. However, when the mutations are arising through time we have to consider whether a point lands in the cone of one mutation or another, this leads to curved edges. We've added a reference to Ralph and Coop 2010, which graphically depicted this point more clearly, and the teaser ``because the radii of the colliding circles differ'' (as it is the special case of equal radii circles that leads to the special property of Voronoi cells).  A longer explanation: when two expanding circles meet, the direction their intersections move in is the average of their tangents; for non-equally sized circles (as with different ages), this direction changes as they expand.}

- line 200 "close to the value obtained just from standing variation" obtained by whom/how? \&  line 202: "close to the value from" RC10: how different are the values and why?


\response{Sorry, we meant obtain by us, and by ``close to'' we meant that they approach these limits at these extremes. Edited accordingly.}


- line 216: add "locally" in front of "established", if appropriate.

\response{added.}

- line 219: add "of order" or an equivalent in front of $s_b^{1/2}$, if appropriate.

\response{added.}

- Figure 2: Not sure why "Both" is in black in the legend with the two other line types are in gre/ay.

\response{Changed this over to being grey for all three.}

- line 258: the expression for the "sliver" is actually not used in (11), but will be useful in (13).

\response{Agreed, but we think it is useful to introduce all of the components first.}

- the paragraph lines 304--323 seems to rather belong to the Discussion section.

\response{We agree that they are discussion flavo(u)red but they naturally go here, having to move this into the discussion would complicate the (already long) discussion more. So we think this is better left here.}

- line 336: "size" could refer to population size or the area of a patch, maybe add "population" to clarify?

\response{We meant the size (i.e. count) of the branching process. Added to text.} 


- line 354: "total amount of area" isn't the environment effectively infinite? Or should one rather understand "sampled area"?

\response{In practice we mean the area where the selection pressure is present.}

- line 419--420: I am not sure I understand here; do you mean that the distant populations could share the same allele? But with the work you have done, including standing variance, shouldn't we expect them to be of independent origin, even if they are neutral?

\response{We think is correct as written. We are referring to a model where a common allele (of single origin) is wide-spread before selection pressures switch. Let us know if we need to clarify further.}

- line 459: distance between the lineages is in time$^{-1}$, but the displacement is in distance?


\response{Clarified this sentence to make clear that $ v/\chi$ Morgans refers to how closely linked lineages must be to hitchhike (spatially) with a selected allele. }

- line 467: another answer is that it is unlikely that all the alleles will strictly have the same beneficial effect $s_b$!

\response{We have extended the text to reflect this.}

\textbf{Typos that will probably be anyway corrected when the article is typeset}

- wide-spread or widespread (lines 57, 59) 

\response{corrected}

- line 225: that (thar)
\response{Fixed.}

- line 495: citet instead of citep

\response{Fixed.}

- the abstract on the title page contains a few typos.

\response{Thanks for all your useful comments Flo!}


\newpage 
{\bf Reviewer 2 (Sam)}

The authors have presented an intriguing approach to the problem of convergent genetic evolution, showing the relative importance of standing variation vs. new mutations and how spatial structure interacts with this. Generally, the manuscript is clearly written and covers a lot of new and interesting ground using very nice analytical models. I very much appreciate the presentation and discussion of the results and the big-picture issues covered at the end on speciation and extinction. As I am not primarily a mathematician, there are certain places where I struggled, and I have identified a few spots that could be clarified to make it easier to understand for a general audience. The derivations appear to be correct and to have made reasonable assumptions, but I did not re-derive them exhaustively.

I have a few comments about the discussion and the general implications of this paper, that might be worth considering. I would like to see more discussion of when this kind of evolution will matter, and if/when we can "safely ignore" it. The examples used in the paper are human-focused, but many species have much smaller total population sizes and ranges, and might never experience this kind of evolution. Is there some way we can get a general prediction about when this might matter?

Consider the following interpretation of Eq. 4: if we want to know whether convergent genetic evolution will occur at all in a given species, then the scale we are interested in is the entire species range, and rho = N. If convergent genetic evolution is unimportant, then the characteristic length has to be larger than the total species range size, and we can find out where this happens by setting chi = 1 and asking what parameters result in $\chi < 1$. This simplifies things quite a bit, as simplifies the $\chi^2$ and $\chi^3$ terms, so that we can rearrange the equation to solve for rho (which is now N):

$$Ncrit == (\sigma * \log[1-s]) / (\sqrt{2}  \mu  \pi * (\sqrt{s} \log[1-s] - \sqrt{2}\sigma) )$$

(see attached mathematica file for a more clear version). If N is less than this value, then the characteristic length would be greater than 1, and convergent genetic evolution would not be expected. I'm sure there are some nicer ways that this could be expressed to show various features of importance when certain parameters are small (for example, when sigma is small, it drops out of the denominator, and then Ncrit is a simple function of sigma in the numerator). The scale of sigma is now expressed as a proportion of the total species range, so sigma can realistically be quite small for some species. Of course, this doesn't help us know how fast the characteristic length decreases once $N > Ncrit$, but Eq. 4 does that. I'm not sure if I've made an obvious blunder in here somewhere, but perhaps this is a useful approach? In any case, it seems worth acknowledging in the discussion that some species are quite unlikely to have convergent genetic evolution, due to their small range size, total population size, or high dispersal distance. The discussion currently says that convergent genetic evolution may be quite common (and line 466 even suggests it may be rare for gene flow to spread a selected allele across the range of a species). Delineating when this is important is especially interesting because the signal of convergent evolution can resemble local adaptation (e.g., line 400-408). If this approach makes sense, it could be discussed along with the section from lines 225-229, which presents a very important conclusion arising from this work.

\response{Thanks, this is a nice way of interpreting the result! We have added a paragraph giving this intepretation in roughly the locaiton in the text suggested.}

%\gc{

%Not sure this helps much. Thoughts Peter? By equating $\rho$ to N is he implicitly measuring $\sigma$ in units of range radius?}
%\plr{I like it.  (and, he mentions that $\sigma$ is as a fraction of the range, er, should be range diameter, I think.)
%What this comes down to is $N > \frac{ \sigma }{ \sqrt{2 s_b} \mu \pi ( 1 + \sigma \sqrt{s_b}/\log(1/(1-s_d)))}$,
%which since $\pi \sqrt{2s_b}$ will be something of order 1, is roughly just $N > \sigma/\mu$, or $N \mu > \sigma$.
%This is a bit surprising, but very concrete.  What do you think?
%}

Another general consideration is that the trait needs to be non-genetically-redundant for the kind of genetic convergent evolution discussed here, in that there needs to be few genes that could evolve to give the phenotype of interest (ideally, just one, given the model formulation). In both skin pigmentation and malaria resistance, there is only one (or a few) gene(s) that can be mutated to give the phenotype, but many quantitative traits are much more genetically redundant. In a highly genetically redundant case, a very different but interesting prediction can be made: if the characteristic length is small, then we might expect different genetic solutions to the same evolutionary problem in populations separated by distance $> \chi$ i.e., genetically non-convergent, but phenotypically convergent evolution). This is an interesting corollary of the work that could be explored in the future. Then, the ratio of {mutational target size within a single gene} vs. {mutation target size in all other genes that affect the trait with similar s} becomes important for whether evolution is genetically convergent or not. While redundant mutations in quantitative traits would likely have small s, they would also have a higher total target mutation rate across many genes, so species with relatively smaller total size might have small chi and high probability of evolving different genetic solutions for the same phenotype. The section on lines 418-426 touches on this a bit. This suggests that the model could be generalized by considering three classes of mutation with the same (redundant) value of s: 1) same nucleotide, 2) same gene, 3) all other genes. The characteristic length just gives the likelihood of there being multiple independent evolutionary lineages that arise and fix locally, making a tessellation (I think?), and the probability of the alleles being from class 1, 2, or 3 is then just determined by their relative mutational target sizes/rates. The type of convergence/parallelism is then defined by whether it is at the level of QTN (class 1), QTL (class 2), or phenotypic with a different genetic basis (class 3). If the approach I suggested above is sound, then this type of evolution would be common for most quantitative traits in most species (because the high mutation rate at the whole-trait level would compensate for the smaller total population size).

\response{We are interested in this area, and agree with the reviewer, but we don't want to expand on it given that the paper is already heavy on discussion.  In general we think that highly polygenic traits may often result in very different sets of alleles/loci being used in different populations. We are currently thinking on expanding on this point to form another paper. }
%\plr{er, should say a bit more here, but unsure how to say something that's not ``yes, we know.'' Isn't there more discussion of this in RC10?}

Perhaps I am carrying this too far, but it suggests part of the reason why we might sometimes see so many outliers when we use approaches like Bayenv to look at putatively locally adapted quantitative traits. If different populations are evolving in response to a spatially homogenous temporal change in selection pressure (e.g., climate change) via highly redundant traits, then as long as the characteristic length is small, they would evolve different alleles at different loci in different populations. Because climatic variables are correlated with space, it is natural that they would often match up with the tesselations, resulting in associations between snps and local environment. As long as the environment changes faster than the period required for drift to homogenize these responses and wipe out the structure, this signature would be prevalent, but spurious. With suitably rapid changes in environment, alleles could have quite different values of s without enough time to homogenize them across the range of the species (e.g., as discussed around line 316). Obviously, I like how the paper discusses how this kind of evolution would generate signatures resembling local adaptation! I guess I'm arguing that this problem could be much more general, and extend to non-genetically convergent evolution.

\response{Agreed. A somewhat related argument is being made in the Bierne et al. 2011 we cite in the discussion.}

Some more specific comments:

- It might be helpful to discuss convergence in terms of "identical by descent" versus "identical in state", at least for the case where the same nucleotide is mutating. The phrase "different or shared genetic routes" (line 31) is a bit ambiguous in this regard, because it's not clear whether "genetic routes" is referring to the evolutionary lineage history of the gene or the actual genetic changes being compared. This becomes clear when you read it a few times. There is a similar problem with evolution producing "different phenotypes" (line 25) because this seems contrary to the definition of convergent evolution (evolution produces similar phenotypes, independently).

\response{We've actually left this ambiguous on purpose, as the results don't depend on whether the mutations are actually the same,
at the same genes, or in totally different places (as you note).  We've added clarification of this point in the first paragraph of ``Model description''.}

- I couldn't reproduce the exact values for the lines in figures 3 \& 4. I've attached the approach I used in the same mathematica file as above; the main text says $\mu = 10^{-6}$, while the legend says $\mu=10^{-5}$, which seems to match better, but Eq. 5 still doesn't seem to match that way. I assume I did something wrong there but I wanted to mention it just in case.

\response{Sorry there was a typo in the caption, we are using a mutation rate of $10^{-6}$ to match the 100 different changes seen at G6PD. We think our calculation is correct, the R code is available (function charLength in standing-var-fns.R). It also gives the correct answers in the limiting cases (no standing variation and only standing variation), so we think it is correctly implemented.}



- line 124-126: I don't understand why introducing new mutations results in curvature of the tessellations. Wouldn't it just result in a bunch of smaller-sized cells being intermingled with the larger cells that arose from standing variation, without changing the tesselation shape? I guess I can't intuitively see why a wave front arising from new mutation would behave differently than a wave front arising from standing variation, but maybe I missed the point here?

\response{See response to reviewer 1 above, we now give a reference to a figure in Ralph and Coop 2010 that makes this point clearer.}

- Conte et al. (2012); Proceedings of the Royal Society B) reviewed cases of gene reuse and parallel and convergent evolution, and found a high probability of gene reuse, further suggesting the importance of this kind of evolution.

\response{Added this reference to the introduction, thanks.}

- line 46: patchy geographically $\rightarrow$ geographically patchy

\response{Fixed.}

- bracket at the bottom of page 4, "this approximation will \ldots": this statement seemed unsubstantiated. Why the log of family size? Perhaps add more justification or move it to somewhere where it fits better?

\response{The log follows from the fact that it takes $\sim log(k)/s$ generations for a selected mutation with selection coefficient $s$ to have $k$ descendents; we've added a note explaining this further.}


- line 168: I'm not clear why $s_d = 0.05$, just because $s_b \ge 0.05$? Perhaps more description here of why this is the case.

\response{This is based on heterozygote advantage calculations for the X chromosome given the observed frequency and $s_b$. We have now explained that more fully. However, we have not shown the calculations as similar ones were presented in Ruwende et al..}

- line 190-196: the alternation between the one-dimensional and two-dimensional equations here is a little confusing and could be explained more clearly. When you say you omit the explicit formula, it takes some looking back and forth to figure out that the explicit formula for 2D is still there in Eq. 4, it's just not rearranged so that there is a single chi term on the left hand side.

\response{We have worked to unpack this a little more for clarity. Thanks for pointing this out.}

- line 201: could add "and standing variation is unimportant" after "is highly deleterious" to clarify.

\response{Done.}

- line 316: have an $\rightarrow$ have a

\response{fixed.}

- Figure 5: legend is missing a word ("benefit"?)

\response{Fixed the caption.}

- line 333-335: If it's "feeling" spatial structure, doesn't that mean that it's interfering with neighbours? Doesn't it grow deterministically until it "feels" structure?

\response{The initial growth is generated by a branching process, where each lineage branch independently at a rate $1+s_b$, so its initial growth is not deterministic.}

\end{document}
