<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>The role of standing variation in geographic convergent adaptation</title>
<!--Generated on Wed Apr 15 16:42:24 2015 by LaTeXML (version 0.8.1) http://dlmf.nist.gov/LaTeXML/.-->

<meta http-equiv="Content-Type" content="application/xhtml+xml; charset=UTF-8"></meta>
<link rel="stylesheet" href="LaTeXML.css" type="text/css"></link>
<link rel="stylesheet" href="ltx-article.css" type="text/css"></link>
<link rel="stylesheet" href="resources/plr-style.css" type="text/css"></link>
<script src="resources/LaTeXML-maybeMathjax.js" type="text/javascript"></script>
<script src="resources/adjust-svg.js" type="text/javascript"></script>
</head>
<body>
<div class="ltx_page_main">
<div class="ltx_page_content">
<div class="ltx_document ltx_authors_1line">
  
  
  
<h1 class="ltx_title ltx_title_document">The role of standing variation in geographic convergent adaptation</h1>

  
<div class="ltx_creator ltx_role_author">
    
<div class="ltx_personname">
Peter L. Ralph<math xmlns="http://www.w3.org/1998/Math/MathML" id="m1" class="ltx_Math" alttext="{}^{*,1}" display="inline"><msup><mi></mi><mrow><mo>*</mo><mo>,</mo><mn>1</mn></mrow></msup></math> 
<br class="ltx_break"></br>email: pralph@usc.edu

<br class="ltx_break"></br>and
<br class="ltx_break"></br>Graham Coop<math xmlns="http://www.w3.org/1998/Math/MathML" id="m2" class="ltx_Math" alttext="{}^{*,2}" display="inline"><msup><mi></mi><mrow><mo>*</mo><mo>,</mo><mn>2</mn></mrow></msup></math> 
<br class="ltx_break"></br>email: gmcoop@ucdavis.edu

</div>
  
</div>
  
<div id="p1" class="ltx_para">
    
<p class="ltx_p ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="p1.m1" class="ltx_Math" alttext="{}^{*}" display="inline"><msup><mi></mi><mo>*</mo></msup></math> These authors contributed equally to this work.</p>
    
<p class="ltx_p ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="p1.m2" class="ltx_Math" alttext="{}^{1}" display="inline"><msup><mi></mi><mn>1</mn></msup></math> Computational Biology and Bioinformatics</p>
    
<p class="ltx_p ltx_align_center">University of Southern California, Los Angeles, CA</p>
    
<p class="ltx_p ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="p1.m3" class="ltx_Math" alttext="{}^{2}" display="inline"><msup><mi></mi><mn>2</mn></msup></math> Center for Population Biology &amp; Department of Evolution and Ecology</p>
    
<p class="ltx_p ltx_align_center">University of California ‚Äì Davis, Davis, CA, 95616</p>
    
  
</div>
  
  
<div id="Sx1" class="ltx_section">
    
<h2 class="ltx_title ltx_title_section">Abstract</h2>

    
<div id="Sx1.p1" class="ltx_para">
      
<p class="ltx_p">The extent to which populations experiencing shared selective pressures
adapt through a shared genetic response
is relevant to many questions in evolutionary biology.
In a number of well studied traits and species,
it appears that convergent evolution within species is common.
In this paper,
we explore how standing, genetic variation contributes to
convergent genetic responses in a geographically spread population,
extending our previous work on the topic.
Geographically limited dispersal slows the spread of each selected allele,
hence allowing other alleles
‚Äì newly arisen mutants or present as standing variation ‚Äì
to spread before any one comes to dominate the population.
When such alleles meet, their progress is substantially slowed
‚Äì if the alleles are selectively equivalent,
they mix slowly, dividing the species range into a random tessellation,
which can be well understood by analogy to a Poisson process model of crystallization.
In this framework,
we derive the geographic scale over which a typical allele is expected to dominate,
the time it takes the species to adapt as a whole,
and the proportion of adaptive alleles that arise from standing variation.
Finally, we explore how negative pleiotropic effects of alleles before an environment change
can bias the subset of alleles that contribute to the species‚Äô adaptive response.
We apply the results to the many geographically localized G6PD
deficiency alleles thought to confer resistance to malaria, <span class="ltx_text ltx_font_italic" style="color:#FF0000;">where the large
mutational target size makes it a likely candidate for adaptation
from standing variation, despite the selective cost of G6PD
deficiency alleles in the absence of malaria.</span>
We find the numbers and geographic spread of these alleles matches our predictions reasonably well,
<span class="ltx_text ltx_font_italic" style="color:#FF0000;">consistent with the view that they arose from a combination of</span>
standing variation and new mutations since the advent of malaria.
Our results suggest that much of adaptation may be geographically local
even when selection pressures are homogeneous.
<span class="ltx_text ltx_font_italic" style="color:#FF0000;">Therefore, we argue that caution must be exercised when arguing that strongly
geographically restricted alleles are necessarily the outcome of local adaptation.</span>
We close by discussing the implications of these results for
ideas of species coherence and the nature of divergence between species.</p>
    
</div>
    
  
</div>
  
<div id="S1" class="ltx_section">
    
<h2 class="ltx_title ltx_title_section"><span class="ltx_tag ltx_tag_section">1 </span>Introduction</h2>

    
<div id="S1.p1" class="ltx_para">
      
<p class="ltx_p">There are an increasing number of examples where
different populations within a species have
adapted to similar environments by means of independent genetic changes.
In some cases this convergent evolution <span class="ltx_text ltx_font_italic" style="color:#FF0000;">is the result of quite
distinct genetic changes, involving very different genes and pathways,</span> despite shared selection pressures;
in other cases independent adaptations are identical down the same
nucleotide change
<cite class="ltx_cite">(Jeong and Rienzo, <a href="#bib.bib32" title="Adaptations to local environments in modern human populations" class="ltx_ref">2014</a>; Stern, <a href="#bib.bib161" title="The genetic causes of convergent evolution" class="ltx_ref">2013</a>; Martin and Orgogozo, <a href="#bib.bib33" title="The Loci of repeated evolution: a catalog of genetic hotspots of phenotypic variation" class="ltx_ref">2013</a>; Conte<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib8" title="The probability of genetic parallelism and convergence in natural populations" class="ltx_ref">2012</a>)</cite>.
Such convergent evolution within populations has been seen for many
carefully studied phenotypes across a range of species,
including drug resistance in pathogens,
resistance to pathogens or pesticides,
and the molecular basis of pigmentation changes.
The phrase ‚Äúparallel evolution‚Äù is also used to refer to such convergent evolution;
here we use these synonymously,
as we are concerned with adaptation within a single
species that can occur via a different or shared genetic routes <cite class="ltx_cite">(see Arendt and Reznick, <a href="#bib.bib101" title="Convergence and parallelism reconsidered: what have we learned about the genetics of adaptation?" class="ltx_ref">2008</a>, for more discussion)</cite>.</p>
    
</div>
    
<div id="S1.p2" class="ltx_para">
      
<p class="ltx_p">The issue of convergent adaptation within species touches on a number
of important questions in evolutionary biology.
These include the extent to which adaptation is shaped by pleiotropic constraints <cite class="ltx_cite">(Haldane, <a href="#bib.bib21" title="The causes of evolution" class="ltx_ref">1932</a>; Orr, <a href="#bib.bib44" title="The probability of parallel evolution" class="ltx_ref">2005</a>)</cite>,
whether adaptation is mutation-limited <cite class="ltx_cite">(Bradshaw, <a href="#bib.bib48" title="The Croonian lecture, 1991: Genostasis and the limits to evolution" class="ltx_ref">1991</a>; Karasov<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib51" title="Evidence that adaptation in Drosophila is not limited by mutation at single sites" class="ltx_ref">2010</a>)</cite>,
and to <span class="ltx_text ltx_font_italic" style="color:#FF0000;">what degree species should be regarded</span> as cohesive units.
Convergent evolution also affects our ability to detect adaptation from population genomic data,
since no single allele sweeps to fixation over the entire area affected by the selection pressure <cite class="ltx_cite">(Pennings and Hermisson, <a href="#bib.bib115" title="Soft sweeps III: the signature of positive selection from recurrent mutation" class="ltx_ref">2006</a>)</cite>.</p>
    
</div>
    
<div id="S1.p3" class="ltx_para">
      
<p class="ltx_p">Convergent evolution can occur even within a well mixed population
subject to a constant selection pressure,
either through selection on multiple mutations present as standing variation within the population
before selection pressures switch <cite class="ltx_cite">(Orr and Betancourt, <a href="#bib.bib43" title="Haldane‚Äôs sieve and adaptation from the standing genetic variation" class="ltx_ref">2001</a>; Hermisson and Pennings, <a href="#bib.bib116" title="Soft sweeps: molecular population genetics of adaptation from standing genetic variation" class="ltx_ref">2005</a>)</cite>,
or due to multiple adaptive alleles that arise after selection pressures switch <cite class="ltx_cite">(Pennings and Hermisson, <a href="#bib.bib117" title="Soft sweeps II - molecular population genetics of adaptation from recurrent mutation or migration" class="ltx_ref">2006</a>)</cite>.
Previous work has shown that
a primary determinant of the probability that multiple alleles contribute to adaptation
is the product of the population size and the mutation rate <cite class="ltx_cite">(see Messer and Petrov, <a href="#bib.bib53" title="Population genomics of rapid adaptation by soft selective sweeps" class="ltx_ref">2013</a>, for a review)</cite>.</p>
    
</div>
    
<div id="S1.p4" class="ltx_para">
      
<p class="ltx_p">Spatial population structure,
as caused for example by geographically limited dispersal,
also increases the chance of convergent evolution.
For example,
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">geographically patchy</span> selection pressures can lead to much higher probability of parallel adaptation than uniform pressures,
since alleles are unable to spread through intervening populations
<cite class="ltx_cite">(Ralph and Coop, <a href="#bib.bib76" title="Convergent evolution during local adaptation to patchy landscapes" class="ltx_ref">2014</a>)</cite>.
In <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite> we formulated a simple model of <span class="ltx_text ltx_font_italic" style="color:#0000FF;">convergent adaptation in
a spatially spread population with local dispersal
that is exposed to some novel, spatially homogeneous selection pressure.
We assumed that a single mutational change was sufficient to adapt
the population after the change in environment. Under this assumption,
selected alleles arise and spread locally, as shown in Figure <a href="#S1.F1" title="Figure¬†1 ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
If the geographic area is large enough multiple selected alleles can
arise independently and spread before any one has spread across all of space.
A somewhat analogous situation also
arises in spatial models of clonal inference in asexuals <cite class="ltx_cite">(Gordo and Campos, <a href="#bib.bib3" title="Adaptive evolution in a spatially structured asexual population" class="ltx_ref">2006</a>; Martens and Hallatschek, <a href="#bib.bib5" title="Interfering waves of adaptation promote spatial mixing" class="ltx_ref">2011</a>; Otwinowski and Krug, <a href="#bib.bib2" title="Clonal interference and Muller‚Äôs ratchet in spatial habitats" class="ltx_ref">2014</a>)</cite>.</span></p>
    
</div>
    
<div id="S1.F1" class="ltx_figure">
      <img src="x9.png" id="S1.F1.g1" class="ltx_graphics ltx_centering" width="541" height="325" alt=""></img>
      
      
<div class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure¬†1: </span>
<span class="ltx_text ltx_font_bold">Simulation of the geographic spread of new mutations</span>. Simulation
on a grid of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F1.m3" class="ltx_Math" alttext="101\times 101" display="inline"><mrow><mn>101</mn><mo>√ó</mo><mn>101</mn></mrow></math> populations with 100 haploid individuals
in each, showing spread of a beneficial allele with advantage <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F1.m4" class="ltx_Math" alttext="s=0.1" display="inline"><mrow><mi>s</mi><mo>=</mo><mn>0.1</mn></mrow></math>.
Times are in numbers of generations, with probability of mutation per
generation 0.3. Different colours show the different independent
origins and spreads of selected alleles. Alleles quick establish and
start to spread, but because they are selective equivalent once they
spread into each other they only mix slowly (note the lack of change
from generation 600 onward). More details of the
simulation are given in <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite>.
</div>
    
</div>
    
<div id="S1.p5" class="ltx_para">
      
<p class="ltx_p"><span class="ltx_text ltx_font_italic" style="color:#0000FF;">Under this model we previously derived</span> the characteristic geographic scale over which
multiple instances of the adaptive allele are expected to arise in parallel,
a characteristic length <span class="ltx_text ltx_font_italic" style="color:#FF0000;">expressed in terms the parameters of interest.</span>
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">In <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite> we assumed that there was no standing variation for the adaptive
allele (e.g., because the allele was very strongly deleterious before the environmental change)</span>,
so that parallel mutation must be due to multiple new mutations occurring after the environmental shift.</p>
    
</div>
    
<div id="S1.p6" class="ltx_para">
      
<p class="ltx_p">In this paper, we extend this spatial model to include standing variation
present at mutation-selection balance before the selection pressures switch.
Below,
we show that convergent adaptation within a widespread species is likely to be
common <span class="ltx_text ltx_font_italic" style="color:#0000FF;">when ranges, population sizes, or mutational targets are large</span>, as has already been seen for a number of traits.
On this basis we argue
that the genetics of adaptation may often be geographically local <span class="ltx_text ltx_font_italic" style="color:#0000FF;">even
when selection pressures are geographically broad</span>,
and that widespread selective sweeps <span class="ltx_text ltx_font_italic" style="color:#FF0000;">should tend to occur only</span>
when adaptation is highly constrained (e.g.¬†by small mutation rate or
the need for a linked combination of alleles).
We discuss the <span class="ltx_text ltx_font_italic" style="color:#0000FF;">history, and implications,</span> of this view for the evolutionary
coherence of species and molecular evolution.</p>
    
</div>
    
    
<div id="S1.SS1" class="ltx_subsection">
      
<h3 class="ltx_title ltx_title_subsection"><span class="ltx_tag ltx_tag_subsection">1.1 </span>Model description</h3>

      
      
<div id="S1.SS1.p1" class="ltx_para">
        
<p class="ltx_p">We assume that the species range
is a large, homogeneous, one‚Äì or two‚Äìdimensional region.
There are two selective classes ‚Äì the <span class="ltx_text ltx_font_italic" style="color:#0000FF;">ancestral</span> type,
and the <span class="ltx_text ltx_font_italic">mutated</span> type <span class="ltx_text ltx_font_italic" style="color:#0000FF;">(which will offer a fitness benefit after the
environmental switch)</span>.
We assume that separately arising mutations are distinguishable ‚Äì
either as selectively equivalent mutations, or by linked <span class="ltx_text ltx_font_italic" style="color:#0000FF;">neutral</span> variation.
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">We assume that all alleles of the mutated type are selectively
equivalent (both in terms how deleterious they were before, and how
advantagous they are after, the environmental shift).
Note that these mutations do not have to arise at the same locus,
just that they are selectively equivalent:
these could be mutations arising at the same base pair,
or knockout mutations of any one of a number of genes in a pathway,
as long as carrying at least one of these alleles
is sufficient to adapt an individual to the new environment.</span>
</p>
      
</div>
      
<div id="S1.SS1.p2" class="ltx_para">
        
<p class="ltx_p">We also suppose that
the mutated type has been at a selective disadvantage
for a sufficiently long enough time in the past to be at selection-mutation equilibrium,
but at a certain time the selective regime changes, so that the mutated type has a selective advantage and quickly spreads to fixation.
After fixation, alleles are either descended
from families of mutants present as standing variation when the selective regime changed,
or from new mutants arising since that time.</p>
      
</div>
      
<div id="S1.SS1.p3" class="ltx_para">
        
<p class="ltx_p">For concreteness, suppose that before time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m1" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>,
the mutant type has fitness <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m2" class="ltx_Math" alttext="1-s_{d}" display="inline"><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow></math> relative to the neutral type
(i.e.¬†it produces on average <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m3" class="ltx_Math" alttext="1-s_{d}" display="inline"><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow></math> times the number of offspring per generation),
and that after time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m4" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>,
the mutant type has fitness <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m5" class="ltx_Math" alttext="1+s_{b}" display="inline"><mrow><mn>1</mn><mo>+</mo><msub><mi>s</mi><mi>b</mi></msub></mrow></math>,
where <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m6" class="ltx_Math" alttext="s_{b}&gt;0" display="inline"><mrow><msub><mi>s</mi><mi>b</mi></msub><mo>&gt;</mo><mn>0</mn></mrow></math> will usually be assumed to be small, and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m7" class="ltx_Math" alttext="0&lt;s_{d}&lt;1" display="inline"><mrow><mn>0</mn><mo>&lt;</mo><msub><mi>s</mi><mi>d</mi></msub><mo>&lt;</mo><mn>1</mn></mrow></math>.
We assume that diploid fitness is additive, or at least that the
important early dynamics are determined by the heterozygous fitness, with no reference to the fitness of the
homozygote for the mutant alleles. <span class="ltx_text ltx_font_italic" style="color:#FF0000;">Note that this means that we
are ignoring the case of totally recessive alleles.</span>
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">We assume that the variance in offspring number is one, see
<cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite> for how this assumption can be relaxed.</span>
As for the other parameters,
suppose that each offspring of a neutral parent is of the mutant type with probability <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m8" class="ltx_Math" alttext="\mu" display="inline"><mi>Œº</mi></math>,
and that the mean squared geographic distance between parent and child
is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m9" class="ltx_Math" alttext="\sigma^{2}" display="inline"><msup><mi>œÉ</mi><mn>2</mn></msup></math>. <span class="ltx_text ltx_font_italic" style="color:#FF0000;">We assume that the dispersal kernel is
not heavy tailed, i.e., falling into the Gaussian domain of attraction, see <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite> for
discussion of heavy tailed kernels and <cite class="ltx_cite">Hallatschek and Fisher (<a href="#bib.bib4" title="Acceleration of evolutionary spread by long-range dispersal" class="ltx_ref">2014</a>)</cite> for recent progress
on rapid spreading due to very long distance dispersal.</span>
The species occupies an area with mean density
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.p3.m10" class="ltx_Math" alttext="\rho" display="inline"><mi>œÅ</mi></math> of alleles per unit area <span class="ltx_text ltx_font_italic" style="color:#0000FF;">(twice the number of diploid
individuals per unit area)</span>.</p>
      
</div>
      
      
<div id="S1.SS1.SSS0.Px1" class="ltx_paragraph">
        
<h5 class="ltx_title ltx_title_paragraph">Rates of origination of standing and new mutations</h5>

        
<div id="S1.SS1.SSS0.Px1.p1" class="ltx_para">
          
<p class="ltx_p">We make use of the commonly used approximation that neglects competition between close relatives,
treating the offspring of a new mutant that appears in an area not already occupied by the mutated type
as a branching process <span class="ltx_text ltx_font_italic" style="color:#0000FF;">in continuous time, measured in generations
(average parental age at birth)</span>.
After <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p1.m1" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>, the offspring of each individual mutant thus forms (approximately) a branching process with growth rate <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p1.m2" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math>,
so each new mutant establishes locally with probability <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p1.m3" class="ltx_Math" alttext="p_{s}\approx 2s_{b}" display="inline"><mrow><msub><mi>p</mi><mi>s</mi></msub><mo>‚âà</mo><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow></mrow></math>.
As in <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite>, each new offspring has a very small probability of being a mutant and establishing locally,
so the collections of times and locations at which mutants appear and establish locally
is well‚Äìapproximated by a Poisson process in space and time.
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">By this we mean that there is a constant rate across space and
time at which new
mutations arise, and the occurance of new mutations in any region of
space-time is independent of that in non-overlapping intervals
in which the mutated type has not already appeared.</span>
The rate of this Poisson process,
i.e.¬†the mean number of new mutants per unit area and per generation that
appear and establish locally after <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p1.m4" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> in areas not already occupied by the mutant type,
<span class="ltx_text ltx_font_italic" style="color:#7303EB;">is the product of the haploid population density, denoted <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p1.m5" class="ltx_Math" alttext="\rho" display="inline"><mi mathcolor="#7303EB">œÅ</mi></math>,
the mutation rate (<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p1.m6" class="ltx_Math" alttext="\mu" display="inline"><mi mathcolor="#7303EB">Œº</mi></math>),
and the probability of fixation <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p1.m7" class="ltx_Math" alttext="p_{s}" display="inline"><msub><mi mathcolor="#7303EB">p</mi><mi mathcolor="#7303EB">s</mi></msub></math>, or</span>
approximately</p>
          
<table id="S1.E1" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E1.m1" class="ltx_Math" alttext="\lambda=2s_{b}\mu\rho" display="block"><mrow><mi>Œª</mi><mo>=</mo><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub><mo>‚Å¢</mo><mi>Œº</mi><mo>‚Å¢</mo><mi>œÅ</mi></mrow></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(1)</span></td></tr>
</table>
          
<p class="ltx_p"><span class="ltx_text ltx_font_italic" style="color:#0000FF;">In geographic areas already occupied by the mutant type,
new mutations are effectively neutral, and so unlikely to establish over the
short-time scales considered here, and are hence excluded. </span></p>
        
</div>
        
        
<div id="S1.SS1.SSS0.Px1.p2" class="ltx_para">
          
<p class="ltx_p">Before <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m1" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>, on the other hand, the allele is deleterious.
The genetic descendants of each new mutation are (with high probability) doomed to extinction,
but may persist for <span class="ltx_text ltx_font_italic" style="color:#FF0000;">a short</span> time (note that we have assumed that <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m2" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math> is not too small).
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">The</span> times and locations of new mutations before <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m3" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> also <span class="ltx_text ltx_font_italic" style="color:#0000FF;">will</span> be
well‚Äìapproximated by a
Poisson process with rate <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m4" class="ltx_Math" alttext="\mu\rho" display="inline"><mrow><mi>Œº</mi><mo>‚Å¢</mo><mi>œÅ</mi></mrow></math> <span class="ltx_text ltx_font_italic" style="color:#0000FF;">since mutations are rare events. Therefore,</span>
the locations of all mutant families extant at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m5" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> whose descendants are destined to fix locally is
a thinning of the original process,
and so, by the Poisson Mapping Theorem, is also a Poisson process.
<span class="ltx_text ltx_font_italic" style="color:#FF0000;">We define <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m6" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi mathcolor="#FF0000">Œª</mi><mn mathcolor="#FF0000" mathvariant="normal">0</mn></msub></math> to be the mean density of this process,
i.e., the geographic density
of standing variants that are present and escape loss when the
environment shifts.</span>
If we assume that the descendants of at most only a few members of any extant mutant family at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m7" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> will survive,
and that these progenitors are near to each other in space,
we can then treat each such family as equivalent to a single new mutation,
but with somewhat larger probability of local establishment,
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">following the environmental shift</span>.
(This approximation will be good
if the logarithm of the size of each extant family is small relative to the establishment time,
<span class="ltx_text ltx_font_italic" style="color:#7303EB;">since then new families quickly ‚Äúcatch up‚Äù to the size of already-extant families,</span>
and the spatial distribution of each is small relative to the spread between the families.)
To find <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m8" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi>Œª</mi><mn>0</mn></msub></math>, consider a mutation that arose <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m9" class="ltx_Math" alttext="T" display="inline"><mi mathcolor="#7303EB">T</mi></math><span class="ltx_text ltx_font_italic" style="color:#7303EB;"> generations ago</span>,
and let <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m10" class="ltx_Math" alttext="Z_{T}" display="inline"><msub><mi mathcolor="#0000FF">Z</mi><mi mathcolor="#0000FF">T</mi></msub></math><span class="ltx_text ltx_font_italic" style="color:#0000FF;"> be the number of its descendants at time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m11" class="ltx_Math" alttext="0" display="inline"><mn mathcolor="#0000FF" mathvariant="normal">0</mn></math></span>.
At time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m12" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>, when the environment shifts, there are <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m13" class="ltx_Math" alttext="Z_{T}" display="inline"><msub><mi>Z</mi><mi>T</mi></msub></math> individuals present with the mutation,
and each has probability <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m14" class="ltx_Math" alttext="p_{s}" display="inline"><msub><mi>p</mi><mi>s</mi></msub></math> of establishing, approximately independently.
Therefore, the probability that at least one descendant of this mutation establish and fix locally is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m15" class="ltx_Math" alttext="1-(1-p_{s})^{Z_{T}}" display="inline"><mrow><mn>1</mn><mo>-</mo><msup><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>s</mi></msub></mrow><mo stretchy="false">)</mo></mrow><msub><mi>Z</mi><mi>T</mi></msub></msup></mrow></math>,
the mean number of clusters of standing variants destined to fix locally, per unit area, is</p>
          
<table id="S2.EGx1" class="ltx_equationgroup ltx_eqn_align">

<tr id="S1.Ex1" class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_td ltx_align_right">
                    <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.Ex1.m1" class="ltx_Math" alttext="\displaystyle\mu\rho\int_{0}^{\infty}\mathbb{E}\left[1-(1-p_{s})^{Z_{T}}\right%
]dT." display="inline"><mrow><mrow><mi>Œº</mi><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><mrow><mstyle displaystyle="true"><msubsup><mo largeop="true" symmetric="true">‚à´</mo><mn>0</mn><mi mathvariant="normal">‚àû</mi></msubsup></mstyle><mrow><mi>ùîº</mi><mo>‚Å¢</mo><mrow><mo>[</mo><mrow><mn>1</mn><mo>-</mo><msup><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>p</mi><mi>s</mi></msub></mrow><mo stretchy="false">)</mo></mrow><msub><mi>Z</mi><mi>T</mi></msub></msup></mrow><mo>]</mo></mrow><mo>‚Å¢</mo><mrow><mo>ùëë</mo><mi>T</mi></mrow></mrow></mrow></mrow><mo>.</mo></mrow></math>
                  </td>
<td class="ltx_eqn_pad"></td></tr>
</table>
          
<p class="ltx_p">For <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m16" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> small, using <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m17" class="ltx_Math" alttext="p_{s}\approx 2s_{b}" display="inline"><mrow><msub><mi>p</mi><mi>s</mi></msub><mo>‚âà</mo><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow></mrow></math> and that <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m18" class="ltx_Math" alttext="\mathbb{E}[Z_{T}]=(1-s_{d})^{T}" display="inline"><mrow><mrow><mi>ùîº</mi><mo>‚Å¢</mo><mrow><mo stretchy="false">[</mo><msub><mi>Z</mi><mi>T</mi></msub><mo stretchy="false">]</mo></mrow></mrow><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mi>T</mi></msup></mrow></math>,
<span class="ltx_text ltx_font_italic" style="color:#7303EB;">we know that <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m19" class="ltx_Math" alttext="\mathbb{E}[(1-2s_{b})^{Z_{T}}]\approx 1-2s_{b}\mathbb{E}[Z_{T}]=1-2s_{b}(1-s_{%
d})^{T}" display="inline"><mrow><mrow><mi mathcolor="#7303EB">ùîº</mi><mo mathcolor="#7303EB" mathvariant="italic">‚Å¢</mo><mrow><mo mathcolor="#7303EB" mathvariant="normal" stretchy="false">[</mo><msup><mrow><mo mathcolor="#7303EB" mathvariant="normal" stretchy="false">(</mo><mrow><mn mathcolor="#7303EB" mathvariant="normal">1</mn><mo mathcolor="#7303EB" mathvariant="normal">-</mo><mrow><mn mathcolor="#7303EB" mathvariant="normal">2</mn><mo mathcolor="#7303EB" mathvariant="italic">‚Å¢</mo><msub><mi mathcolor="#7303EB">s</mi><mi mathcolor="#7303EB">b</mi></msub></mrow></mrow><mo mathcolor="#7303EB" mathvariant="normal" stretchy="false">)</mo></mrow><msub><mi mathcolor="#7303EB">Z</mi><mi mathcolor="#7303EB">T</mi></msub></msup><mo mathcolor="#7303EB" mathvariant="normal" stretchy="false">]</mo></mrow></mrow><mo mathcolor="#7303EB" mathvariant="normal">‚âà</mo><mrow><mn mathcolor="#7303EB" mathvariant="normal">1</mn><mo mathcolor="#7303EB" mathvariant="normal">-</mo><mrow><mn mathcolor="#7303EB" mathvariant="normal">2</mn><mo mathcolor="#7303EB" mathvariant="italic">‚Å¢</mo><msub><mi mathcolor="#7303EB">s</mi><mi mathcolor="#7303EB">b</mi></msub><mo mathcolor="#7303EB" mathvariant="italic">‚Å¢</mo><mi mathcolor="#7303EB">ùîº</mi><mo mathcolor="#7303EB" mathvariant="italic">‚Å¢</mo><mrow><mo mathcolor="#7303EB" mathvariant="normal" stretchy="false">[</mo><msub><mi mathcolor="#7303EB">Z</mi><mi mathcolor="#7303EB">T</mi></msub><mo mathcolor="#7303EB" mathvariant="normal" stretchy="false">]</mo></mrow></mrow></mrow><mo mathcolor="#7303EB" mathvariant="normal">=</mo><mrow><mn mathcolor="#7303EB" mathvariant="normal">1</mn><mo mathcolor="#7303EB" mathvariant="normal">-</mo><mrow><mn mathcolor="#7303EB" mathvariant="normal">2</mn><mo mathcolor="#7303EB" mathvariant="italic">‚Å¢</mo><msub><mi mathcolor="#7303EB">s</mi><mi mathcolor="#7303EB">b</mi></msub><mo mathcolor="#7303EB" mathvariant="italic">‚Å¢</mo><msup><mrow><mo mathcolor="#7303EB" mathvariant="normal" stretchy="false">(</mo><mrow><mn mathcolor="#7303EB" mathvariant="normal">1</mn><mo mathcolor="#7303EB" mathvariant="normal">-</mo><msub><mi mathcolor="#7303EB">s</mi><mi mathcolor="#7303EB">d</mi></msub></mrow><mo mathcolor="#7303EB" mathvariant="normal" stretchy="false">)</mo></mrow><mi mathcolor="#7303EB">T</mi></msup></mrow></mrow></mrow></math>,</span>
resulting in the approximation</p>
          
<table id="S2.EGx2" class="ltx_equationgroup ltx_eqn_align">

<tr id="S1.E2" class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_td ltx_align_right">
                    <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E2.m1" class="ltx_Math" alttext="\displaystyle\lambda_{0}" display="inline"><msub><mi>Œª</mi><mn>0</mn></msub></math>
                  </td>
<td class="ltx_td ltx_align_left">
                    <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E2.m2" class="ltx_Math" alttext="\displaystyle=\mu\rho\int_{0}^{\infty}2s_{b}(1-s_{d})^{T}dT" display="inline"><mrow><mi></mi><mo>=</mo><mrow><mi>Œº</mi><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><mrow><mstyle displaystyle="true"><msubsup><mo largeop="true" symmetric="true">‚à´</mo><mn>0</mn><mi mathvariant="normal">‚àû</mi></msubsup></mstyle><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub><mo>‚Å¢</mo><msup><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mi>T</mi></msup><mo>‚Å¢</mo><mrow><mo>ùëë</mo><mi>T</mi></mrow></mrow></mrow></mrow></mrow></math>
                  </td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(2)</span></td></tr>
<tr id="S1.E3" class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_td ltx_align_right"></td>
<td class="ltx_td ltx_align_left">
                    <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E3.m2" class="ltx_Math" alttext="\displaystyle=\frac{2\mu\rho s_{b}}{-\log(1-s_{d})}." display="inline"><mrow><mrow><mi></mi><mo>=</mo><mstyle displaystyle="true"><mfrac><mrow><mn>2</mn><mo>‚Å¢</mo><mi>Œº</mi><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow><mrow><mo>-</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mfrac></mstyle></mrow><mo>.</mo></mrow></math>
                  </td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(3)</span></td></tr>
<tr id="S1.E4" class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_td ltx_align_right"></td>
<td class="ltx_td ltx_align_left">
                    <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E4.m2" class="ltx_Math" alttext="\displaystyle=\frac{\lambda}{(-\log(1-s_{d}))}." display="inline"><mrow><mrow><mi></mi><mo>=</mo><mstyle displaystyle="true"><mfrac><mi>Œª</mi><mrow><mo stretchy="false">(</mo><mrow><mo>-</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow><mo stretchy="false">)</mo></mrow></mfrac></mstyle></mrow><mo>.</mo></mrow></math>
                  </td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(4)</span></td></tr>
</table>
          
<p class="ltx_p">Note that for small <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m20" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math> this can also be found by taking the expected frequency under mutation-selection
balance <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m21" class="ltx_Math" alttext="\mu/s_{d}" display="inline"><mrow><mi>Œº</mi><mo>/</mo><msub><mi>s</mi><mi>d</mi></msub></mrow></math> <cite class="ltx_cite">(Haldane, <a href="#bib.bib20" title="A mathematical theory of natural and artificial selection, part V: Selection and mutation" class="ltx_ref">1927</a>, <a href="#bib.bib19" title="The effect of variation of fitness" class="ltx_ref">1937</a>)</cite> multiplying it by the
population density to obtain the expected number of chromosomes per unit area
carrying the deleterious allele, with each of these having a
probability <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px1.p2.m22" class="ltx_Math" alttext="2s_{b}" display="inline"><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow></math> of escaping loss <cite class="ltx_cite">(an analogous approach to
that taken by  Orr and Betancourt, <a href="#bib.bib43" title="Haldane‚Äôs sieve and adaptation from the standing genetic variation" class="ltx_ref">2001</a>)</cite>
.</p>
        
</div>
      
</div>
      
<div id="S1.SS1.SSS0.Px2" class="ltx_paragraph">
        
<h5 class="ltx_title ltx_title_paragraph">Geographic spread of alleles</h5>

        
<div id="S1.SS1.SSS0.Px2.p1" class="ltx_para">
          
<p class="ltx_p">Once an allele has become locally established it can begin to spread across space.
We assume that the allele, once established,
quickly settles down to spread spatially as a traveling wave of constant speed.
The behavior of this wave of advance of a beneficial allele was first described by
<cite class="ltx_cite">Fisher (<a href="#bib.bib177" title="The wave of advance of advantageous genes" class="ltx_ref">1937</a>)</cite> and <cite class="ltx_cite">Kolmogorov, Petrovskii,  and Piscunov (<a href="#bib.bib77" title="A study of the equation of diffusion with increase in the quantity of matter, and its application to a biological problem" class="ltx_ref">1937</a>)</cite>.
Under reasonably general conditions, the speed of advance of this wave is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px2.p1.m1" class="ltx_Math" alttext="v=\sigma\sqrt{2s}" display="inline"><mrow><mi>v</mi><mo>=</mo><mrow><mi>œÉ</mi><mo>‚Å¢</mo><msqrt><mrow><mn>2</mn><mo>‚Å¢</mo><mi>s</mi></mrow></msqrt></mrow></mrow></math>.
See <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite> for a more thorough review of these travelling waves.
Note that the speed of the wave will vary
with details of the space that individuals migrate across
<cite class="ltx_cite">(e.g.¬†see  Slatkin, <a href="#bib.bib79" title="The rate of spread of an advantageous allele in a subdivided population" class="ltx_ref">1976</a>; Slatkin and Charlesworth, <a href="#bib.bib78" title="The spatial distribution of transient alleles in a subdivided population: a simulation study" class="ltx_ref">1978</a>,  for comparisons to migration on discrete grids)</cite>.</p>
        
</div>
      
</div>
      
<div id="S1.SS1.SSS0.Px3" class="ltx_paragraph">
        
<h5 class="ltx_title ltx_title_paragraph">Putting it together.</h5>

        
<div id="S1.SS1.SSS0.Px3.p1" class="ltx_para">
          
<p class="ltx_p">Now, we can put these ingredients together for a simple model of the geographic spread of alleles,
a cartoon example of which is shown in Figure <a href="#S1.F2" title="Figure¬†2 ‚Ä£ Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
Initially, when the selection pressures change at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px3.p1.m1" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>,
a set of standing variants can start to spread having escaped loss through drift.
The originating mutations of these variants are depicted by lightning bolts, and occur at a density <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px3.p1.m2" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi>Œª</mi><mn>0</mn></msub></math> across space.
They spread at velocity <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px3.p1.m3" class="ltx_Math" alttext="v" display="inline"><mi>v</mi></math>, carving out cones in space-time.
As these alleles proceed in their geographic spread, other new alleles can arise and become established in parallel,
whose origins are indicated by stars.
These new mutations arise and become established at rate <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px3.p1.m4" class="ltx_Math" alttext="\lambda" display="inline"><mi>Œª</mi></math>.</p>
        
</div>
        
        
<div id="S1.SS1.SSS0.Px3.p2" class="ltx_para">
          
<p class="ltx_p">As we outlined in <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite>, this model of
geographic convergent evolution, when <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px3.p2.m1" class="ltx_Math" alttext="\lambda_{0}=0" display="inline"><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>=</mo><mn>0</mn></mrow></math>, is analogous to a model of crystallization due to <cite class="ltx_cite">Kolmogorov (<a href="#bib.bib63" title="On the statistical theory of metal crystallization" class="ltx_ref">1937</a>)</cite>.
In this model, nucleation sites form at random at a constant rate in time and space
and initiate the radial growth of new crystals.
After their initial spread, the different orientations of crystals form a random tessellation of space,
whose properties have been studied by <cite class="ltx_cite">M√∏ller (<a href="#bib.bib64" title="Random Johnson-Mehl tessellations" class="ltx_ref">1992</a>, <a href="#bib.bib62" title="Generation of Johnson-Mehl crystals and comparative analysis of models for random nucleation" class="ltx_ref">1995</a>)</cite> and others <cite class="ltx_cite">(Bollob√°s and Riordan, <a href="#bib.bib57" title="Percolation on random Johnson-Mehl tessellations and related models" class="ltx_ref">2008</a>; Gilbert, <a href="#bib.bib61" title="Random subdivisions of space into crystals" class="ltx_ref">1962</a>)</cite>.
The generalized version of this process, for non-constant wave speeds
and inhomogeneous Poisson processes is known as the
Kolmogorov‚ÄìJohnson‚ÄìMehl‚ÄìAvrami tessellation <cite class="ltx_cite">(Fanfoni and Tomellini, <a href="#bib.bib60" title="The Johnson-Mehl-Avrami-Kolmogorov model: A brief review" class="ltx_ref">1998</a>)</cite>.
Our combined process with both standing variation and new mutation is
a special case of the KJMA tessellation,
where the spatial-temporally homogeneous Poisson origination process of new mutations,
is supplemented by a single pulse of origination points at time zero with spatial density <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px3.p2.m2" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi>Œª</mi><mn>0</mn></msub></math>.
(For the purposes of analogy, we could imagine that before time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS1.SSS0.Px3.p2.m3" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>
the temperature is high enough that nucleation sites appear but do not persist long.)</p>
        
</div>
        
<div id="S1.SS1.SSS0.Px3.p3" class="ltx_para">
          
<p class="ltx_p">If we ignore the effects of new mutation,
then everything about the process is relatively simple:
each point in space will be first reached by the wave whose origination point lies closest to it.
This random tessellation of space is known as a Poisson-Voronoi tessellation <cite class="ltx_cite">(M√∏ller, <a href="#bib.bib45" title="Lectures on random voronoi tessellations" class="ltx_ref">1994</a>)</cite>
(i.e.¬†the cells formed by assigning regions of space to the nearest point in a Poisson process).
The properties of this tessellation by alleles is
determined by the spatial locations of the initiation points,
which are sampled from a spatially homogeneous Poisson process,
independent of the rate of spatial spread.
Introducing new mutations cause some qualitative changes
beyond dependence on new parameters:
the cells formed by a Voronoi tessellation have straight sides,
but the introduction of new mutations cause these to curve
<cite class="ltx_cite">(because the radii of the colliding circles differ; see Figure 1 of  Ralph and Coop, <a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>,  for a graphical depiction of this point)</cite>.</p>
        
</div>
        
<div id="S1.F2" class="ltx_figure">
          <img src="x10.png" id="S1.F2.g1" class="ltx_graphics ltx_centering" width="677" height="523" alt=""></img>
          
          
<div class="ltx_caption"><span class="ltx_tag ltx_tag_figure">Figure¬†2: </span>
<span class="ltx_text ltx_font_bold">Cartoon space-time diagram of the geographic spread of standing and new mutations</span>.
Time runs down the page, and a single spatial dimension runs across the page.
The environment switches at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m15" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>,
following which adaptive alleles that successfully escape loss by drift
spread locally from both standing variants (lightning bolts) and new
mutations (stars). <span class="ltx_text ltx_font_italic" style="color:#0000FF;">(Note that the standing variants arose at times
prior to <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m16" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi mathcolor="#0000FF">t</mi><mo mathcolor="#0000FF" mathvariant="normal">=</mo><mn mathcolor="#0000FF" mathvariant="normal">0</mn></mrow></math>.)</span>
The mean spatial density of successful standing variants at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m17" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m18" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi>Œª</mi><mn>0</mn></msub></math>,
and the mean density in (space <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m19" class="ltx_Math" alttext="\times" display="inline"><mo>√ó</mo></math> time) of successful new mutations is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m20" class="ltx_Math" alttext="\lambda" display="inline"><mi>Œª</mi></math>.
These successful alleles spread spatially outwards at speed <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m21" class="ltx_Math" alttext="v" display="inline"><mi>v</mi></math>,
so the space-time profile of their spread forms a cone. Any mutation
that arises in an area already within a cone will not have a selective advantage.
When these adaptive alleles meet,
their rapid advance is halted because they no longer have an advantage,
so their boundaries form straight lines on short time-scales
(over longer time-scales, they will mix into each other).
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">One of the ways we derive various results in the text is by taking a
space-time point <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m22" class="ltx_Math" alttext="(x,t)" display="inline"><mrow><mo mathcolor="#0000FF" mathvariant="normal" stretchy="false">(</mo><mi mathcolor="#0000FF">x</mi><mo mathcolor="#0000FF" mathvariant="normal">,</mo><mi mathcolor="#0000FF">t</mi><mo mathcolor="#0000FF" mathvariant="normal" stretchy="false">)</mo></mrow></math> and asking the probability that no successful
alleles have spread there yet.</span> The grey dashed line shows a cone radiating backward in space-time,
originating at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m23" class="ltx_Math" alttext="(x,t)" display="inline"><mrow><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math>, with slope <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m24" class="ltx_Math" alttext="v" display="inline"><mi>v</mi></math>; its radius at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m25" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m26" class="ltx_Math" alttext="vt" display="inline"><mrow><mi>v</mi><mo>‚Å¢</mo><mi>t</mi></mrow></math>.
This shows the location <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m27" class="ltx_Math" alttext="x" display="inline"><mi>x</mi></math> is not yet adapted at time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F2.m28" class="ltx_Math" alttext="t" display="inline"><mi>t</mi></math> because no successful alleles appear within this grey cone.
</div>
        
</div>
        
      
</div>
    
</div>
    
<div id="S1.SS2" class="ltx_subsection">
      
<h3 class="ltx_title ltx_title_subsection"><span class="ltx_tag ltx_tag_subsection">1.2 </span>G6PD example</h3>

      
<div id="S1.SS2.p1" class="ltx_para">
        
<p class="ltx_p">Below we describe a number of properties of our process, but we will first introduce
a motivating example to provide some concrete
numbers to use for illustrative purposes.</p>
      
</div>
      
<div id="S1.SS2.p2" class="ltx_para">
        
<p class="ltx_p">Over roughly the past ten thousand years, alleles conferring resistance to malaria have arisen in a number of genes
and spread through human populations in areas where malaria <span class="ltx_text ltx_font_italic" style="color:#FF0000;">has been</span> endemic <cite class="ltx_cite">(Kwiatkowski, <a href="#bib.bib89" title="How malaria has affected the human genome and what human genetics can teach us about malaria" class="ltx_ref">2005</a>)</cite>.
A number of these alleles appear to be examples of convergent adaptation,
as different derived mutations in the same gene are seen in different individuals.
For example,
a number of changes that confer malaria resistance have been observed in the <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p2.m1" class="ltx_Math" alttext="\beta" display="inline"><mi>Œ≤</mi></math>-globin gene;
and the sickle cell allele may plausibly have arisen by up to five independent occurrences
of the same base pair mutation at different locations within Africa
<cite class="ltx_cite">(Flint<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib59" title="The population genetics of the haemoglobinopathies" class="ltx_ref">1998</a>; Ralph and Coop, <a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite>.
Another particularly impressive case of convergent evolution <span class="ltx_text ltx_font_italic" style="color:#FF0000;">is presented by</span> the numerous changes throughout the X-linked G6PD gene,
with upward of 50 polymorphic variants (above <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p2.m2" class="ltx_Math" alttext="1\%" display="inline"><mrow><mn>1</mn><mo lspace="0pt" rspace="3.5pt">%</mo></mrow></math> local frequency) having so far been described
that lower the activity of the enzyme <cite class="ltx_cite">(Howes<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib87" title="Spatial distribution of G6PD deficiency variants across malaria-endemic regions" class="ltx_ref">2013</a>; Minucci<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib88" title="Glucose-6-phosphate dehydrogenase (G6PD) mutations database: review of the ‚Äúold‚Äù and update of the new mutations" class="ltx_ref">2012</a>)</cite>.
These alleles are now found at a combined frequency of around 8% frequency in malaria endemic areas,
rarely exceeding 20% <cite class="ltx_cite">(Howes<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib90" title="G6PD deficiency prevalence and estimates of affected populations in malaria endemic countries: a geostatistical model-based map" class="ltx_ref">2012</a>)</cite>.
Whether these all confer resistance to malaria is unknown,
but malaria is thought to be the primary driver of these polymorphisms
<cite class="ltx_cite">(see Hedrick, <a href="#bib.bib13" title="Population genetics of malaria resistance in humans." class="ltx_ref">2011</a>,  for a general review)</cite>.
Three G6PD deficiency alleles are particularly common and relatively well studied:
the <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p2.m3" class="ltx_Math" alttext="A-" display="inline"><mrow><mi>A</mi><mo>-</mo></mrow></math> allele found in much of sub-Saharan Africa;
the Med allele found in the Mediterranean and Middle East;
and the Mahidol allele found Myanmar and Thailand.
<span class="ltx_text ltx_font_italic" style="color:#FF0000;">
The protective effects of these G6PD alleles are complicated, and are likely heterogeneous
across study populations and the form of malaria considered
<cite class="ltx_cite">(see  Manjurano<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib7" title="African glucose-6-phosphate dehydrogenase alleles associated with protection from severe malaria in heterozygous females in Tanzania" class="ltx_ref">2015</a>; Malaria Genomic Epidemiology Network, <a href="#bib.bib6" title="Reappraisal of known malaria resistance loci in a large multicenter study" class="ltx_ref">2014</a>,  for recent discussion)</cite>.
The <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p2.m4" class="ltx_Math" alttext="A-" display="inline"><mrow><mi mathcolor="#FF0000">A</mi><mo mathcolor="#FF0000" mathvariant="normal">-</mo></mrow></math> and Mahidol alleles are thought to offer some protective effects against Plasmodium falciparum and P.¬†vivax in both
heterozygote females and/or hemizygote males <cite class="ltx_cite">(Ruwende<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib95" title="Natural selection of hemi- and heterozygotes for G6PD deficiency in Africa by resistance to severe malaria" class="ltx_ref">1995</a>; Louicharoen<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib94" title="Positively selected G6PD-Mahidol mutation reduces Plasmodium vivax density in Southeast Asians" class="ltx_ref">2009</a>; Manjurano<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib7" title="African glucose-6-phosphate dehydrogenase alleles associated with protection from severe malaria in heterozygous females in Tanzania" class="ltx_ref">2015</a>)</cite>. </span>
Haplotype-based analysis of genetic diversity surrounding
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p2.m5" class="ltx_Math" alttext="A-" display="inline"><mrow><mi>A</mi><mo>-</mo></mrow></math>, Med, and Mahidol suggest that they have spread over the past
few thousand years <cite class="ltx_cite">(Tishkoff<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib98" title="Haplotype diversity and linkage disequilibrium at human G6PD: recent origin of alleles that confer malarial resistance" class="ltx_ref">2001</a>; Slatkin, <a href="#bib.bib97" title="A Bayesian method for jointly estimating allele age and selection intensity" class="ltx_ref">2008</a>; Saunders<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib96" title="The extent of linkage disequilibrium caused by selection on G6PD in humans" class="ltx_ref">2005</a>; Louicharoen<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib94" title="Positively selected G6PD-Mahidol mutation reduces Plasmodium vivax density in Southeast Asians" class="ltx_ref">2009</a>)</cite>,
consistent with the age of other known malaria resistance alleles.
Population genetic analyses suggest that these three variants each have a hemizygote/heterozygote
selection coefficient of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p2.m6" class="ltx_Math" alttext="0.05-0.3" display="inline"><mrow><mn>0.05</mn><mo>-</mo><mn>0.3</mn></mrow></math>
<cite class="ltx_cite">(Tishkoff<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib98" title="Haplotype diversity and linkage disequilibrium at human G6PD: recent origin of alleles that confer malarial resistance" class="ltx_ref">2001</a>; Slatkin, <a href="#bib.bib97" title="A Bayesian method for jointly estimating allele age and selection intensity" class="ltx_ref">2008</a>; Saunders<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib96" title="The extent of linkage disequilibrium caused by selection on G6PD in humans" class="ltx_ref">2005</a>; Louicharoen<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib94" title="Positively selected G6PD-Mahidol mutation reduces Plasmodium vivax density in Southeast Asians" class="ltx_ref">2009</a>)</cite>.
This is in reasonable agreement with the selection coefficients calculated by
<cite class="ltx_cite">Ruwende<span class="ltx_text ltx_bib_etal"> et al.</span> (<a href="#bib.bib95" title="Natural selection of hemi- and heterozygotes for G6PD deficiency in Africa by resistance to severe malaria" class="ltx_ref">1995</a>)</cite> on the basis of the present day levels of
resistance to malaria due to the <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p2.m7" class="ltx_Math" alttext="A-" display="inline"><mrow><mi>A</mi><mo>-</mo></mrow></math> allele. 
<br class="ltx_break"></br></p>
      
</div>
      
      
<div id="S1.SS2.p3" class="ltx_para">
        
<p class="ltx_p">Given <span class="ltx_text ltx_font_italic" style="color:#FF0000;">such strong selection the alleles</span> should have risen
quickly to fixation, so their presence at intermediate frequency,
over a broad geographic area, makes it a good candidate for a recently
balanced polymorphism due to heterozygote advantage <cite class="ltx_cite">(note that
the conditions for a balanced polymorphism are
complicated by the hemizygosity of males, see Hedrick, <a href="#bib.bib13" title="Population genetics of malaria resistance in humans." class="ltx_ref">2011</a>; Pamillo, <a href="#bib.bib12" title="Genic variation at sex-linked loci: quantification of regular selection models" class="ltx_ref">1979</a>)</cite>.
Indeed, hemizygous males and homozygous females suffer from G6PD deficiency,
and homozygote females may also not be protected against malaria <cite class="ltx_cite">(Manjurano<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib7" title="African glucose-6-phosphate dehydrogenase alleles associated with protection from severe malaria in heterozygous females in Tanzania" class="ltx_ref">2015</a>; Malaria Genomic Epidemiology Network, <a href="#bib.bib6" title="Reappraisal of known malaria resistance loci in a large multicenter study" class="ltx_ref">2014</a>)</cite>.
The theory we use regarding the ‚Äúwave of advance‚Äù <cite class="ltx_cite">(Fisher, <a href="#bib.bib177" title="The wave of advance of advantageous genes" class="ltx_ref">1937</a>)</cite>
applies as well in the case of heterozygote advantage
<cite class="ltx_cite">(Aronson and Weinberger, <a href="#bib.bib58" title="Nonlinear diffusion in population genetics, combustion, and nerve pulse propagation" class="ltx_ref">1975</a>)</cite>, with the selected allele spreading
locally to the equilibrium frequency (rather than fixation).
Therefore, our framework is applicable to the spread of G6PD, with speed determined by the advantage of heterozygotes when rare.
We assume that before malaria became prevalent,
G6PD deficiency alleles suffered a decrease in relative fitness of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p3.m1" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math> in heterozygote and homozygotes females and hemizygote males.
Assuming that the underlying causes and strength of this drop in fitness have not changed,
we estimate that <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p3.m2" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math> has to have been upward of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p3.m3" class="ltx_Math" alttext="\sim 0.05" display="inline"><mrow><mi></mi><mo>‚àº</mo><mn>0.05</mn></mrow></math> (if <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p3.m4" class="ltx_Math" alttext="s_{b}\geq 0.05" display="inline"><mrow><msub><mi>s</mi><mi>b</mi></msub><mo>‚â•</mo><mn>0.05</mn></mrow></math>),
in order to have resulted in the equilibrium frequency seen today in
areas with endemic malaria <cite class="ltx_cite">(based on heterozygote advantage
calculations for the X chromosome, results not shown, see also Ruwende<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib95" title="Natural selection of hemi- and heterozygotes for G6PD deficiency in Africa by resistance to severe malaria" class="ltx_ref">1995</a>)</cite>.</p>
      
</div>
      
      
<div id="S1.SS2.p4" class="ltx_para">
        
<p class="ltx_p">The geographic area of Central and Eastern Asia with malaria is
on the order of ten million square kilometers.
In that area there are at least <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m1" class="ltx_Math" alttext="15" display="inline"><mn>15</mn></math> common, <span class="ltx_text ltx_font_italic" style="color:#7303EB;">clinically relevant</span> variants <cite class="ltx_cite">(see Figure <a href="#S1.F3" title="Figure¬†3 ‚Ä£ 1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>, from Howes<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib87" title="Spatial distribution of G6PD deficiency variants across malaria-endemic regions" class="ltx_ref">2013</a>)</cite>.
<span class="ltx_text ltx_font_italic" style="color:#7303EB;">(These are type 2 variants that express at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m2" class="ltx_Math" alttext="{}&lt;50\%" display="inline"><mrow><mi></mi><mo mathcolor="#7303EB" mathvariant="normal">&lt;</mo><mrow><mn mathcolor="#7303EB" mathvariant="normal">50</mn><mo lspace="0pt" mathcolor="#7303EB" mathvariant="normal" rspace="3.5pt">%</mo></mrow></mrow></math> enzyme activity,
predispose individuals to haemolytic anaemia,
and are found in at least 10 localities;
see <cite class="ltx_cite">Howes<span class="ltx_text ltx_bib_etal"> et al.</span> (<a href="#bib.bib87" title="Spatial distribution of G6PD deficiency variants across malaria-endemic regions" class="ltx_ref">2013</a>)</cite> for more details.)</span>
Therefore, the average width of an area occupied by an allele is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m3" class="ltx_Math" alttext="\sqrt{10^{7}/15}\approx 800" display="inline"><mrow><msqrt><mrow><msup><mn>10</mn><mn>7</mn></msup><mo>/</mo><mn>15</mn></mrow></msqrt><mo>‚âà</mo><mn>800</mn></mrow></math>km.
The coding region of G6PD is 515 codons long,
and around 140 distinct deficiency alleles have been observed.
Assuming a mutation rate of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m4" class="ltx_Math" alttext="\approx 10^{-8}" display="inline"><mrow><mi></mi><mo>‚âà</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>8</mn></mrow></msup></mrow></math> per base pair per generation,
we can take as an order-of-magnitude estimate <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m5" class="ltx_Math" alttext="\mu\approx 10^{-6}" display="inline"><mrow><mi>Œº</mi><mo>‚âà</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>6</mn></mrow></msup></mrow></math> per generation.
The dispersal and demographic parameters of humans in the past few thousand years is unclear,
particularly as we are concerned with the ‚Äúeffective‚Äù population density
(i.e.¬†population density divided by variance in offspring number).
We therefore will use two reasonable values for the effective population density: <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m6" class="ltx_Math" alttext="\rho=2" display="inline"><mrow><mi>œÅ</mi><mo>=</mo><mn>2</mn></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m7" class="ltx_Math" alttext="0.2" display="inline"><mn>0.2</mn></math> people per km<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m8" class="ltx_Math" alttext="{}^{2}" display="inline"><msup><mi></mi><mn>2</mn></msup></math>,
and three values for the dispersal distance: <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m9" class="ltx_Math" alttext="\sigma=10" display="inline"><mrow><mi>œÉ</mi><mo>=</mo><mn>10</mn></mrow></math>, <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m10" class="ltx_Math" alttext="50" display="inline"><mn>50</mn></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS2.p4.m11" class="ltx_Math" alttext="100" display="inline"><mn>100</mn></math> kilometers per generation.
Clearly, human migration has been shaped both by local dispersal and larger-scale expansions
<cite class="ltx_cite">(see Pickrell and Reich, <a href="#bib.bib14" title="Toward a new history and geography of human genes informed by ancient DNA" class="ltx_ref">2014</a>, for a recent discussion)</cite>,
so these parameters only provide a rough view of the process.</p>
      
</div>
      
<div id="S1.F3" class="ltx_figure">
        <img src="x11.png" id="S1.F3.g1" class="ltx_graphics ltx_centering" width="541" height="244" alt=""></img>
        
        
        
<div class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure¬†3: </span>
<span class="ltx_text ltx_font_bold">Map of G6PD-deficiency allele frequencies across Asia.</span>
The pie chart shows the frequency of G6PD-deficiency alleles.
The size of the pie chart indicates the number of G6PD-deficient individuals sampled.
Countries with endemic malaria are colored yellow.
Figure taken from <cite class="ltx_cite">Howes<span class="ltx_text ltx_bib_etal"> et al.</span> (<a href="#bib.bib87" title="Spatial distribution of G6PD deficiency variants across malaria-endemic regions" class="ltx_ref">2013</a>)</cite>
<a href="http://www.malariajournal.com/content/12/1/418" title="" class="ltx_ref ltx_url"><span class="ltx_text ltx_font_typewriter">http://www.malariajournal.com/content/12/1/418</span></a>.
</div>
      
</div>
      
    
</div>
    
<div id="S1.SS3" class="ltx_subsection">
      
<h3 class="ltx_title ltx_title_subsection"><span class="ltx_tag ltx_tag_subsection">1.3 </span>The geographic resolution of adaptation from new and standing variation</h3>

      
      
<div id="S1.SS3.p1" class="ltx_para">
        
<p class="ltx_p">In <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite>, studying the model without standing variation,
we defined a <span class="ltx_text ltx_font_italic">characteristic length</span> which gave the spatial scale across which mutants with distinct origins would establish.
This was proportional to the mean distance between neighboring established mutants,
but had the advantage of being easier to calculate.
Furthermore, the time scale over which adaptation occurred could be found by dividing the characteristic length
by the speed at which the mutants spread.
We first define a similar characteristic length for this new model. </p>
      
</div>
      
<div id="S1.SS3.p2" class="ltx_para">
        
<p class="ltx_p">Suppose we fix our attention on a particular new mutation that happens to be the first to occur in some region.
If it does not encounter other locally fixed, beneficial mutants,
it will cover a distance <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m1" class="ltx_Math" alttext="L" display="inline"><mi>L</mi></math> in time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m2" class="ltx_Math" alttext="L/v" display="inline"><mrow><mi>L</mi><mo>/</mo><mi>v</mi></mrow></math>. <span class="ltx_text ltx_font_italic" style="color:#0000FF;">In doing so (in two-dimensions) it will have
started from a point in space and spread out to cover a circular geographic region of area <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m3" class="ltx_Math" alttext="\pi L^{2}" display="inline"><mrow><mi mathcolor="#0000FF">œÄ</mi><mo mathcolor="#0000FF" mathvariant="italic">‚Å¢</mo><msup><mi mathcolor="#0000FF">L</mi><mn mathcolor="#0000FF" mathvariant="normal">2</mn></msup></mrow></math>.
The cylinder in space-time with this circle as a base and
height <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m4" class="ltx_Math" alttext="L/v" display="inline"><mrow><mi mathcolor="#0000FF">L</mi><mo mathcolor="#0000FF" mathvariant="normal">/</mo><mi mathcolor="#0000FF">v</mi></mrow></math> has volume of this cylinder is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m5" class="ltx_Math" alttext="\pi L^{2}\times(L/v)" display="inline"><mrow><mrow><mi mathcolor="#0000FF">œÄ</mi><mo mathcolor="#0000FF" mathvariant="italic">‚Å¢</mo><msup><mi mathcolor="#0000FF">L</mi><mn mathcolor="#0000FF" mathvariant="normal">2</mn></msup></mrow><mo mathcolor="#0000FF" mathvariant="normal">√ó</mo><mrow><mo mathcolor="#0000FF" mathvariant="normal" stretchy="false">(</mo><mrow><mi mathcolor="#0000FF">L</mi><mo mathcolor="#0000FF" mathvariant="normal">/</mo><mi mathcolor="#0000FF">v</mi></mrow><mo mathcolor="#0000FF" mathvariant="normal" stretchy="false">)</mo></mrow></mrow></math>.
Therefore, the number of other successfully established mutations that
would have appeared</span> in the circle it has covered up until this time is Poisson with mean
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m6" class="ltx_Math" alttext="\lambda_{0}\pi L^{2}+\lambda\pi L^{3}/v" display="inline"><mrow><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>L</mi><mn>2</mn></msup></mrow><mo>+</mo><mrow><mrow><mi>Œª</mi><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>L</mi><mn>3</mn></msup></mrow><mo>/</mo><mi>v</mi></mrow></mrow></math> in two dimensions
(and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m7" class="ltx_Math" alttext="2\lambda_{0}L+2\lambda L^{2}/v" display="inline"><mrow><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>Œª</mi><mn>0</mn></msub><mo>‚Å¢</mo><mi>L</mi></mrow><mo>+</mo><mrow><mrow><mn>2</mn><mo>‚Å¢</mo><mi>Œª</mi><mo>‚Å¢</mo><msup><mi>L</mi><mn>2</mn></msup></mrow><mo>/</mo><mi>v</mi></mrow></mrow></math> in one dimension).
Therefore, if we define <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m8" class="ltx_Math" alttext="\chi" display="inline"><mi>œá</mi></math> for a two-dimensional model to be the unique positive solution to</p>
        
<table id="S1.E5" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E5.m1" class="ltx_Math" alttext="\lambda_{0}\pi\chi^{2}+\lambda\pi\chi^{3}/v=1," display="block"><mrow><mrow><mrow><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>œá</mi><mn>2</mn></msup></mrow><mo>+</mo><mrow><mrow><mi>Œª</mi><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>œá</mi><mn>3</mn></msup></mrow><mo>/</mo><mi>v</mi></mrow></mrow><mo>=</mo><mn>1</mn></mrow><mo>,</mo></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(5)</span></td></tr>
</table>
        
<p class="ltx_p">then <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m9" class="ltx_Math" alttext="\chi" display="inline"><mi>œá</mi></math> gives the distance spread unobstructed by the descendants of a new mutant
before it is expected that one other successful mutation would have arisen in the area covered so far.
The explicit formula for <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m10" class="ltx_Math" alttext="\chi" display="inline"><mi>œá</mi></math> in two dimensions <span class="ltx_text ltx_font_italic" style="color:#0000FF;">can be found by
rearranging eqn. (<a href="#S1.E5" title="(5) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>) but is cumbersome;</span> here we omit
it. <span class="ltx_text ltx_font_italic" style="color:#0000FF;">In one dimension things are a little prettier</span> and the characteristic length is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m11" class="ltx_Math" alttext="\chi=(\sqrt{\lambda_{0}^{2}+2\lambda/v}-\lambda_{0})/(2\lambda/v)" display="inline"><mrow><mi>œá</mi><mo>=</mo><mrow><mrow><mo stretchy="false">(</mo><mrow><msqrt><mrow><msubsup><mi>Œª</mi><mn>0</mn><mn>2</mn></msubsup><mo>+</mo><mrow><mrow><mn>2</mn><mo>‚Å¢</mo><mi>Œª</mi></mrow><mo>/</mo><mi>v</mi></mrow></mrow></msqrt><mo>-</mo><msub><mi>Œª</mi><mn>0</mn></msub></mrow><mo stretchy="false">)</mo></mrow><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><mrow><mn>2</mn><mo>‚Å¢</mo><mi>Œª</mi></mrow><mo>/</mo><mi>v</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math>.
Substituting expressions for <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m12" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi>Œª</mi><mn>0</mn></msub></math>, <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m13" class="ltx_Math" alttext="\lambda" display="inline"><mi>Œª</mi></math>, and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m14" class="ltx_Math" alttext="v" display="inline"><mi>v</mi></math> from above,
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">in two-dimensions</span> we can rewrite eqn. (<a href="#S1.E5" title="(5) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>) as</p>
        
<table id="S1.E6" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E6.m1" class="ltx_Math" alttext="\frac{\sqrt{2s_{b}}}{-\log(1-s_{d})}\chi^{2}+\frac{1}{\sigma}\chi^{3}=\frac{1}%
{\sqrt{2s_{b}}\rho\mu\pi}" display="block"><mrow><mrow><mrow><mfrac><msqrt><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow></msqrt><mrow><mo>-</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mfrac><mo>‚Å¢</mo><msup><mi>œá</mi><mn>2</mn></msup></mrow><mo>+</mo><mrow><mfrac><mn>1</mn><mi>œÉ</mi></mfrac><mo>‚Å¢</mo><msup><mi>œá</mi><mn>3</mn></msup></mrow></mrow><mo>=</mo><mfrac><mn>1</mn><mrow><msqrt><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow></msqrt><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><mi>Œº</mi><mo>‚Å¢</mo><mi>œÄ</mi></mrow></mfrac></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(6)</span></td></tr>
</table>
        
<p class="ltx_p">From this we see that <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m15" class="ltx_Math" alttext="\chi" display="inline"><mi>œá</mi></math> decreases with <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m16" class="ltx_Math" alttext="\rho" display="inline"><mi>œÅ</mi></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m17" class="ltx_Math" alttext="\mu" display="inline"><mi>Œº</mi></math>.
Furthermore, for large <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m18" class="ltx_Math" alttext="\sigma" display="inline"><mi>œÉ</mi></math>, the characteristic length <span class="ltx_text ltx_font_italic" style="color:#0000FF;">approaches
the value we would obtain just</span> from standing variation:</p>
        
<table id="S1.E7" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E7.m1" class="ltx_Math" alttext="\chi=\sqrt{-\log(1-s_{d})/2s_{b}\rho\mu\pi}+O(1/\sigma)." display="block"><mrow><mrow><mi>œá</mi><mo>=</mo><mrow><msqrt><mrow><mo>-</mo><mrow><mrow><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>/</mo><mn>2</mn></mrow><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><mi>Œº</mi><mo>‚Å¢</mo><mi>œÄ</mi></mrow></mrow></msqrt><mo>+</mo><mrow><mi>O</mi><mo>‚Å¢</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>/</mo><mi>œÉ</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow><mo>.</mo></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(7)</span></td></tr>
</table>
        
<p class="ltx_p">On the other hand, if the mutant allele is highly deleterious before <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m19" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>,
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">then standing variation is</span> unimportant the characteristic length is <span class="ltx_text ltx_font_italic" style="color:#0000FF;">approaches</span> the value from <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite>:</p>
        
<table id="S1.E8" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E8.m1" class="ltx_Math" alttext="\chi=(\sigma/\rho\mu\pi\sqrt{2s_{b}})^{1/3}+O(1/\log(1-s_{d}))." display="block"><mrow><mrow><mi>œá</mi><mo>=</mo><mrow><msup><mrow><mo stretchy="false">(</mo><mrow><mrow><mi>œÉ</mi><mo>/</mo><mi>œÅ</mi></mrow><mo>‚Å¢</mo><mi>Œº</mi><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msqrt><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow></msqrt></mrow><mo stretchy="false">)</mo></mrow><mrow><mn>1</mn><mo>/</mo><mn>3</mn></mrow></msup><mo>+</mo><mrow><mi>O</mi><mo>‚Å¢</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>/</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow><mo>.</mo></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(8)</span></td></tr>
</table>
        
<p class="ltx_p">These two end points help build our intuition for the interaction of
parameters in shaping the geographic scale of convergent evolution.
By the above calculation, we know that the relevant mutations occur about distance <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m20" class="ltx_Math" alttext="\chi" display="inline"><mi>œá</mi></math> apart,
and occur within the first <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m21" class="ltx_Math" alttext="\chi/v" display="inline"><mrow><mi>œá</mi><mo>/</mo><mi>v</mi></mrow></math> generations.
Said another way, if we look in a circular region of space of radius <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m22" class="ltx_Math" alttext="\chi" display="inline"><mi>œá</mi></math> over <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p2.m23" class="ltx_Math" alttext="\chi/v" display="inline"><mrow><mi>œá</mi><mo>/</mo><mi>v</mi></mrow></math> generations,
we expect to find roughly one mutational origin.
</p>
      
</div>
      
      
<div id="S1.SS3.p3" class="ltx_para">
        
<p class="ltx_p">In Figures <a href="#S1.F4" title="Figure¬†4 ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">4</span></a> and <a href="#S1.F5" title="Figure¬†5 ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">5</span></a>, we show the range of characteristic lengths as a function
of various parameters chosen to match the evolution of malaria
resistance at G6PD.
These curves match our intuition that <span class="ltx_text ltx_font_italic" style="color:#0000FF;">higher</span> population densities result in
smaller characteristic lengths (as would higher mutation rates).
Allowing standing variation increases <span class="ltx_text ltx_font_italic" style="color:#0000FF;">the
input of new alleles</span> and so decreases the characteristic length below that predicted
by only new mutations (i.e.¬†equation (<a href="#S1.E8" title="(8) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>), <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite>).
In turn, increasing the prior deleterious effect of the allele (<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m1" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math>)
acts to increase the
characteristic length until it reaches that predicted by new mutation alone.
Higher dispersal distances lead to larger characteristic lengths,
since more rapid geographic spread block other mutations from establishing.
A larger selective advantage (<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m2" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math>) acts in two conflicting
ways: aiding the rapid geographic spread of established alleles, and also helping more independent copies
<span class="ltx_text ltx_font_italic" style="color:#FF0000;">to</span> escape drift and become established.
The effect of helping<span class="ltx_text ltx_font_italic" style="color:#0000FF;">locally</span> establish alleles wins out, since increasing the selective benefit <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m3" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> decreases the characteristic length.
(This can be shown in general by differentiating (<a href="#S1.E6" title="(6) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a>) with respect to <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m4" class="ltx_Math" alttext="z=\sqrt{s_{b}}" display="inline"><mrow><mi>z</mi><mo>=</mo><msqrt><msub><mi>s</mi><mi>b</mi></msub></msqrt></mrow></math>,
showing that <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m5" class="ltx_Math" alttext="\partial_{z}\chi=-(C_{1}z\chi+C_{2}\chi^{2})/(C_{3}z^{2}+C_{4}z\chi)\leq 0" display="inline"><mrow><mrow><msub><mo>‚àÇ</mo><mi>z</mi></msub><mo>‚Å°</mo><mi>œá</mi></mrow><mo>=</mo><mrow><mo>-</mo><mrow><mrow><mo stretchy="false">(</mo><mrow><mrow><msub><mi>C</mi><mn>1</mn></msub><mo>‚Å¢</mo><mi>z</mi><mo>‚Å¢</mo><mi>œá</mi></mrow><mo>+</mo><mrow><msub><mi>C</mi><mn>2</mn></msub><mo>‚Å¢</mo><msup><mi>œá</mi><mn>2</mn></msup></mrow></mrow><mo stretchy="false">)</mo></mrow><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><mrow><msub><mi>C</mi><mn>3</mn></msub><mo>‚Å¢</mo><msup><mi>z</mi><mn>2</mn></msup></mrow><mo>+</mo><mrow><msub><mi>C</mi><mn>4</mn></msub><mo>‚Å¢</mo><mi>z</mi><mo>‚Å¢</mo><mi>œá</mi></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow><mo>‚â§</mo><mn>0</mn></mrow></math> for appropriate nonnegative constants <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m6" class="ltx_Math" alttext="C_{1-4}" display="inline"><msub><mi>C</mi><mrow><mn>1</mn><mo>-</mo><mn>4</mn></mrow></msub></math>.)
This effect is strongest when only standing variation contributes (<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m7" class="ltx_Math" alttext="s_{b}^{-1/2}" display="inline"><msubsup><mi>s</mi><mi>b</mi><mrow><mo>-</mo><mrow><mn>1</mn><mo>/</mo><mn>2</mn></mrow></mrow></msubsup></math>, equation (<a href="#S1.E7" title="(7) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>)),
as in that case the speed of spread does not matter <span class="ltx_text ltx_font_italic" style="color:#FF0000;">only the
initial density of established alleles after the environmental shift</span>.
The dependence of the characteristic length on <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m8" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> when only new
mutations contribute is much weaker (<span class="ltx_text ltx_font_italic" style="color:#0000FF;">of order</span> <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m9" class="ltx_Math" alttext="s_{b}^{-1/6}" display="inline"><msubsup><mi>s</mi><mi>b</mi><mrow><mo>-</mo><mrow><mn>1</mn><mo>/</mo><mn>6</mn></mrow></mrow></msubsup></math>, equation (<a href="#S1.E8" title="(8) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>))
Overall, the range of characteristic lengths observed are reasonably
consistent with the average diameter of a G6PD variant in Eurasia of
800km, especially for the lower population density, as long as the
fitness cost of G6PD-deficiency alleles before malaria (<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p3.m10" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math>) was <span class="ltx_text ltx_font_italic" style="color:#FF0000;">high</span>.</p>
      
</div>
      
      
<div id="S1.SS3.p4" class="ltx_para">
        
<p class="ltx_p">Note that the conflicting roles of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p4.m1" class="ltx_Math" alttext="\rho" display="inline"><mi>œÅ</mi></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS3.p4.m2" class="ltx_Math" alttext="\sigma" display="inline"><mi>œÉ</mi></math> mean that even in
species where levels of neutral differentiation are low, geographic
convergent adaptation may be common. This is because low levels of
neutral genetic differentiation between geographic regions can be due to high population
densities rather than high dispersal distances, and high population
densities would allow convergent adaptation. As such geographic convergent
evolution may be common even in species with little neutral population structure.</p>
      
</div>
      
<div id="S1.F4" class="ltx_figure">
        <img src="x12.png" id="S1.F4.g1" class="ltx_graphics ltx_centering" width="677" height="323" alt=""></img>
        
        
        
<div class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure¬†4: </span> <span class="ltx_text ltx_font_bold">Characteristic length, in kilometers, as a function of selective disadvantage</span>,
compared to the corresponding quantity without standing variation (equation (<a href="#S1.E8" title="(8) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>))
and to the quantity only considering standing variants (equation (<a href="#S1.E7" title="(7) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>)).
The other parameters are chosen to match those of G6PD,
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F4.m3" class="ltx_Math" alttext="\mu=10^{-6}" display="inline"><mrow><mi mathcolor="#0000FF">Œº</mi><mo mathcolor="#0000FF">=</mo><msup><mn mathcolor="#0000FF">10</mn><mrow><mo mathcolor="#0000FF">-</mo><mn mathcolor="#0000FF">6</mn></mrow></msup></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F4.m4" class="ltx_Math" alttext="s_{b}=0.05" display="inline"><mrow><msub><mi>s</mi><mi>b</mi></msub><mo>=</mo><mn>0.05</mn></mrow></math>. As the characteristic length with
only standing variation is independent of dispersal distance it is
plotted as a single black dotted line.
</div>
      
</div>
      
<div id="S1.F5" class="ltx_figure">
        
        <img src="x13.png" id="S1.F5.g1" class="ltx_graphics ltx_centering" width="677" height="323" alt=""></img>
        
        
        
<div class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure¬†5: </span> <span class="ltx_text ltx_font_bold">Characteristic length</span>, in kilometers, as a function of selective advantage, at two population densities,
holding the prior disadvantage of the allele fixed at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F5.m3" class="ltx_Math" alttext="s_{d}=0.05" display="inline"><mrow><msub><mi>s</mi><mi>d</mi></msub><mo>=</mo><mn>0.05</mn></mrow></math> and the mutation rate fixed at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F5.m4" class="ltx_Math" alttext="\mu=10^{-6}" display="inline"><mrow><mi mathcolor="#0000FF">Œº</mi><mo mathcolor="#0000FF">=</mo><msup><mn mathcolor="#0000FF">10</mn><mrow><mo mathcolor="#0000FF">-</mo><mn mathcolor="#0000FF">6</mn></mrow></msup></mrow></math>.
We compare this to the corresponding quantity without standing
variation (equation (<a href="#S1.E8" title="(8) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>))
and to the quantity only considering standing variants (equation (<a href="#S1.E7" title="(7) ‚Ä£ 1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a>)).
</div>
      
</div>
      
    
</div>
    
<div id="S1.SS4" class="ltx_subsection">
      
<h3 class="ltx_title ltx_title_subsection"><span class="ltx_tag ltx_tag_subsection">1.4 </span>Time to adaptation</h3>

      
<div id="S1.SS4.p1" class="ltx_para">
        
<p class="ltx_p">It is also straightforward to compute the mean time until adaptation.
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">Imagine a geographic location, and let <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m1" class="ltx_Math" alttext="\tau\geq 0" display="inline"><mrow><mi mathcolor="#0000FF">œÑ</mi><mo mathcolor="#0000FF" mathvariant="normal">‚â•</mo><mn mathcolor="#0000FF" mathvariant="normal">0</mn></mrow></math> be the time at
which this location</span> is first reached by some advantageous mutation.
Then, as can be seen from the
perspective of the grey dot in Figure¬†<a href="#S1.F2" title="Figure¬†2 ‚Ä£ Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>,
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m2" class="ltx_Math" alttext="\tau&gt;t" display="inline"><mrow><mi>œÑ</mi><mo>&gt;</mo><mi>t</mi></mrow></math> if and only if the cone with point at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m3" class="ltx_Math" alttext="(x,t)" display="inline"><mrow><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math> and slope <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m4" class="ltx_Math" alttext="v" display="inline"><mi>v</mi></math>
extending back to <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m5" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> (light grey dashed lines) is empty of
successful mutations. Since we assume that <span class="ltx_text ltx_font_italic" style="color:#FF0000;">successful alleles</span> arise as a Poisson process,
in two dimensions</p>
        
<table id="S1.E9" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E9.m1" class="ltx_Math" alttext="\mathbb{P}\{\tau&gt;t\}=\exp\left(-\lambda_{0}\pi v^{2}t^{2}-\lambda\pi v^{2}t^{3%
}/3\right)," display="block"><mrow><mi>‚Ñô</mi><mrow><mo stretchy="false">{</mo><mi>œÑ</mi><mo>&gt;</mo><mi>t</mi><mo stretchy="false">}</mo></mrow><mo>=</mo><mi>exp</mi><mrow><mo>(</mo><mo>-</mo><msub><mi>Œª</mi><mn>0</mn></msub><mi>œÄ</mi><msup><mi>v</mi><mn>2</mn></msup><msup><mi>t</mi><mn>2</mn></msup><mo>-</mo><mi>Œª</mi><mi>œÄ</mi><msup><mi>v</mi><mn>2</mn></msup><msup><mi>t</mi><mn>3</mn></msup><mo>/</mo><mn>3</mn><mo>)</mo></mrow><mo>,</mo></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(9)</span></td></tr>
</table>
        
<p class="ltx_p">i.e., the combined probability that area of a circle of radius <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m6" class="ltx_Math" alttext="vt" display="inline"><mrow><mi>v</mi><mo>‚Å¢</mo><mi>t</mi></mrow></math> surrounding our
point at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m7" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> was free of successful standing variants, and no
successful, new mutations arose in the cone (that has radius <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m8" class="ltx_Math" alttext="r=vt" display="inline"><mrow><mi>r</mi><mo>=</mo><mrow><mi>v</mi><mo>‚Å¢</mo><mi>t</mi></mrow></mrow></math> at
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m9" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> and height <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m10" class="ltx_Math" alttext="h=t" display="inline"><mrow><mi>h</mi><mo>=</mo><mi>t</mi></mrow></math>, and hence volume <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m11" class="ltx_Math" alttext="\pi r^{2}h/3" display="inline"><mrow><mrow><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>r</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>h</mi></mrow><mo>/</mo><mn>3</mn></mrow></math>).
Since <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p1.m12" class="ltx_Math" alttext="\mathbb{E}[\tau]=\int_{0}^{\infty}\mathbb{P}\{\tau&gt;t\}dt" display="inline"><mrow><mi>ùîº</mi><mrow><mo stretchy="false">[</mo><mi>œÑ</mi><mo stretchy="false">]</mo></mrow><mo>=</mo><msubsup><mo largeop="true" symmetric="true">‚à´</mo><mn>0</mn><mi mathvariant="normal">‚àû</mi></msubsup><mi>‚Ñô</mi><mrow><mo stretchy="false">{</mo><mi>œÑ</mi><mo>&gt;</mo><mi>t</mi><mo stretchy="false">}</mo></mrow><mi>d</mi><mi>t</mi></mrow></math>,</p>
        
<table id="S2.EGx3" class="ltx_equationgroup ltx_eqn_align">

<tr id="S1.E10" class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_td ltx_align_right">
                  <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E10.m1" class="ltx_Math" alttext="\displaystyle\mathbb{E}[\tau]" display="inline"><mrow><mi>ùîº</mi><mo>‚Å¢</mo><mrow><mo stretchy="false">[</mo><mi>œÑ</mi><mo stretchy="false">]</mo></mrow></mrow></math>
                </td>
<td class="ltx_td ltx_align_left">
                  <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E10.m2" class="ltx_Math" alttext="\displaystyle=\int_{0}^{\infty}\exp\left(-\lambda_{0}\pi v^{2}t^{2}-\lambda\pi
v%
^{2}t^{3}/3\right)dt." display="inline"><mrow><mrow><mi></mi><mo>=</mo><mrow><mstyle displaystyle="true"><msubsup><mo largeop="true" symmetric="true">‚à´</mo><mn>0</mn><mi mathvariant="normal">‚àû</mi></msubsup></mstyle><mrow><mrow><mi>exp</mi><mo>‚Å°</mo><mrow><mo>(</mo><mrow><mrow><mo>-</mo><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><msup><mi>t</mi><mn>2</mn></msup></mrow></mrow><mo>-</mo><mrow><mrow><mi>Œª</mi><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><msup><mi>t</mi><mn>3</mn></msup></mrow><mo>/</mo><mn>3</mn></mrow></mrow><mo>)</mo></mrow></mrow><mo>‚Å¢</mo><mrow><mo>ùëë</mo><mi>t</mi></mrow></mrow></mrow></mrow><mo>.</mo></mrow></math>
                </td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(10)</span></td></tr>
</table>
        
<p class="ltx_p"><span class="ltx_text ltx_font_italic" style="color:#0000FF;">For applications we evaluate this integral numerically. </span> </p>
      
</div>
      
<div id="S1.SS4.p2" class="ltx_para">
        
<p class="ltx_p">In Figure <a href="#S1.F6" title="Figure¬†6 ‚Ä£ 1.4 Time to adaptation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">6</span></a> we show the mean time until adaptation
for various values of the parameters chosen to match the case of
adaptation at G6PD. Increasing <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p2.m1" class="ltx_Math" alttext="\sigma" display="inline"><mi>œÉ</mi></math> and decreasing <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p2.m2" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math> lower
the time to adaptation, as alleles spread geographically more quickly
and are present as standing variation more often
respectively. Increasing <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS4.p2.m3" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> strongly decreases the time to
adaptation, as it both causes more alleles to escape drift and to
rapidly spread. Given that the G6PD alleles likely spread over a few
thousand years, i.e.¬†less than a few hundred generations,
this time scale seems quite plausible, except perhaps for the lowest dispersal distances.</p>
      
</div>
      
<div id="S1.F6" class="ltx_figure">
        <img src="x14.png" id="S1.F6.g1" class="ltx_graphics ltx_centering" width="677" height="323" alt=""></img>
        
        
        
<div class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure¬†6: </span> <span class="ltx_text ltx_font_bold">Mean adaptation times</span> as a function of <span class="ltx_text ltx_font_italic" style="color:#0000FF;">selective advantage and disadvantage.</span>
We compare these to the corresponding quantities without standing variation and only considering standing variants.
</div>
      
</div>
      
    
</div>
    
<div id="S1.SS5" class="ltx_subsection">
      
<h3 class="ltx_title ltx_title_subsection"><span class="ltx_tag ltx_tag_subsection">1.5 </span>The contribution of standing variation.</h3>

      
<div id="S1.SS5.p1" class="ltx_para">
        
<p class="ltx_p">We can also ask in our framework to address what proportion of new adaptive variants arise from standing variation.
We have defined <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m1" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi>Œª</mi><mn>0</mn></msub></math> to be the mean density
of standing variants that are present and escape loss when the
environment shifts.
We will define <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m2" class="ltx_Math" alttext="\gamma" display="inline"><mi>Œ≥</mi></math> to be the mean density of newly arising alleles
that spread having arisen in an area free of other adaptive alleles.
Since the probability that a mutant arising at location <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m3" class="ltx_Math" alttext="x" display="inline"><mi>x</mi></math> and time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m4" class="ltx_Math" alttext="t" display="inline"><mi>t</mi></math>
is lucky enough to be born in a location not already occupied by mutants
is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m5" class="ltx_Math" alttext="\mathbb{P}\{\tau&gt;t\}" display="inline"><mrow><mi>‚Ñô</mi><mrow><mo stretchy="false">{</mo><mi>œÑ</mi><mo>&gt;</mo><mi>t</mi><mo stretchy="false">}</mo></mrow></mrow></math>,
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">we can see <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m6" class="ltx_Math" alttext="\gamma=\int_{0}^{\infty}\lambda\mathbb{P}\{\tau&gt;t\}dt" display="inline"><mrow><mi mathcolor="#0000FF">Œ≥</mi><mo mathcolor="#0000FF" mathvariant="normal">=</mo><msubsup><mo largeop="true" mathcolor="#0000FF" mathvariant="normal" symmetric="true">‚à´</mo><mn mathcolor="#0000FF" mathvariant="normal">0</mn><mi mathcolor="#0000FF" mathvariant="normal">‚àû</mi></msubsup><mi mathcolor="#0000FF">Œª</mi><mi mathcolor="#0000FF">‚Ñô</mi><mrow><mo mathcolor="#0000FF" mathvariant="normal" stretchy="false">{</mo><mi mathcolor="#0000FF">œÑ</mi><mo mathcolor="#0000FF" mathvariant="normal">&gt;</mo><mi mathcolor="#0000FF">t</mi><mo mathcolor="#0000FF" mathvariant="normal" stretchy="false">}</mo></mrow><mi mathcolor="#0000FF">d</mi><mi mathcolor="#0000FF">t</mi></mrow></math>, and hence
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m7" class="ltx_Math" alttext="\gamma=\lambda\mathbb{E}[\tau]" display="inline"><mrow><mi mathcolor="#0000FF">Œ≥</mi><mo mathcolor="#0000FF" mathvariant="normal">=</mo><mrow><mi mathcolor="#0000FF">Œª</mi><mo mathcolor="#0000FF" mathvariant="italic">‚Å¢</mo><mi mathcolor="#0000FF">ùîº</mi><mo mathcolor="#0000FF" mathvariant="italic">‚Å¢</mo><mrow><mo mathcolor="#0000FF" mathvariant="normal" stretchy="false">[</mo><mi mathcolor="#0000FF">œÑ</mi><mo mathcolor="#0000FF" mathvariant="normal" stretchy="false">]</mo></mrow></mrow></mrow></math>.</span>
Therefore, the mean proportion of <span class="ltx_text ltx_font_italic" style="color:#0000FF;">adapted</span> patches that come from standing variation
is</p>
        
<table id="S1.E11" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E11.m1" class="ltx_Math" alttext="\lambda_{0}/(\lambda_{0}+\gamma)=\frac{1}{1-\log(1/(1-s_{d}))\mathbb{E}[\tau]}," display="block"><mrow><mrow><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>+</mo><mi>Œ≥</mi></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>=</mo><mfrac><mn>1</mn><mrow><mn>1</mn><mo>-</mo><mrow><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>‚Å¢</mo><mi>ùîº</mi><mo>‚Å¢</mo><mrow><mo stretchy="false">[</mo><mi>œÑ</mi><mo stretchy="false">]</mo></mrow></mrow></mrow></mfrac></mrow><mo>,</mo></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(11)</span></td></tr>
</table>
        
<p class="ltx_p">using the fact that <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m8" class="ltx_Math" alttext="\lambda_{0}=\lambda/\log(1/(1-s_{d}))" display="inline"><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>=</mo><mrow><mi>Œª</mi><mo>/</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mi>d</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></math>.
There are <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m9" class="ltx_Math" alttext="\lambda_{0}+\gamma" display="inline"><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>+</mo><mi>Œ≥</mi></mrow></math> patches per unit area, so
the typical patch (informally, the patch around a randomly chosen
successful mutation; formally, drawn from the Palm measure
<cite class="ltx_cite">(Cox and Isham, <a href="#bib.bib181" title="Point processes" class="ltx_ref">1980</a>)</cite>) occupies area <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p1.m10" class="ltx_Math" alttext="1/(\lambda_{0}+\gamma)" display="inline"><mrow><mn>1</mn><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>+</mo><mi>Œ≥</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></math>.</p>
      
</div>
      
<div id="S1.SS5.p2" class="ltx_para">
        
<p class="ltx_p">We can also find the mean proportion of space covered by standing
variants (we will restrict ourselves to two dimensions).
At time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m1" class="ltx_Math" alttext="t" display="inline"><mi>t</mi></math> a geographic location has not yet been reached by the
mutation with probability given by equation (<a href="#S1.E9" title="(9) ‚Ä£ 1.4 Time to adaptation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">9</span></a>).
<span class="ltx_text ltx_font_italic" style="color:#FF0000;">Given that it has not been reached by <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m2" class="ltx_Math" alttext="t" display="inline"><mi mathcolor="#FF0000">t</mi></math>, the probability</span> that it will be reached by time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m3" class="ltx_Math" alttext="t+dt" display="inline"><mrow><mi>t</mi><mo>+</mo><mrow><mi>d</mi><mo>‚Å¢</mo><mi>t</mi></mrow></mrow></math>
by a standing variant is approximately <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m4" class="ltx_Math" alttext="2\lambda_{0}\pi v^{2}tdt" display="inline"><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>Œª</mi><mn>0</mn></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>t</mi><mo>‚Å¢</mo><mi>d</mi><mo>‚Å¢</mo><mi>t</mi></mrow></math>,
which is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m5" class="ltx_Math" alttext="\lambda_{0}" display="inline"><msub><mi>Œª</mi><mn>0</mn></msub></math> multiplied by the thin slice of extra area in our
expanded circle at <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m6" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math>, which has gone from a radius <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m7" class="ltx_Math" alttext="vt" display="inline"><mrow><mi>v</mi><mo>‚Å¢</mo><mi>t</mi></mrow></math> to
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m8" class="ltx_Math" alttext="v(t+dt)" display="inline"><mrow><mi>v</mi><mo>‚Å¢</mo><mrow><mo stretchy="false">(</mo><mrow><mi>t</mi><mo>+</mo><mrow><mi>d</mi><mo>‚Å¢</mo><mi>t</mi></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></math>. The corresponding probability that the point is reached by a new
variant is
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m9" class="ltx_Math" alttext="\lambda\pi v^{2}t^{2}dt" display="inline"><mrow><mi>Œª</mi><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><msup><mi>t</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>d</mi><mo>‚Å¢</mo><mi>t</mi></mrow></math>, which is <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m10" class="ltx_Math" alttext="\lambda" display="inline"><mi>Œª</mi></math> multiplied by the
sliver of extra volume in our space-time cone at time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m11" class="ltx_Math" alttext="t+dt" display="inline"><mrow><mi>t</mi><mo>+</mo><mrow><mi>d</mi><mo>‚Å¢</mo><mi>t</mi></mrow></mrow></math> compared to that at time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p2.m12" class="ltx_Math" alttext="t" display="inline"><mi>t</mi></math>.
Therefore, the mean proportion of space covered by standing variants,
is</p>
        
<table id="S1.E12" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E12.m1" class="ltx_Math" alttext="z_{0}=\int_{0}^{\infty}{2\lambda_{0}\pi v^{2}t}\;\exp\left(-\lambda_{0}\pi v^{%
2}t^{2}-\lambda\pi v^{2}t^{3}/3\right)dt," display="block"><mrow><mrow><msub><mi>z</mi><mn>0</mn></msub><mo>=</mo><mrow><msubsup><mo largeop="true" symmetric="true">‚à´</mo><mn>0</mn><mi mathvariant="normal">‚àû</mi></msubsup><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>Œª</mi><mn>0</mn></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><mpadded width="+2.8pt"><mi>t</mi></mpadded><mo>‚Å¢</mo><mrow><mi>exp</mi><mo>‚Å°</mo><mrow><mo>(</mo><mrow><mrow><mo>-</mo><mrow><msub><mi>Œª</mi><mn>0</mn></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><msup><mi>t</mi><mn>2</mn></msup></mrow></mrow><mo>-</mo><mrow><mrow><mi>Œª</mi><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><msup><mi>t</mi><mn>3</mn></msup></mrow><mo>/</mo><mn>3</mn></mrow></mrow><mo>)</mo></mrow></mrow><mo>‚Å¢</mo><mrow><mo>ùëë</mo><mi>t</mi></mrow></mrow></mrow></mrow><mo>,</mo></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(12)</span></td></tr>
</table>
        
<p class="ltx_p"><span class="ltx_text ltx_font_italic" style="color:#FF0000;">this is the probability a given location is reached first by a
standing variant (which follows from competing our two exponential
waiting times).</span>
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">For applications we evaluate this integral numerically.</span>
</p>
      
</div>
      
<div id="S1.SS5.p3" class="ltx_para">
        
<p class="ltx_p">Furthermore, if we define <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p3.m1" class="ltx_Math" alttext="a_{0}" display="inline"><msub><mi>a</mi><mn>0</mn></msub></math> to be the mean area occupied by a typical standing variant,
then <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p3.m2" class="ltx_Math" alttext="a_{0}" display="inline"><msub><mi>a</mi><mn>0</mn></msub></math> is given by the proportion of the range occupied by standing variants divided by the mean density of unique standing variants,
i.e.¬†<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p3.m3" class="ltx_Math" alttext="a_{0}=z_{0}/\lambda_{0}" display="inline"><mrow><msub><mi>a</mi><mn>0</mn></msub><mo>=</mo><mrow><msub><mi>z</mi><mn>0</mn></msub><mo>/</mo><msub><mi>Œª</mi><mn>0</mn></msub></mrow></mrow></math>.
We can solve for <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p3.m4" class="ltx_Math" alttext="a_{+}" display="inline"><msub><mi>a</mi><mo>+</mo></msub></math>, the corresponding mean area occupied by a given new variant,
using the formula <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS5.p3.m5" class="ltx_Math" alttext="a_{0}/a_{+}=z_{0}/(1-z_{0})" display="inline"><mrow><mrow><msub><mi>a</mi><mn>0</mn></msub><mo>/</mo><msub><mi>a</mi><mo>+</mo></msub></mrow><mo>=</mo><mrow><msub><mi>z</mi><mn>0</mn></msub><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>z</mi><mn>0</mn></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math>.</p>
      
</div>
      
<div id="S1.SS5.p4" class="ltx_para">
        
<p class="ltx_p">In Figure <a href="#S1.F7" title="Figure¬†7 ‚Ä£ 1.5 The contribution of standing variation. ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">7</span></a> we show the proportion of
alleles that spread from standing variation, and the proportion of
geographic space covered by standing variants for parameters chosen to match our G6PD example.
Even for relatively large deleterious costs prior to the environmental switch,
standing variants still make up quite a large proportion of the adaptive alleles,
and an even larger proportion of the range
(they <span class="ltx_text ltx_font_italic" style="color:#FF0000;">occupy a larger area than new mutations, since they get a head start</span>).</p>
      
</div>
      
<div id="S1.F7" class="ltx_figure">
        <img src="x15.png" id="S1.F7.g1" class="ltx_graphics ltx_centering" width="677" height="323" alt=""></img>
        
        
<div class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure¬†7: </span> <span class="ltx_text ltx_font_bold">Mean proportion of patches arising from standing variation,</span>
<span class="ltx_text ltx_font_italic" style="color:#7303EB;">as a function of selective disadvantage, at two population densities.</span>
The parameters given are for the G6PD example.
Both panels show the expected proportion of patches that
arise from standing variation (dotted lines, equation (<a href="#S1.E11" title="(11) ‚Ä£ 1.5 The contribution of standing variation. ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">11</span></a>)),
and the expected proportion of
geographic space that is covered by adaptation from standing variation
(solid lines, equation (<a href="#S1.E12" title="(12) ‚Ä£ 1.5 The contribution of standing variation. ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">12</span></a>)).
We set <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F7.m3" class="ltx_Math" alttext="\mu=10^{-5}" display="inline"><mrow><mi>Œº</mi><mo>=</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>5</mn></mrow></msup></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F7.m4" class="ltx_Math" alttext="s_{b}=0.05" display="inline"><mrow><msub><mi>s</mi><mi>b</mi></msub><mo>=</mo><mn>0.05</mn></mrow></math> to match our G6PD example.</div>
      
</div>
      
    
</div>
    
<div id="S1.SS6" class="ltx_subsection">
      
<h3 class="ltx_title ltx_title_subsection"><span class="ltx_tag ltx_tag_subsection">1.6 </span>Multiple variant types</h3>

      
<div id="S1.SS6.p1" class="ltx_para">
        
<p class="ltx_p">Another problem that we can address with this work is
the extent to which pleiotropy biases
adaptation towards the repeated use of particular subset of loci
(i.e.¬†convergence at genetic level).
While many alleles may confer the beneficial phenotype,
not all will contribute equally to adaptation if they have negative pleiotropic consequences.
There are at least two ways that negative pleiotropy can contribute to high rates of convergence
when adapting if a single change is sufficient for adaptation.
First, negative pleiotropic effects can reduce
the overall beneficial selection coefficient of an allele in the new environment,
making them unlikely to become established and slow to spread
(and in the worst case making them deleterious).
This first effect has been well studied by a number of authors
<cite class="ltx_cite">(Orr, <a href="#bib.bib36" title="Adaptation and the cost of complexity" class="ltx_ref">2000</a>; Otto, <a href="#bib.bib38" title="Two steps forward, one step back: the pleiotropic effects of favoured alleles" class="ltx_ref">2004</a>; Welch and Waxman, <a href="#bib.bib35" title="Modularity and the cost of complexity" class="ltx_ref">2003</a>; Chevin<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib37" title="Fisher‚Äôs model and the genomics of adaptation: restricted pleiotropy, heterogenous mutation, and parallel evolution" class="ltx_ref">2010</a>)</cite> and its role in genetic
convergence examined <cite class="ltx_cite">(Orr, <a href="#bib.bib44" title="The probability of parallel evolution" class="ltx_ref">2005</a>; Chevin<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib37" title="Fisher‚Äôs model and the genomics of adaptation: restricted pleiotropy, heterogenous mutation, and parallel evolution" class="ltx_ref">2010</a>; Unckless and Orr, <a href="#bib.bib34" title="Dobzhansky-Muller incompatibilities and adaptation to a shared environment" class="ltx_ref">2009</a>)</cite>.
A second contribution is that alleles that have
less negative pleiotropy are more likely to be present as standing variation
before the environmental shift, and so are more able to respond
immediately.</p>
      
</div>
      
<div id="S1.SS6.p2" class="ltx_para">
        
<p class="ltx_p">Here we focus primarily on the second effect.
Let‚Äôs imagine for the moment <span class="ltx_text ltx_font_italic" style="color:#7303EB;">that there are several classes of beneficial allele,</span>
all having the same beneficial selection coefficient
(<span class="ltx_text ltx_font_italic" style="color:#0000FF;">or</span> at least that beneficial selection coefficients are similar enough that our selective exclusion
approximation holds over the time scale on which we examine the process).
Each class of mutations <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p2.m1" class="ltx_Math" alttext="j" display="inline"><mi>j</mi></math> has its own mutation rate <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p2.m2" class="ltx_Math" alttext="\mu_{j}" display="inline"><msub><mi>Œº</mi><mi>j</mi></msub></math> and selective disadvantage <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p2.m3" class="ltx_Math" alttext="s_{d,j}" display="inline"><msub><mi>s</mi><mrow><mi>d</mi><mo>,</mo><mi>j</mi></mrow></msub></math> prior to the environmental switch.
As they have the same beneficial selection coefficient after the switch,
all of the waves travel outward at a rate <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p2.m4" class="ltx_Math" alttext="v" display="inline"><mi>v</mi></math>.
Then, the density of type <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p2.m5" class="ltx_Math" alttext="j" display="inline"><mi>j</mi></math> standing variants per unit area
and the input rate of <span class="ltx_text ltx_font_italic">de novo</span> variants per unit area per generation are, respectively,</p>
        
<table id="S2.EGx4" class="ltx_equationgroup ltx_eqn_align">

<tr id="S1.E13" class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_td ltx_align_right">
                  <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E13.m1" class="ltx_Math" alttext="\displaystyle\lambda_{0,j}=\frac{2\mu_{j}\rho s_{b}}{-\log(1-s_{d,j})}\qquad%
\text{and}\qquad\lambda_{j}=2\mu_{j}\rho s_{b}." display="inline"><mrow><mrow><mrow><msub><mi>Œª</mi><mrow><mn>0</mn><mo>,</mo><mi>j</mi></mrow></msub><mo>=</mo><mrow><mstyle displaystyle="true"><mfrac><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>Œº</mi><mi>j</mi></msub><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow><mrow><mo>-</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mrow><mi>d</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mfrac></mstyle><mo mathvariant="italic" separator="true">‚ÄÉ‚ÄÉ</mo><mtext>and</mtext></mrow></mrow><mo mathvariant="italic" separator="true">‚ÄÉ‚ÄÉ</mo><mrow><msub><mi>Œª</mi><mi>j</mi></msub><mo>=</mo><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>Œº</mi><mi>j</mi></msub><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow></mrow></mrow><mo>.</mo></mrow></math>
                </td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(13)</span></td></tr>
</table>
        
<p class="ltx_p">Using these rates, and an argument analogous to that used to derive
equation (<a href="#S1.E12" title="(12) ‚Ä£ 1.5 The contribution of standing variation. ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">12</span></a>),
at the time when every location has been reached by an adaptive allele,
the proportion of the species range covered by alleles of type <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p2.m6" class="ltx_Math" alttext="j" display="inline"><mi>j</mi></math> is</p>
        
<table id="S1.E14" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E14.m1" class="ltx_Math" alttext="p_{j}=\int_{0}^{\infty}\left(2\lambda_{0,j}\pi v^{2}t+\lambda_{j}\pi v^{2}t^{2%
}\right)\;\exp\left(-\sum_{k}(\lambda_{0,k}\pi v^{2}t^{2}+\lambda_{k}\pi v^{2}%
t^{3}/3)\right)dt." display="block"><mrow><mrow><msub><mi>p</mi><mi>j</mi></msub><mo>=</mo><mrow><msubsup><mo largeop="true" symmetric="true">‚à´</mo><mn>0</mn><mi mathvariant="normal">‚àû</mi></msubsup><mrow><mrow><mo>(</mo><mrow><mrow><mn>2</mn><mo>‚Å¢</mo><msub><mi>Œª</mi><mrow><mn>0</mn><mo>,</mo><mi>j</mi></mrow></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>t</mi></mrow><mo>+</mo><mrow><msub><mi>Œª</mi><mi>j</mi></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><msup><mi>t</mi><mn>2</mn></msup></mrow></mrow><mo rspace="5.3pt">)</mo></mrow><mo>‚Å¢</mo><mrow><mi>exp</mi><mo>‚Å°</mo><mrow><mo>(</mo><mrow><mo>-</mo><mrow><munder><mo largeop="true" movablelimits="false" symmetric="true">‚àë</mo><mi>k</mi></munder><mrow><mo stretchy="false">(</mo><mrow><mrow><msub><mi>Œª</mi><mrow><mn>0</mn><mo>,</mo><mi>k</mi></mrow></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><msup><mi>t</mi><mn>2</mn></msup></mrow><mo>+</mo><mrow><mrow><msub><mi>Œª</mi><mi>k</mi></msub><mo>‚Å¢</mo><mi>œÄ</mi><mo>‚Å¢</mo><msup><mi>v</mi><mn>2</mn></msup><mo>‚Å¢</mo><msup><mi>t</mi><mn>3</mn></msup></mrow><mo>/</mo><mn>3</mn></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow><mo>)</mo></mrow></mrow><mo>‚Å¢</mo><mrow><mo>ùëë</mo><mi>t</mi></mrow></mrow></mrow></mrow><mo>.</mo></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(14)</span></td></tr>
</table>
        
<p class="ltx_p">If we only allow standing variation this collapses to</p>
        
<table id="S1.E15" class="ltx_equation">

<tr class="ltx_equation ltx_align_baseline">
<td class="ltx_eqn_pad"></td>
<td class="ltx_align_center"><math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.E15.m1" class="ltx_Math" alttext="p_{j}=\frac{\mu_{j}/\log(1-s_{d,j})}{\sum_{k}\mu_{k}/\log(1-s_{d,k})}," display="block"><mrow><mrow><msub><mi>p</mi><mi>j</mi></msub><mo>=</mo><mfrac><mrow><msub><mi>Œº</mi><mi>j</mi></msub><mo>/</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mrow><mi>d</mi><mo>,</mo><mi>j</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow><mrow><msub><mo largeop="true" symmetric="true">‚àë</mo><mi>k</mi></msub><mrow><msub><mi>Œº</mi><mi>k</mi></msub><mo>/</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>-</mo><msub><mi>s</mi><mrow><mi>d</mi><mo>,</mo><mi>k</mi></mrow></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></mfrac></mrow><mo>,</mo></mrow></math></td>
<td class="ltx_eqn_pad"></td>
<td rowspan="1" class="ltx_align_middle ltx_align_right"><span class="ltx_tag ltx_tag_equation">(15)</span></td></tr>
</table>
        
<p class="ltx_p">while if we only allow new variation,
i.e.¬†if all variants are highly deleterious before the environment switches,
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p2.m7" class="ltx_Math" alttext="p_{j}=\mu_{j}/(\sum_{k}\mu_{k})" display="inline"><mrow><msub><mi>p</mi><mi>j</mi></msub><mo>=</mo><mrow><msub><mi>Œº</mi><mi>j</mi></msub><mo>/</mo><mrow><mo stretchy="false">(</mo><mrow><msub><mo largeop="true" symmetric="true">‚àë</mo><mi>k</mi></msub><msub><mi>Œº</mi><mi>k</mi></msub></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math>.</p>
      
</div>
      
<div id="S1.SS6.p3" class="ltx_para">
        
<p class="ltx_p">To illustrate some of the properties of this model,
let‚Äôs imagine the somewhat extreme scenario in which there is a single base pair
at which a possible mutation is relatively free of negative pleiotropy (call this class <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p3.m1" class="ltx_Math" alttext="1" display="inline"><mn>1</mn></math>);
and a larger mutational target where changes
have more serious pleiotropic consequences in the ancestral environment (class <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p3.m2" class="ltx_Math" alttext="2" display="inline"><mn>2</mn></math>).
We set <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p3.m3" class="ltx_Math" alttext="s_{d,1}\leq s_{d,2}=0.05" display="inline"><mrow><msub><mi>s</mi><mrow><mi>d</mi><mo>,</mo><mn>1</mn></mrow></msub><mo>‚â§</mo><msub><mi>s</mi><mrow><mi>d</mi><mo>,</mo><mn>2</mn></mrow></msub><mo>=</mo><mn>0.05</mn></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p3.m4" class="ltx_Math" alttext="\mu_{1}=1\times 10^{-8}" display="inline"><mrow><msub><mi>Œº</mi><mn>1</mn></msub><mo>=</mo><mrow><mn>1</mn><mo>√ó</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>8</mn></mrow></msup></mrow></mrow></math>,
assume that both classes share a beneficial selection coefficient of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p3.m5" class="ltx_Math" alttext="s_{b}=0.05" display="inline"><mrow><msub><mi>s</mi><mi>b</mi></msub><mo>=</mo><mn>0.05</mn></mrow></math>,
and think of the second class of alleles as arising at one of ten, one hundred, or one thousand base pairs.
We show the expected proportion of space covered by
the rarer, class <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p3.m6" class="ltx_Math" alttext="1" display="inline"><mn>1</mn></math> mutations in Figure <a href="#S1.F8" title="Figure¬†8 ‚Ä£ 1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>.
<span class="ltx_text ltx_font_italic" style="color:#FF0000;">As expected intuitively,</span> the contribution of the rarer mutation decreases as the
mutational target of the second class becomes larger, and as the
difference in the negative pleiotropic consequences of the two classes of alleles decreases.
The case with standing variation only is the best case scenario
for the rarer mutation, so its rate of introduction after <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p3.m7" class="ltx_Math" alttext="t=0" display="inline"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math> is necessarily lower.
However, the standing-variation-only case does seem to provide a
reasonable rule of thumb, especially for parameter combinations,
such as high population densities and high dispersal distances,
that increase the contribution of standing variation (and similarly for high <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p3.m8" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math>).</p>
      
</div>
      
<div id="S1.SS6.p4" class="ltx_para">
        
<p class="ltx_p">It is natural to also incorporate differences in the beneficial selection coefficients of the different classes of alleles,
to allow for negative pleiotropic effects acting to suppress the advantage of an allele once the environment switches.
One simple way to do this is to simply replace <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m1" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> with <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m2" class="ltx_Math" alttext="s_{b,j}" display="inline"><msub><mi>s</mi><mrow><mi>b</mi><mo>,</mo><mi>j</mi></mrow></msub></math>
resulting in a class-specific new allele establishment rates (<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m3" class="ltx_Math" alttext="\lambda_{j}" display="inline"><msub><mi>Œª</mi><mi>j</mi></msub></math>) and rates of spread (<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m4" class="ltx_Math" alttext="v_{j}" display="inline"><msub><mi>v</mi><mi>j</mi></msub></math>).
These then could be used in equation (<a href="#S1.E14" title="(14) ‚Ä£ 1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">14</span></a>).
For instance, we could extend the two class model above
so that class one has the additional advantage of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m5" class="ltx_Math" alttext="s_{1,b}&gt;s_{2,b}" display="inline"><mrow><msub><mi>s</mi><mrow><mn>1</mn><mo>,</mo><mi>b</mi></mrow></msub><mo>&gt;</mo><msub><mi>s</mi><mrow><mn>2</mn><mo>,</mo><mi>b</mi></mrow></msub></mrow></math>.
This would further increase the contribution of the rarer class,
because this class would both overcome drift more often and spread more rapidly.
However, a straightforward application of the logic of used to construct equation (<a href="#S1.E14" title="(14) ‚Ä£ 1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">14</span></a>)
fails once the different allelic types meet,
since the assumption of selective exclusion no longer holds:
alleles with higher <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m6" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> will spread, at a lower speed, into regions occupied by alleles with lower <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m7" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math>.
Given enough time, the most advantageous type (type 1, in this case) would spread everywhere,
and so substituting multiple values for <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m8" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> in equation (<a href="#S1.E14" title="(14) ‚Ä£ 1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">14</span></a>)
would only provide a short-term approximation to a longer term dynamic.
Even if the initial tessellation has formed with purely class 2 alleles,
the first allele would have a selective advantage <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m9" class="ltx_Math" alttext="\delta s=s_{b,1}-s_{b,2}" display="inline"><mrow><mrow><mi>Œ¥</mi><mo>‚Å¢</mo><mi>s</mi></mrow><mo>=</mo><mrow><msub><mi>s</mi><mrow><mi>b</mi><mo>,</mo><mn>1</mn></mrow></msub><mo>-</mo><msub><mi>s</mi><mrow><mi>b</mi><mo>,</mo><mn>2</mn></mrow></msub></mrow></mrow></math>,
and so would arise at rate <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m10" class="ltx_Math" alttext="2\rho\mu_{1}\delta s" display="inline"><mrow><mn>2</mn><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><msub><mi>Œº</mi><mn>1</mn></msub><mo>‚Å¢</mo><mi>Œ¥</mi><mo>‚Å¢</mo><mi>s</mi></mrow></math> and would spread at speed <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m11" class="ltx_Math" alttext="\sigma\sqrt{\delta s}" display="inline"><mrow><mi>œÉ</mi><mo>‚Å¢</mo><msqrt><mrow><mi>Œ¥</mi><mo>‚Å¢</mo><mi>s</mi></mrow></msqrt></mrow></math>.
An extension of our Poisson process model could incorporate these effects,
by thinning the Poisson process of establishing mutations by correctly,
but is considerably less tractable.
Whether allele <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m12" class="ltx_Math" alttext="2" display="inline"><mn>2</mn></math> persists would depend on the linkage arrangements between loci.
If the loci underlying allele 1 and 2 are unlinked, then allele 1 can spread without disrupting allele 2.
However, if they are linked, the spread of allele <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m13" class="ltx_Math" alttext="1" display="inline"><mn>1</mn></math> may push allele <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS6.p4.m14" class="ltx_Math" alttext="2" display="inline"><mn>2</mn></math> out of the population.
More complicated dynamics,
including spatial Dobzhansky-Muller incompatibilities <cite class="ltx_cite">(Kondrashov, <a href="#bib.bib99" title="Accumulation of Dobzhansky-Muller incompatibilities within a spatially structured population" class="ltx_ref">2003</a>; Ralph and Coop, <a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite>
could ensue if there are epistatic interactions between the alleles.</p>
      
</div>
      
      
<div id="S1.F8" class="ltx_figure">
        <img src="x16.png" id="S1.F8.g1" class="ltx_graphics ltx_centering" width="676" height="624" alt=""></img>
        
        
<div class="ltx_caption ltx_centering"><span class="ltx_tag ltx_tag_figure">Figure¬†8: </span> <span class="ltx_text ltx_font_bold">Proportion of space covered by rarer but less negatively pleiotropic
mutation</span>. Empty circles give the result for standing variation only,
equation (<a href="#S1.E15" title="(15) ‚Ä£ 1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">15</span></a>). Lines give
the result allowing both standing and <span class="ltx_text ltx_font_italic">de novo</span> mutations (equation (<a href="#S1.E14" title="(14) ‚Ä£ 1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">14</span></a>)).
Here we hold <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F8.m3" class="ltx_Math" alttext="s_{b}=s_{d,2}=0.05" display="inline"><mrow><msub><mi>s</mi><mi>b</mi></msub><mo>=</mo><msub><mi>s</mi><mrow><mi>d</mi><mo>,</mo><mn>2</mn></mrow></msub><mo>=</mo><mn>0.05</mn></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.F8.m4" class="ltx_Math" alttext="\mu_{1}=1\times 10^{-8}" display="inline"><mrow><msub><mi>Œº</mi><mn>1</mn></msub><mo>=</mo><mrow><mn>1</mn><mo>√ó</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>8</mn></mrow></msup></mrow></mrow></math>.
</div>
      
</div>
    
</div>
    
<div id="S1.SS7" class="ltx_subsection">
      
<h3 class="ltx_title ltx_title_subsection"><span class="ltx_tag ltx_tag_subsection">1.7 </span>Local establishment and comparison to panmixia</h3>

      
<div id="S1.SS7.p1" class="ltx_para">
        
<p class="ltx_p">In the above, we have assumed that once the mutation appears,
conditional on eventual fixation, it begins to spread spatially at speed <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p1.m1" class="ltx_Math" alttext="v" display="inline"><mi>v</mi></math> instantly,
effectively neglecting the time it must first spend escaping demographic stochasticity.
In <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite> we addressed this by noting that there would be no change at all in our results
if all mutations had to wait the same amount of time before fixing locally,
and that this time was short relative to the time it took the wave to spread across the characteristic length;
we then showed via simulation that this was reasonable in certain situations.
In this section we examine this assumption in more detail, although mostly through heuristic arguments,
and also compare the results above to the results without geographic structure of <cite class="ltx_cite">Pennings and Hermisson (<a href="#bib.bib117" title="Soft sweeps II - molecular population genetics of adaptation from recurrent mutation or migration" class="ltx_ref">2006</a>)</cite>.</p>
      
</div>
      
<div id="S1.SS7.p2" class="ltx_para">
        
<p class="ltx_p">We are assuming that shortly after a new mutation appears,
it can be approximated by a branching process growing at rate <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m1" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi mathcolor="#0000FF">s</mi><mi mathcolor="#0000FF">b</mi></msub></math>
until the point that it grows large enough to ‚Äúfeel‚Äù spatial structure,
at which point it begins to spread as a more‚Äìor‚Äìless deterministic wave.
Although we are not aware of good analysis of this transition,
the relevant size <span class="ltx_text ltx_font_italic" style="color:#0000FF;">of the branching process</span> when spatial structure becomes important
should be something close to <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m2" class="ltx_Math" alttext="\sigma^{2}\rho" display="inline"><mrow><msup><mi>œÉ</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>œÅ</mi></mrow></math>
(i.e.¬†Wright‚Äôs ‚Äúlocal effective population size‚Äù, <cite class="ltx_cite">Wright (<a href="#bib.bib31" title="Isolation by Distance" class="ltx_ref">1943</a>)</cite>).
Let <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m3" class="ltx_Math" alttext="Z_{t}" display="inline"><msub><mi>Z</mi><mi>t</mi></msub></math> be a continuous-time branching process with <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m4" class="ltx_Math" alttext="Z_{0}=1" display="inline"><mrow><msub><mi>Z</mi><mn>0</mn></msub><mo>=</mo><mn>1</mn></mrow></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m5" class="ltx_Math" alttext="\mathbb{E}[Z_{t}]=e^{s_{b}t}" display="inline"><mrow><mrow><mi>ùîº</mi><mo>‚Å¢</mo><mrow><mo stretchy="false">[</mo><msub><mi>Z</mi><mi>t</mi></msub><mo stretchy="false">]</mo></mrow></mrow><mo>=</mo><msup><mi>e</mi><mrow><msub><mi>s</mi><mi>b</mi></msub><mo>‚Å¢</mo><mi>t</mi></mrow></msup></mrow></math>.
Then we know that there exists a random variable <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m6" class="ltx_Math" alttext="W" display="inline"><mi>W</mi></math> such that <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m7" class="ltx_Math" alttext="\lim_{t\to\infty}e^{-s_{b}t}Z_{t}=W" display="inline"><mrow><mrow><msub><mo>lim</mo><mrow><mi>t</mi><mo>‚Üí</mo><mi mathvariant="normal">‚àû</mi></mrow></msub><mo>‚Å°</mo><mrow><msup><mi>e</mi><mrow><mo>-</mo><mrow><msub><mi>s</mi><mi>b</mi></msub><mo>‚Å¢</mo><mi>t</mi></mrow></mrow></msup><mo>‚Å¢</mo><msub><mi>Z</mi><mi>t</mi></msub></mrow></mrow><mo>=</mo><mi>W</mi></mrow></math> almost surely,
so that if <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m8" class="ltx_Math" alttext="\tau" display="inline"><mi>œÑ</mi></math> is the time <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m9" class="ltx_Math" alttext="Z_{t}" display="inline"><msub><mi>Z</mi><mi>t</mi></msub></math> reaches size <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m10" class="ltx_Math" alttext="\sigma^{2}\rho" display="inline"><mrow><msup><mi>œÉ</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>œÅ</mi></mrow></math>, then <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m11" class="ltx_Math" alttext="\sigma^{2}\rho=Z_{\tau}\approx e^{s_{b}\tau}W" display="inline"><mrow><mrow><msup><mi>œÉ</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>œÅ</mi></mrow><mo>=</mo><msub><mi>Z</mi><mi>œÑ</mi></msub><mo>‚âà</mo><mrow><msup><mi>e</mi><mrow><msub><mi>s</mi><mi>b</mi></msub><mo>‚Å¢</mo><mi>œÑ</mi></mrow></msup><mo>‚Å¢</mo><mi>W</mi></mrow></mrow></math> <cite class="ltx_cite">(Jagers, <a href="#bib.bib131" title="Branching processes with biological applications" class="ltx_ref">1975</a>)</cite>.
From this we know that <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m12" class="ltx_Math" alttext="\tau\approx(1/s_{b})(\log(\sigma^{2}\rho)-\log W)" display="inline"><mrow><mi>œÑ</mi><mo>‚âà</mo><mrow><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>/</mo><msub><mi>s</mi><mi>b</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mo>‚Å¢</mo><mrow><mo stretchy="false">(</mo><mrow><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><msup><mi>œÉ</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>œÅ</mi></mrow><mo stretchy="false">)</mo></mrow></mrow><mo>-</mo><mrow><mi>log</mi><mo>‚Å°</mo><mi>W</mi></mrow></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></math>;
although more detailed information is available (e.g.¬†a central limit theorem for <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p2.m13" class="ltx_Math" alttext="\tau" display="inline"><mi>œÑ</mi></math>, <cite class="ltx_cite">Nagaev (<a href="#bib.bib180" title="A limit theorem for a supercritical branching process" class="ltx_ref">1971</a>)</cite>),
we will stick to the loose interpretation.</p>
      
</div>
      
<div id="S1.SS7.p3" class="ltx_para">
        
<p class="ltx_p">So, roughly speaking, we need to evaluate the importance of a delay of about <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p3.m1" class="ltx_Math" alttext="T=(1/s_{b})\log(\sigma^{2}\rho)" display="inline"><mrow><mi>T</mi><mo>=</mo><mrow><mrow><mo stretchy="false">(</mo><mrow><mn>1</mn><mo>/</mo><msub><mi>s</mi><mi>b</mi></msub></mrow><mo stretchy="false">)</mo></mrow><mo>‚Å¢</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><msup><mi>œÉ</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>œÅ</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></math>.
New mutations will appear and become established during this time if
<math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p3.m2" class="ltx_Math" alttext="2\rho\sigma^{2}\mu s_{b}\geq 1/T" display="inline"><mrow><mrow><mn>2</mn><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><msup><mi>œÉ</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>Œº</mi><mo>‚Å¢</mo><msub><mi>s</mi><mi>b</mi></msub></mrow><mo>‚â•</mo><mrow><mn>1</mn><mo>/</mo><mi>T</mi></mrow></mrow></math>,
i.e.¬†if <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p3.m3" class="ltx_Math" alttext="2\rho\sigma^{2}\mu\geq 1/\log(\sigma^{2}\rho)" display="inline"><mrow><mrow><mn>2</mn><mo>‚Å¢</mo><mi>œÅ</mi><mo>‚Å¢</mo><msup><mi>œÉ</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>Œº</mi></mrow><mo>‚â•</mo><mrow><mn>1</mn><mo>/</mo><mrow><mi>log</mi><mo>‚Å°</mo><mrow><mo stretchy="false">(</mo><mrow><msup><mi>œÉ</mi><mn>2</mn></msup><mo>‚Å¢</mo><mi>œÅ</mi></mrow><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></math>.
Our model will still be a good approximation, however,
as long as <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p3.m4" class="ltx_Math" alttext="T" display="inline"><mi>T</mi></math> is short relative to the time a wave takes to spread between nearby mutational origins.
This can be worked out,
but it is simpler to note that if the converse is true
<span class="ltx_text ltx_font_italic" style="color:#FF0000;">(i.e., that reaching local fixation is slow compared to the spread of the wave)</span>,
the process is largely unaffected by spatial structure,
and so the panmictic model is a good approximation for the true process.
</p>
      
</div>
      
      
<div id="S1.SS7.p4" class="ltx_para">
        
<p class="ltx_p"><cite class="ltx_cite">Pennings and Hermisson (<a href="#bib.bib117" title="Soft sweeps II - molecular population genetics of adaptation from recurrent mutation or migration" class="ltx_ref">2006</a>)</cite> show that under a panmictic model with certain assumptions on the parameters,
the number of independent origins due to both standing variation and new mutation seen in a sample of size <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p4.m1" class="ltx_Math" alttext="n" display="inline"><mi>n</mi></math>
has approximately the Ewens distribution with parameters <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p4.m2" class="ltx_Math" alttext="n" display="inline"><mi>n</mi></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p4.m3" class="ltx_Math" alttext="\theta=4N\mu" display="inline"><mrow><mi>Œ∏</mi><mo>=</mo><mrow><mn>4</mn><mo>‚Å¢</mo><mi>N</mi><mo>‚Å¢</mo><mi>Œº</mi></mrow></mrow></math>.
As <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p4.m4" class="ltx_Math" alttext="n" display="inline"><mi>n</mi></math> increases, the total number of types seen grows as <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p4.m5" class="ltx_Math" alttext="\log n" display="inline"><mrow><mi>log</mi><mo>‚Å°</mo><mi>n</mi></mrow></math>.
In our model, we can increase total population size by increasing either the density <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p4.m6" class="ltx_Math" alttext="\rho" display="inline"><mi>œÅ</mi></math> or the total amount of area.
In either case, the predicted number of distinct types grows linearly with <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p4.m7" class="ltx_Math" alttext="n" display="inline"><mi>n</mi></math>,
much faster than under panmixia.</p>
      
</div>
      
<div id="S1.SS7.p5" class="ltx_para">
        
<p class="ltx_p">The results of <cite class="ltx_cite">Hermisson and Pennings (<a href="#bib.bib116" title="Soft sweeps: molecular population genetics of adaptation from standing genetic variation" class="ltx_ref">2005</a>)</cite> and <cite class="ltx_cite">Pennings and Hermisson (<a href="#bib.bib117" title="Soft sweeps II - molecular population genetics of adaptation from recurrent mutation or migration" class="ltx_ref">2006</a>)</cite> suggest that in a
panmictic population the number of independent alleles (and their frequencies) in a sample
is nearly independent of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p5.m1" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p5.m2" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math> <cite class="ltx_cite">(although this
breaks down with fluctuating population size,
 Wilson<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib52" title="Soft selective sweeps in complex demographic scenarios" class="ltx_ref">2014</a>)</cite>. In the panmictic model the
lack of dependence on <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p5.m3" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> comes about because while increasing <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p5.m4" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math>
increases the rate at which independent mutations become established,
it also accelerates the frequency gain of established alleles,
hence decreasing the time period in which new alleles can arise and hope to be at significant frequency in the population.
These two effects <span class="ltx_text ltx_font_italic" style="color:#FF0000;">approximately</span> cancel each other out leading to no strong effect of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p5.m5" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> on the number of independent alleles.
Decreasing <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p5.m6" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math> increases the number of standing variants within a population,
increasing the number of alleles that manage to establish and spread from standing variation <cite class="ltx_cite">(Hermisson and Pennings, <a href="#bib.bib116" title="Soft sweeps: molecular population genetics of adaptation from standing genetic variation" class="ltx_ref">2005</a>; Orr and Betancourt, <a href="#bib.bib43" title="Haldane‚Äôs sieve and adaptation from the standing genetic variation" class="ltx_ref">2001</a>)</cite>.
However, having more established standing alleles
acts to exclude the spread of new alleles that arise once the environment switches.
These two opposing effects again cancel out, leading to little overall
effect of <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p5.m7" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math> on the number of independent alleles.
In contrast, our results show that the characteristic length
(closely related to the density of independent alleles)
depends on both <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p5.m8" class="ltx_Math" alttext="s_{b}" display="inline"><msub><mi>s</mi><mi>b</mi></msub></math> and <math xmlns="http://www.w3.org/1998/Math/MathML" id="S1.SS7.p5.m9" class="ltx_Math" alttext="s_{d}" display="inline"><msub><mi>s</mi><mi>d</mi></msub></math> in a geographically spread population.
Like the panmictic model, in our model, alleles also act to exclude each other;
however, the geographic spread of an allele is slow compared to the initial exponential growth of an allele in a panmictic population.
That means that the role of selection in helping
alleles become established can dominate, leading to more independent origins, both by
being weaker before and stronger after the environmental switch.</p>
      
</div>
      
    
</div>
  
</div>
  
<div id="S2" class="ltx_section">
    
<h2 class="ltx_title ltx_title_section"><span class="ltx_tag ltx_tag_section">2 </span>Discussion</h2>

    
<div id="S2.p1" class="ltx_para">
      
<p class="ltx_p">When the geographical area where a species experiences a
selection pressure is greater than our characteristic
length we expect multiple independent alleles to arise and
spread in that area. Our results suggest that convergent evolution
among populations may be quite common, <span class="ltx_text ltx_font_italic" style="color:#0000FF;">at least when population
densities and mutational target sizes are not too small, and dispersal is
limited across the range. While considering smaller mutational
target sizes (e.g., less than the 100bp in our example) or lower population densities would lower the probability of convergence
within a given geographical area, it would also act to considerably lengthen
the time to adaptation. In such cases adaptation may simply fail to
occur at all or populations may adapt via some other means (e.g., by using a
broader mutational target).
</span></p>
    
</div>
    
<div id="S2.p2" class="ltx_para">
      
<p class="ltx_p"><span class="ltx_text ltx_font_italic" style="color:#0000FF;">Our inclusion of standing variation
before environmental change greatly increase the probability of
convergence within a species.</span>
While at face value this increase seems unsurprising,
this relationship differs markedly from the case of multiple competing
alleles in a panmictic population <cite class="ltx_cite">(see discussion above and in  Hermisson and Pennings, <a href="#bib.bib116" title="Soft sweeps: molecular population genetics of adaptation from standing genetic variation" class="ltx_ref">2005</a>; Pennings and Hermisson, <a href="#bib.bib117" title="Soft sweeps II - molecular population genetics of adaptation from recurrent mutation or migration" class="ltx_ref">2006</a>)</cite>.
Importantly, allowing standing variation may greatly lower the time until the species becomes adapted
across the geographic range of the selection pressure. We have also
shown that adaptation through standing variation biases the type of variation towards those alleles
with fewer pleiotropic effects, since these are more common as standing variation before the environmental shift.
This bias can in some cases easily overcome quite significant differences in mutational target sizes
among loci allowing the same locus to be repeatedly the source of
adaptation even if there are seemingly many different routes to adaptation.
(See Figure¬†<a href="#S1.F8" title="Figure¬†8 ‚Ä£ 1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">8</span></a>.)</p>
    
</div>
    
<div id="S2.SS7.SSS0.Px1" class="ltx_paragraph">
      
<h5 class="ltx_title ltx_title_paragraph">The confusing signal of geographic convergent evolution.</h5>

      
<div id="S2.SS7.SSS0.Px1.p1" class="ltx_para">
        
<p class="ltx_p">As we have argued in <cite class="ltx_cite">Ralph and Coop (<a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>)</cite> the ease with which
geographic convergent adaptation occurs means that we should
incorporate it more widely into our thinking about the genetic basis
of adaptation. For example, the absence of European skin pigmentation alleles in
ancient DNA from Europeans who lived several thousand years ago
has led to the suggestion that these individuals had dark
pigmentation <cite class="ltx_cite">(Olalde<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib41" title="Derived immune and ancestral pigmentation alleles in a 7,000-year-old mesolithic european" class="ltx_ref">2014</a>; Lazaridis<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib40" title="Ancient human genomes suggest three ancestral populations for present-day europeans" class="ltx_ref">2014</a>; Wilde<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib39" title="Direct evidence for positive selection of skin, hair, and eye pigmentation in Europeans during the last 5,000 years" class="ltx_ref">2014</a>)</cite>.
However, given our results and the partially convergent
basis of skin pigmentation between Europeans and East Asians
<cite class="ltx_cite">(Norton<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib17" title="Genetic evidence for the convergent evolution of light skin in Europeans and east Asians" class="ltx_ref">2007</a>; Edwards<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib18" title="Association of the OCA2 polymorphism His615Arg with melanin content in east Asian populations: further evidence of convergent evolution of skin pigmentation" class="ltx_ref">2010</a>)</cite> it seems just as plausible that these ancient individuals
adapted to high latitudes via a different complement of ‚Äúlight-skin‚Äù pigmentation alleles;
to our knowledge, we have no strong evidence either way.
Such convergence may considerably complicate the
exploration of phenotypes and adaptation among populations using variants mapped in a
limited set of populations <cite class="ltx_cite">(Berg and Coop, <a href="#bib.bib46" title="A population genetic signal of polygenic adaptation" class="ltx_ref">2014</a>)</cite>.</p>
      
</div>
      
<div id="S2.SS7.SSS0.Px1.p2" class="ltx_para">
        
<p class="ltx_p">More generally, if geographic convergence is common, we should often expect to see
selected alleles that are strongly geographically restricted as they
have simply not had time for neutral gene flow to spread them across the
landscape. <span class="ltx_text ltx_font_italic" style="color:#0000FF;">Such convergent alleles will be <math xmlns="http://www.w3.org/1998/Math/MathML" id="S2.SS7.SSS0.Px1.p2.m1" class="ltx_Math" alttext="F_{ST}" display="inline"><msub><mi mathcolor="#0000FF">F</mi><mrow><mi mathcolor="#0000FF">S</mi><mo mathcolor="#0000FF" mathvariant="italic">‚Å¢</mo><mi mathcolor="#0000FF">T</mi></mrow></msub></math>-outliers compared to
older neutral variation, untill there is sufficient time for
migration to smooth out them out across the landscape.</span>
This pattern may be very hard to distinguish from local adaptation using population genomic approaches
alone. This is especially problematic as boundaries between convergent alleles may often
occur where gene flow rates are low, i.e.¬†historical and ecological breaks, even if
the alleles concerned have no bearing on the ecological differences
across these breaks <cite class="ltx_cite">(see Bierne<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib16" title="The coupling hypothesis: why genome scans may fail to map local adaptation genes" class="ltx_ref">2011</a>, for a wide-ranging discussion of how allelic differentiation may build along particular zones)</cite>.
We are rarely so fortunate as to know as much about the genetics, phenotypic distributions,
and potential selection agents as we do for malaria resistance in humans.
Therefore, we must be wary of mistaking the strange spatial distributions of particular alleles
for adaptation to some very specific selection pressure (e.g.¬†the distribution of the Mahidol allele in Figure <a href="#S1.F3" title="Figure¬†3 ‚Ä£ 1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>),
when they are simply elements of a larger geographic mosaic of alleles responding to a broadly shared selection pressure.</p>
      
</div>
      
      
<div id="S2.SS7.SSS0.Px1.p3" class="ltx_para">
        
<p class="ltx_p">Each of these local sweeps will be associated with the haplotype on which the particular allele arose.
Under the parameter regime we study,
standing variants are still quite young,
so we do not expect a strongly reduced hitchhiking effect.
As such, following the initial period of
adaptation, we should expect the population to be partitioned into a
set of geographically restricted long haplotypes.
Given sufficient time these haplotypes will mix together through
migration and drift,
potentially leading to a <span class="ltx_text ltx_font_italic" style="color:#FF0000;">within population</span> signal of a sweep from multiple independent
mutations if our selected allele occurs at the same locus <cite class="ltx_cite">(Pennings and Hermisson, <a href="#bib.bib115" title="Soft sweeps III: the signature of positive selection from recurrent mutation" class="ltx_ref">2006</a>)</cite>,
or to multiple partial sweeps if the loci are scattered across the
genome <cite class="ltx_cite">(Coop and Ralph, <a href="#bib.bib11" title="Patterns of neutral diversity under general models of selective sweeps" class="ltx_ref">2012</a>)</cite>.</p>
      
</div>
      
<div id="S2.SS7.SSS0.Px1.p4" class="ltx_para">
        
<p class="ltx_p">Our results are predicated on the idea that adaptive variants are
initially rare within populations, i.e.¬†they are reasonably
deleterious before the environment switches. <span class="ltx_text ltx_font_italic" style="color:#FF0000;"> In contrast, if they adapt via common variation even distant populations could have
a shared basis of adaptation, e.g.¬†previously neutral (or nearly neutral) variation, shared among populations.
If many loci contribute to variation in a trait, then
selection on any one allele may be weak,
which might lead adaptation to use the same alleles in different populations.
However, sufficiently differentiated populations may still
adapt via different genetic routes, as the constellation of alleles
that respond to selection quickest will be somewhat different due to
drift among populations</span> <cite class="ltx_cite">(Barton, <a href="#bib.bib30" title="The divergence of a polygenic system subject to stabilizing selection, mutation and drift" class="ltx_ref">1989</a>)</cite>.
Therefore, it may be the case that polygenic traits are even less susceptible to a shared
genetic basis to adaptation across populations than simple traits.
However, we currently lack good models and methods with which to test this.
</p>
      
</div>
      
    
</div>
    
<div id="S2.SS7.SSS0.Px2" class="ltx_paragraph">
      
<h5 class="ltx_title ltx_title_paragraph">Are species held together by widespread selective sweeps?</h5>

      
<div id="S2.SS7.SSS0.Px2.p1" class="ltx_para">
        
<p class="ltx_p">Our results touch on an old debate on the evolutionary coherence of species.
Mayr and many others have argued that species are coherent
evolutionary units because they are united by shared gene flow <cite class="ltx_cite">(pages 521‚Äì522 in Mayr, <a href="#bib.bib68" title="Animal species and evolution" class="ltx_ref">1963</a>)</cite>.
However, this argument has been questioned by a number of authors
based on relatively high levels of differentiation, and low rates of dispersal, in many species <cite class="ltx_cite">(Ehrlich and Raven, <a href="#bib.bib73" title="Differentiation of populations" class="ltx_ref">1969</a>; Levin, <a href="#bib.bib75" title="The nature of plant species" class="ltx_ref">1979</a>)</cite>.
Even if gene flow is not high enough to prevent neutral differentiation or local adaptation,
a number of authors have argued that species are cohesive
if gene flow is high enough for globally selected alleles
(and their hitchhiking haplotypes)
to spread across entire species
<cite class="ltx_cite">(see also Rieseberg and Burke, <a href="#bib.bib85" title="The biological reality of species: gene flow, selection, and collective evolution" class="ltx_ref">2001</a>; Morjan and Rieseberg, <a href="#bib.bib100" title="How species evolve collectively: implications of gene flow and selection for the spread of advantageous alleles" class="ltx_ref">2004</a>; Ellstrand, <a href="#bib.bib74" title="Is gene flow the most important evolutionary force in plants?" class="ltx_ref">2014</a>)</cite>.
At present, large scale genotyping and sequencing projects,
along with more sophisticated methods,
are highlighting ever more signals of gene flow between populations and species <cite class="ltx_cite">(Patterson<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib23" title="Ancient admixture in human history" class="ltx_ref">2012</a>; Sousa and Hey, <a href="#bib.bib22" title="Understanding the origin of species with genome-scale data: modelling gene flow" class="ltx_ref">2013</a>; Hellenthal<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib24" title="A genetic atlas of human admixture history" class="ltx_ref">2014</a>)</cite>.
However, our work on geographic convergent adaptation
<cite class="ltx_cite">(see also Ralph and Coop, <a href="#bib.bib120" title="Parallel adaptation: one or many waves of advance of an advantageous allele?" class="ltx_ref">2010</a>, <a href="#bib.bib76" title="Convergent evolution during local adaptation to patchy landscapes" class="ltx_ref">2014</a>)</cite>
suggests that species should often adapt to
widespread selection pressures through convergent evolution rather
than waiting for a single allele to migrate across the range.</p>
      
</div>
      
      
<div id="S2.SS7.SSS0.Px2.p2" class="ltx_para">
        
<p class="ltx_p">In support of this idea, putative recent selective sweeps seem to often be geographically restricted <cite class="ltx_cite">(Pickrell<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib26" title="Signals of recent positive selection in a worldwide sample of human populations" class="ltx_ref">2009</a>; Coop<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib25" title="The role of geography in human adaptation" class="ltx_ref">2009</a>; Granka<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib29" title="Limited evidence for classic selective sweeps in African populations" class="ltx_ref">2012</a>)</cite>,
rather than species-wide <cite class="ltx_cite">(but see Clark<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib27" title="Common sequence polymorphisms shaping genetic diversity in Arabidopsis thaliana" class="ltx_ref">2007</a>; Long<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib28" title="Massive genomic variation and strong selection in Arabidopsis thaliana lines from Sweden" class="ltx_ref">2013</a>,  for a potential example)</cite>.
This is likely in part due to the relatively low incidence of widespread selection pressures,
but as noted above even when we know of widespread selection
pressures (e.g.¬†malaria) the response is usually convergent, not shared, across large spatial scales.
On the other hand,
introgression of adaptive alleles across species and sub-species boundaries,
suggests that selected alleles do sometimes spread despite low
migration rates <cite class="ltx_cite">(see  Hedrick, <a href="#bib.bib54" title="Adaptive introgression in animals: examples and comparison to new mutation and standing variation as sources of adaptive variation" class="ltx_ref">2013</a>,  for a recent review)</cite>.
However, at least some of these cases
may be caused by introgression of haplotype complexes
consisting of many, tightly linked, beneficial alleles (that perhaps
are inaccessible by mutation over reasonable time-scales for a population in a new
environment).
Currently we can only scan genomes for species-wide sweeps
in those few organisms with population-scale sequence data,
and so we do not know if these observations generalize to most species.
This is rapidly changing,
and will allow us to form a much improved picture of the relationship
between the level of neutral population structure,
and the age and geographic spread of selected alleles across many species.
</p>
      
</div>
      
      
<div id="S2.SS7.SSS0.Px2.p3" class="ltx_para">
        
<p class="ltx_p">Even if selective sweeps only bring alleles to fixation locally,
they are still potentially a stronger homogenizing force
than neutral mixing through migration.
Under neutral mixing, the mean number of generations back to the most recent common ancestor
is on order of the total effective population size.
This quantity has not been worked out for a model with simultaneous local sweeps,
but will be somewhat analogous to the ‚Äúspatial <math xmlns="http://www.w3.org/1998/Math/MathML" id="S2.SS7.SSS0.Px2.p3.m1" class="ltx_Math" alttext="\Lambda" display="inline"><mi mathvariant="normal">Œõ</mi></math>-Fleming‚ÄìViot‚Äù models of <cite class="ltx_cite">Barton<span class="ltx_text ltx_bib_etal"> et al.</span> (<a href="#bib.bib179" title="Modelling evolution in a spatial continuum" class="ltx_ref">2013a</a>)</cite>,
in which local sweeps occur independently across the range.
Lineages that are <span class="ltx_text ltx_font_italic" style="color:#0000FF;">closely linked to the sweeping allele</span> (<math xmlns="http://www.w3.org/1998/Math/MathML" id="S2.SS7.SSS0.Px2.p3.m2" class="ltx_Math" alttext="\sim v/\chi" display="inline"><mrow><mi></mi><mo>‚àº</mo><mrow><mi>v</mi><mo>/</mo><mi>œá</mi></mrow></mrow></math> Morgans) will be moved
towards the center of the sweep (a displacement <math xmlns="http://www.w3.org/1998/Math/MathML" id="S2.SS7.SSS0.Px2.p3.m3" class="ltx_Math" alttext="O(\chi)" display="inline"><mrow><mi>O</mi><mo>‚Å¢</mo><mrow><mo stretchy="false">(</mo><mi>œá</mi><mo stretchy="false">)</mo></mrow></mrow></math>),
and pairs of lineages caught up in the same sweep could be forced to coalesce
<cite class="ltx_cite">(see  Barton<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib106" title="Genetic hitchhiking in spatially extended populations" class="ltx_ref">2013b</a>, for work on geographic
hitchhiking)</cite>. In this case lineages and alleles are literally
hitchhiking across space.
The overall rate of lineage movement and coalescence
depends on the rate of sweeps, their geographic scale, and the rate of recombination,
and could be calculated by combining the result presented here with
<cite class="ltx_cite">Barton<span class="ltx_text ltx_bib_etal"> et al.</span> (<a href="#bib.bib179" title="Modelling evolution in a spatial continuum" class="ltx_ref">2013a</a>)</cite> and <cite class="ltx_cite">Barton<span class="ltx_text ltx_bib_etal"> et al.</span> (<a href="#bib.bib106" title="Genetic hitchhiking in spatially extended populations" class="ltx_ref">2013b</a>)</cite>.
However, if geographic sweeps are common
then this may substantially speed up the rate of mixing compared to
neutral drift and migration.</p>
      
</div>
      
    
</div>
    
<div id="S2.SS7.SSS0.Px3" class="ltx_paragraph">
      
<h5 class="ltx_title ltx_title_paragraph">How then do substitutions occur?</h5>

      
<div id="S2.SS7.SSS0.Px3.p1" class="ltx_para">
        
<p class="ltx_p">If it is rare for gene flow to rapidly spread selected alleles across a species range,
how then do selected alleles ever become fixed within species?
Drift alone will act only slowly to sort variants within species into
divergence among species.
<span class="ltx_text ltx_font_italic" style="color:#7303EB;">Of course, effective population sizes estimated from genetic diversity
are nearly always small enough that fixation of neutral alleles by drift
is expected to take less time than the divergence time between species.
However, this observation is somewhat circular,
and as reviewed by <cite class="ltx_cite">Leffler<span class="ltx_text ltx_bib_etal"> et al.</span> (<a href="#bib.bib182" title="Revisiting an old riddle: What determines genetic diversity levels within species?" class="ltx_ref">2012</a>)</cite>,
we do not understand why the implied ‚Äúeffective population sizes‚Äù differ so much
from census population sizes;
selective processes (as described here) are likely part of the explanation <cite class="ltx_cite">(Corbett-Detig<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib183" title="Natural selection constrains neutral diversity across A wide range of species" class="ltx_ref">2015</a>)</cite>.
</span>
<span class="ltx_text ltx_font_italic" style="color:#0000FF;">Slight selective differences in the pleiotropic effects (or linked
background) among convergent alleles could allow one allele to
press into areas occupied by other alleles.</span>
Furthermore, repeated bouts of adaptation in particular genomic regions may act
to push a subset of previously selected alleles to fixation across the
species range, through the spread of the genetic backgrounds on which
they arise.
However, it seems likely that this is a slow process compared to the
initial rapid spread of selected alleles.</p>
      
</div>
    
</div>
    
<div id="S2.SS7.SSS0.Px4" class="ltx_paragraph">
      
<h5 class="ltx_title ltx_title_paragraph">Speciation and extinction as phases of molecular evolution?</h5>

      
<div id="S2.SS7.SSS0.Px4.p1" class="ltx_para">
        
<p class="ltx_p">One potential resolution is that many selected alleles achieve
fixation, not through their own species-wide spread, but rather
through subsequent large-scale changes in geographic range size
induced by extirpation of the species over parts of its range
<cite class="ltx_cite">(see  Barton<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib179" title="Modelling evolution in a spatial continuum" class="ltx_ref">2013a</a>, and references therein for how such a model could be constructed)</cite>.
Such drops in range size may fix, or radically change the species-wide frequency,
of alleles previously restricted to small portion of a species range.
Furthermore, many modes of speciation are proposed to occur through a
geographically-limited subset of populations forming the basis of new species,
e.g.¬†the splitting off of part of the range through a vicariance event
or dispersal of a subset of individuals.
In this case, speciation will cause geographic assortment of
polymorphic ancestral variation, again acting to fix variants within
newly formed species that were previously polymorphic across ancestral species ranges.</p>
      
</div>
      
      
<div id="S2.SS7.SSS0.Px4.p2" class="ltx_para">
        
<p class="ltx_p">Such ideas are not completely new and represent a perhaps logical consequence of
an allopatric or parapatric view of the biogeography of speciation.
However, it is worth revisiting this idea as geographically broad population genomic
sampling allows us to return to themes
in biogeography.
Along similar lines, Futuyma has argued that much of the adaptive differentiation within species,
e.g.¬†adaptation to local conditions,
may be ephemeral and subject to loss due to local extinction and the mixing
following the collapse of population structure <cite class="ltx_cite">(Futuyma, <a href="#bib.bib84" title="Evolutionary constraint and ecological consequences" class="ltx_ref">2010</a>, <a href="#bib.bib82" title="On the role of species in anagenesis" class="ltx_ref">1987</a>)</cite>.
Futuyma offered this as an explanation of the pattern of punctuated equilibrium <cite class="ltx_cite">(Eldredge and Gould, <a href="#bib.bib67" title="Punctuated equilibria: an alternative to phyletic gradualism" class="ltx_ref">1972</a>)</cite>,
and argued that the observation of stasis and rapid anagenesis associated with speciation
were consistent with micro-evolution.
Futuyma argued that despite rapid adaptation over short time-scales, we
may observe morphological stasis in the fossil record as much of this
adaptation is lost to local extinction and the collapse of population structure
<cite class="ltx_cite">(see also Lieberman and Dudgeon, <a href="#bib.bib47" title="An evaluation of stabilizing selection as a mechanism for stasis" class="ltx_ref">1996</a>; Eldredge<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib81" title="The dynamics of evolutionary stasis" class="ltx_ref">2005</a>; Futuyma, <a href="#bib.bib84" title="Evolutionary constraint and ecological consequences" class="ltx_ref">2010</a>)</cite>.
Furthermore, he suggests that speciation
may act as ratchet to prevent the loss of differentiation, acting to
maintain adaptive changes among populations, and prevent their loss by interbreeding.
At face value the rate of species formation seems too low to contribute to this process.
However, <cite class="ltx_cite">Rosenblum<span class="ltx_text ltx_bib_etal"> et al.</span> (<a href="#bib.bib83" title="Goldilocks Meets Santa Rosalia: An Ephemeral Speciation Model Explains Patterns of Diversification Across Time Scales" class="ltx_ref">2012</a>)</cite>, and many others,
have argued that the rate of speciation may well be quite high,
but that the majority of incipient species do not persist long due to
reabsorption or extinction.
Changes in range size, due to local extinction, can also be very rapid
on the time-scales over which alleles may spread on the landscape <cite class="ltx_cite">(Gaston, <a href="#bib.bib10" title="The structure and dynamics of geographic ranges" class="ltx_ref">2003</a>; Hewitt, <a href="#bib.bib9" title="Some genetic consequences of ice ages, and their role in divergence and speciation" class="ltx_ref">1996</a>)</cite>.
Repeated bouts of extinction and speciation will send waves of alleles
to fixation along particular lineages.
</p>
      
</div>
      
      
<div id="S2.SS7.SSS0.Px4.p3" class="ltx_para">
        
<p class="ltx_p">Such a link between speciation and substitution would not imply that
substitutions should necessarily be thought of as being clustered at splits in inferred phylogenies
<cite class="ltx_cite">(see  Pennell<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib71" title="Is there room for punctuated equilibrium in macroevolution?" class="ltx_ref">2014a</a>; Venditti and Pagel, <a href="#bib.bib70" title="Plenty of room for punctuational change" class="ltx_ref">2014</a>; Pennell<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib69" title="Speciation is unlikely to drive divergence rates" class="ltx_ref">2014b</a>, for a recent exchange on this)</cite>.
Neutral substitutions are unaffected by this process,
because they accumulate in a clocklike manner along lineages,
as dictated by the mutation rate,
regardless of the geographic details of their polymorphic stage.
Turning to the accumulation of adaptive substitutions,
it is likely that splits in phylogenies are only a tiny proportion of all incipient speciation events,
because extinction rates may be high <cite class="ltx_cite">(Rosenblum<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib83" title="Goldilocks Meets Santa Rosalia: An Ephemeral Speciation Model Explains Patterns of Diversification Across Time Scales" class="ltx_ref">2012</a>)</cite>,
and so every lineage has likely passed through many ‚Äúspeciation events‚Äù in addition to the observed ones.
Under these assumptions,
spatial polymorphisms could accumulate gradually in geographically restricted populations
between the large-scale biogeographic events that cause their fixation or loss.
This effectively decorrelates the time at which new alleles arise and when they fix in the species,
an effect similar to that pointed out by <cite class="ltx_cite">Gillespie (<a href="#bib.bib56" title="Substitution processes in molecular evolution. II. Exchangeable models from population genetics" class="ltx_ref">1994</a>)</cite>.</p>
      
</div>
      
      
<div id="S2.SS7.SSS0.Px4.p4" class="ltx_para">
        
<p class="ltx_p">This view would not imply that adaptive evolution or speciation is driven by the
shifting balance or genetic revolutions
<cite class="ltx_cite">(Wright, <a href="#bib.bib66" title="The roles of mutation, inbreeding, crossbreeding and selection in evolution." class="ltx_ref">1932</a>; Mayr, <a href="#bib.bib65" title="Evolution as a process" class="ltx_ref">1954</a>)</cite>,
whereby genetic drift allows
populations to cross fitness valleys and substitute novel epistatic combinations.
Although geographic lineage sorting via speciation and extinction
can be thought of as very large-scale genetic drift events,
in the models we study here the initial spread of alleles is due to selection, not drift
<cite class="ltx_cite">(see also  Futuyma, <a href="#bib.bib55" title="Speciational trends and the role of species in macroevolution" class="ltx_ref">1989</a>, for discussion)</cite>.
</p>
      
</div>
      
      
<div id="S2.SS7.SSS0.Px4.p5" class="ltx_para">
        
<p class="ltx_p">There is evidence that a reasonable fraction of genome-wide substitutions are fixed by positive selection in a number of species
<cite class="ltx_cite">(most notably Drosophila,  Sella<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib15" title="Pervasive natural selection in the Drosophila genome?" class="ltx_ref">2009</a>)</cite>.
Under the geographic view of fixation,
selection has played a strong role in the establishment of these alleles locally.
As we get more broadly geographic population genomics sampling for a range of species
we will have the opportunity to study whether the class of alleles that contribute to local differentiation
are similar to those underlying species divergence,
and the extent to which the answer to this depends on the
age and type of population structure within species.</p>
      
</div>
      
<div id="S2.SS7.SSS0.Px4.p6" class="ltx_para">
        
<p class="ltx_p">Finally, we close by noting that range expansion and speciation
are obviously not separate from adaptive differentiation.
The invasion of new geographic areas may lead to a burst of adaptive differentiation,
at least in a subset of genes, and speciation may be associated with rapidly adaption to novel environments.
Conversely, if the geographic spread of adaptive alleles within ranges is slow
<span class="ltx_text ltx_font_italic" style="color:#FF0000;">(e.g.¬†if they only offer a local advantage or if they are selectively excluded)</span>
this may allow Dobzhansky-Muller incompatibilities to arise within species,
effectively offering a mechanism for hybrid incompatibilities to evolve in parapatry,
and fracturing the range <cite class="ltx_cite">(Bank<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib42" title="The limits to parapatric speciation: Dobzhansky-Muller incompatibilities in a continent-island model" class="ltx_ref">2012</a>; Kondrashov, <a href="#bib.bib99" title="Accumulation of Dobzhansky-Muller incompatibilities within a spatially structured population" class="ltx_ref">2003</a>; Bierne<span class="ltx_text ltx_bib_etal"> et al.</span>, <a href="#bib.bib16" title="The coupling hypothesis: why genome scans may fail to map local adaptation genes" class="ltx_ref">2011</a>)</cite>.
The alleles that act as components in many of the Dobzhansky-Muller incompatibilities studied to date
are geographically restricted <cite class="ltx_cite">(see Cutter, <a href="#bib.bib80" title="The polymorphic prelude to Bateson-Dobzhansky-Muller incompatibilities" class="ltx_ref">2012</a>)</cite>.
Therefore, it seems possible that populations within species may often be tending towards speciation,
and that as outlined here that this may drive some proportion of molecular divergence.</p>
      
</div>
    
</div>
    
<div id="S2.SS7.SSSx1" class="ltx_subsubsection">
      
<h4 class="ltx_title ltx_title_subsubsection">Acknowledgements</h4>

      
<div id="S2.SS7.SSSx1.p1" class="ltx_para">
        
<p class="ltx_p">We thank Michael Turelli, Jon Seger, and the Coop lab for helpful
conversations. We thank Florence D√©barre, Douglas Futuyma, Michael Whitlock, Molly Przeworski, and Sam Yeaman for helpful comments on an
earlier version of the paper. All figures and calculations were
performed in R <cite class="ltx_cite">(R Core Team, <a href="#bib.bib1" title="R: A language and environment for statistical computing" class="ltx_ref">2014</a>)</cite>. The figure color paletes were generated using
Karthik Ram‚Äôs ‚ÄúWes Anderson‚Äù R color palette <a href="https://github.com/karthik/wesanderson" title="" class="ltx_ref ltx_url"><span class="ltx_text ltx_font_typewriter">https://github.com/karthik/wesanderson</span></a>.</p>
      
</div>
    
</div>
  
</div>
  
<div id="bib" class="ltx_bibliography">
    
<h2 class="ltx_title ltx_title_bibliography">References</h2>

  
<ul id="L1" class="ltx_biblist">
<li id="bib.bib101" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. Arendt and D. Reznick (2008)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Convergence and parallelism reconsidered: what have we learned about the genetics of adaptation?</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Trends Ecol Evol</span> <span class="ltx_text ltx_bib_volume">23</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†26‚Äì32</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_ref ltx_bib_external ltx_ref_self">Link</span>,
<a href="http://dx.doi.org/10.1016/j.tree.2007.09.011" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p1" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib58" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">D. Aronson and H. Weinberger (1975)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Nonlinear diffusion in population genetics, combustion, and nerve pulse propagation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Partial Differential Equations and Related Topics</span>, <span class="ltx_text ltx_bib_pages"> pp.¬†5‚Äì49</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1007/BFb0070595" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p3" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib42" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">C. Bank, R. B√ºrger and J. Hermisson (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The limits to parapatric speciation: Dobzhansky-Muller incompatibilities in a continent-island model</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">191</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†845‚Äì863</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1534/genetics.111.137513" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://www.genetics.org/content/191/3/845.abstract" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text ltx_bib_external">http://www.genetics.org/content/191/3/845.full.pdf+html</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p6" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib179" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">N. H. Barton, A. M. Etheridge and A. V√©ber (2013a)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Modelling evolution in a spatial continuum</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Journal of Statistical Mechanics: Theory and
Experiment</span> <span class="ltx_text ltx_bib_volume">2013</span> (<span class="ltx_text ltx_bib_number">01</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†P01002</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://stacks.iop.org/1742-5468/2013/i=01/a=P01002" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p3" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>,
<a href="#S2.SS7.SSS0.Px4.p1" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib106" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">N. H. Barton, A. M. Etheridge, J. Kelleher and A. V√©ber (2013b)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Genetic hitchhiking in spatially extended populations</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Theoretical Population Biology</span> <span class="ltx_text ltx_bib_volume">87</span>, <span class="ltx_text ltx_bib_pages"> pp.¬†75‚Äì89</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1016/j.tpb.2012.12.001" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 0040-5809</span>,
<a href="http://www.sciencedirect.com/science/article/pii/S0040580912001359" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p3" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib30" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">N. Barton (1989)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The divergence of a polygenic system subject to stabilizing selection, mutation and drift</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics Research</span> <span class="ltx_text ltx_bib_volume">54</span>, <span class="ltx_text ltx_bib_pages"> pp.¬†59‚Äì78</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 1469-5073</span>,
<a href="http://dx.doi.org/10.1017/S0016672300028378" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://journals.cambridge.org/article_S0016672300028378" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px1.p4" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib46" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. J. Berg and G. Coop (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">A population genetic signal of polygenic adaptation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS genetics</span> <span class="ltx_text ltx_bib_volume">10</span> (<span class="ltx_text ltx_bib_number">8</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†e1004412</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px1.p1" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib16" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">N. Bierne, J. Welch, E. Loire, F. Bonhomme and P. David (2011)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The coupling hypothesis: why genome scans may fail to map local adaptation genes</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Molecular Ecology</span> <span class="ltx_text ltx_bib_volume">20</span> (<span class="ltx_text ltx_bib_number">10</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†2044‚Äì2072</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px1.p2" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>,
<a href="#S2.SS7.SSS0.Px4.p6" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib57" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">B. Bollob√°s and O. Riordan (2008)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Percolation on random Johnson-Mehl tessellations and related models</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Probab. Theory Related Fields</span> <span class="ltx_text ltx_bib_volume">140</span> (<span class="ltx_text ltx_bib_number">3-4</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†319‚Äì343</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://arxiv.org/abs/math/0610716" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px3.p2" title="Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib48" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">A. D. Bradshaw (1991)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The Croonian lecture, 1991: Genostasis and the limits to evolution</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Philosophical Transactions: Biological Sciences</span> <span class="ltx_text ltx_bib_volume">333</span> (<span class="ltx_text ltx_bib_number">1267</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†pp. 289‚Äì305</span> (<span class="ltx_text ltx_bib_language">English</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.jstor.org/stable/55615" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 09628436</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p2" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib37" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">L. M. Chevin, G. Martin and T. Lenormand (2010)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Fisher‚Äôs model and the genomics of adaptation: restricted pleiotropy, heterogenous mutation, and parallel evolution</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Evolution</span> <span class="ltx_text ltx_bib_volume">64</span> (<span class="ltx_text ltx_bib_number">11</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†3213‚Äì3231</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS6.p1" title="1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.6</span></a>.
</span></li>
<li id="bib.bib27" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">R. M. Clark, G. Schweikert, C. Toomajian, S. Ossowski, G. Zeller, P. Shinn, N. Warthmann, T. T. Hu, G. Fu, D. A. Hinds, H. Chen, K. A. Frazer, D. H. Huson, B. Scholkopf, M. Nordborg, G. Ratsch, J. R. Ecker and D. Weigel (2007)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Common sequence polymorphisms shaping genetic diversity in <span class="ltx_text ltx_font_italic">Arabidopsis thaliana</span></span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Science</span> <span class="ltx_text ltx_bib_volume">317</span> (<span class="ltx_text ltx_bib_number">5836</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†338‚Äì342</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p2" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib8" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">G. L. Conte, M. E. Arnegard, C. L. Peichel and D. Schluter (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The probability of genetic parallelism and convergence in natural populations</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Proceedings of the Royal Society of London B:
Biological Sciences</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1098/rspb.2012.2146" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<span class="ltx_text isbn ltx_bib_external">ISBN 1471-2954</span>,
<span class="ltx_text issn ltx_bib_external">ISSN 0962-8452</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p1" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib25" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">G. Coop, J. K. Pickrell, J. Novembre, S. Kudaravalli, J. Li, D. Absher, R. M. Myers, L. L. Cavalli-Sforza, M. W. Feldman and J. K. Pritchard (2009)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The role of geography in human adaptation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS Genet</span> <span class="ltx_text ltx_bib_volume">5</span> (<span class="ltx_text ltx_bib_number">6</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19503611" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/10.1371/journal.pgen.1000500" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p2" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib11" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">G. Coop and P. Ralph (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Patterns of neutral diversity under general models of selective sweeps</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">192</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†205‚Äì224</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px1.p3" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib183" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">R. B. Corbett-Detig, D. L. Hartl and T. B. Sackton (2015)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Natural selection constrains neutral diversity across A wide range of species</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS Biol</span> <span class="ltx_text ltx_bib_volume">13</span> (<span class="ltx_text ltx_bib_number">4</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†e1002112</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1371/journal.pbio.1002112" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1002112" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px3.p1" title="How then do substitutions occur? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib181" class="ltx_bibitem ltx_bib_book"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">D. R. Cox and V. Isham (1980)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Point processes</span>.
</span>
<span class="ltx_bibblock"> <span class="ltx_text ltx_bib_publisher">Chapman &amp; Hall</span>, <span class="ltx_text ltx_bib_place">London</span>.
</span>
<span class="ltx_bibblock">Note: <span class="ltx_text ltx_bib_note">Monographs on Applied Probability and Statistics</span>
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text isbn ltx_bib_external">ISBN 0-412-21910-7</span>,
<a href="http://www.ams.org/mathscinet-getitem?mr=598033" title="" class="ltx_ref mr ltx_bib_external">MathReview (D. Vere-Jones)</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS5.p1" title="1.5 The contribution of standing variation. ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.5</span></a>.
</span></li>
<li id="bib.bib80" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">A. D. Cutter (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The polymorphic prelude to Bateson-Dobzhansky-Muller incompatibilities</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Trends Ecol. Evol. (Amst.)</span> <span class="ltx_text ltx_bib_volume">27</span> (<span class="ltx_text ltx_bib_number">4</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†209‚Äì218</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p6" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib18" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">M. Edwards, A. Bigham, J. Tan, S. Li, A. Gozdzik, K. Ross, L. Jin and E. J. Parra (2010)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Association of the <span class="ltx_text ltx_font_italic">OCA2</span> polymorphism His615Arg with melanin content in east Asian populations: further evidence of convergent evolution of skin pigmentation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS Genet</span> <span class="ltx_text ltx_bib_volume">6</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†e1000867</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1371%2Fjournal.pgen.1000867" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/10.1371/journal.pgen.1000867" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px1.p1" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib73" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. R. Ehrlich and P. H. Raven (1969)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Differentiation of populations</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Science</span> <span class="ltx_text ltx_bib_volume">165</span> (<span class="ltx_text ltx_bib_number">3899</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1228‚Äì1232</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1126/science.165.3899.1228" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://www.sciencemag.org/content/165/3899/1228.short" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text ltx_bib_external">http://www.sciencemag.org/content/165/3899/1228.full.pdf</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib81" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">N. Eldredge, J. Thompson, P. Brakefield, S. Gavrilets, D. Jablonski, J. Jackson, R. Lenski, B. Lieberman, M. McPeek and W. Miller (2005)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The dynamics of evolutionary stasis</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Paleobiology</span> <span class="ltx_text ltx_bib_volume">31</span> (<span class="ltx_text ltx_bib_number">2, S</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†133‚Äì145</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1666/0094-8373(2005)031[0133:TDOES]2.0.CO;2" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 0094-8373</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p2" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib67" class="ltx_bibitem ltx_bib_inproceedings"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">N. Eldredge and S. Gould (1972)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Punctuated equilibria: an alternative to phyletic gradualism</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p2" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib74" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">N. C. Ellstrand (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Is gene flow the most important evolutionary force in plants?</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">American journal of botany</span> <span class="ltx_text ltx_bib_volume">101</span> (<span class="ltx_text ltx_bib_number">5</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†737‚Äì753</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib60" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">M. Fanfoni and M. Tomellini (1998)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The Johnson-Mehl-Avrami-Kolmogorov model: A brief review</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Il Nuovo cimento. D</span> <span class="ltx_text ltx_bib_volume">20</span> (<span class="ltx_text ltx_bib_number">7-8</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1171‚Äì1182</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px3.p2" title="Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib177" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">R. Fisher (1937)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The wave of advance of advantageous genes</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Ann. Eugenics</span> <span class="ltx_text ltx_bib_volume">7</span>, <span class="ltx_text ltx_bib_pages"> pp.¬†353‚Äì369</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://digital.library.adelaide.edu.au/coll/special/fisher/152.pdf" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px2.p1" title="Geographic spread of alleles ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>,
<a href="#S1.SS2.p3" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib59" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. Flint, R. M. Harding, A. J. Boyce and J. B. Clegg (1998)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The population genetics of the haemoglobinopathies</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Baillieres Clin Haematol</span> <span class="ltx_text ltx_bib_volume">11</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1‚Äì51</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10872472" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib84" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">D. J. Futuyma (2010)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Evolutionary constraint and ecological consequences</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Evolution</span> <span class="ltx_text ltx_bib_volume">64</span> (<span class="ltx_text ltx_bib_number">7</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1865‚Äì1884</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p2" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib82" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">D. Futuyma (1987)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">On the role of species in anagenesis</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">American Naturalist</span> <span class="ltx_text ltx_bib_volume">130</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†465‚Äì473</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1086/284724" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 0003-0147</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p2" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib55" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">D. J. Futuyma (1989)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Speciational trends and the role of species in macroevolution</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">The American Naturalist</span> <span class="ltx_text ltx_bib_volume">134</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†pp. 318‚Äì321</span> (<span class="ltx_text ltx_bib_language">English</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.jstor.org/stable/2462106" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 00030147</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p4" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib10" class="ltx_bibitem ltx_bib_book"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">K. J. Gaston (2003)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The structure and dynamics of geographic ranges</span>.
</span>
<span class="ltx_bibblock"> <span class="ltx_text ltx_bib_publisher">Oxford University Press</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p2" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib61" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">E. N. Gilbert (1962)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Random subdivisions of space into crystals</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Ann. Math. Statist.</span> <span class="ltx_text ltx_bib_volume">33</span>, <span class="ltx_text ltx_bib_pages"> pp.¬†958‚Äì972</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://projecteuclid.org/euclid.aoms/1177704464" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 0003-4851</span>,
<a href="http://www.ams.org/mathscinet-getitem?mr=0144253" title="" class="ltx_ref mr ltx_bib_external">MathReview (H. Bauer)</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px3.p2" title="Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib56" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. H. Gillespie (1994)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Substitution processes in molecular evolution. II. Exchangeable models from population genetics</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Evolution</span> <span class="ltx_text ltx_bib_volume">48</span> (<span class="ltx_text ltx_bib_number">4</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†pp. 1101‚Äì1113</span> (<span class="ltx_text ltx_bib_language">English</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.jstor.org/stable/2410370" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 00143820</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p3" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib3" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">I. Gordo and P. R. A. Campos (2006)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Adaptive evolution in a spatially structured asexual population</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetica</span> <span class="ltx_text ltx_bib_volume">127</span> (<span class="ltx_text ltx_bib_number">1-3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†217‚Äì229</span> (<span class="ltx_text ltx_bib_language">eng</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 0016-6707</span>,
<a href="http://dx.doi.org/10.1007/s10709-005-4012-9" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p4" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib29" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. M. Granka, B. M. Henn, C. R. Gignoux, J. M. Kidd, C. D. Bustamante and M. W. Feldman (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Limited evidence for classic selective sweeps in African populations</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">192</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1049‚Äì1064</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p2" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib20" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. B. S. Haldane (1927)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">A mathematical theory of natural and artificial selection, part V: Selection and mutation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Mathematical Proceedings of the Cambridge
Philosophical Society</span> <span class="ltx_text ltx_bib_volume">23</span>, <span class="ltx_text ltx_bib_pages"> pp.¬†838‚Äì844</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 1469-8064</span>,
<a href="http://dx.doi.org/10.1017/S0305004100015644" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://journals.cambridge.org/article_S0305004100015644" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px1.p2" title="Rates of origination of standing and new mutations ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib21" class="ltx_bibitem ltx_bib_inbook"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. B. S. Haldane (1932)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The causes of evolution</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_pages"> pp.¬†75‚Äì76</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p2" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib19" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. B. S. Haldane (1937)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The effect of variation of fitness</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">The American Naturalist</span> <span class="ltx_text ltx_bib_volume">71</span> (<span class="ltx_text ltx_bib_number">735</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†pp. 337‚Äì349</span> (<span class="ltx_text ltx_bib_language">English</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.jstor.org/stable/2457289" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 00030147</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px1.p2" title="Rates of origination of standing and new mutations ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib4" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">O. Hallatschek and D. S. Fisher (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Acceleration of evolutionary spread by long-range dispersal</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Proceedings of the National Academy of Sciences</span> <span class="ltx_text ltx_bib_volume">111</span> (<span class="ltx_text ltx_bib_number">46</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†E4911‚ÄìE4919</span> (<span class="ltx_text ltx_bib_language">en</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 0027-8424, 1091-6490</span>,
<a href="http://dx.doi.org/10.1073/pnas.1404663111" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.p3" title="1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib54" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. W. Hedrick (2013)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Adaptive introgression in animals: examples and comparison to new mutation and standing variation as sources of adaptive variation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Molecular ecology</span> <span class="ltx_text ltx_bib_volume">22</span> (<span class="ltx_text ltx_bib_number">18</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†4606‚Äì4618</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p2" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib13" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. Hedrick (2011)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Population genetics of malaria resistance in humans.</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Heredity</span> <span class="ltx_text ltx_bib_volume">107</span> (<span class="ltx_text ltx_bib_number">4</span>).
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>,
<a href="#S1.SS2.p3" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib24" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">G. Hellenthal, G. B. Busby, G. Band, J. F. Wilson, C. Capelli, D. Falush and S. Myers (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">A genetic atlas of human admixture history</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Science</span> <span class="ltx_text ltx_bib_volume">343</span> (<span class="ltx_text ltx_bib_number">6172</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†747‚Äì751</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib116" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. Hermisson and P. S. Pennings (2005)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Soft sweeps: molecular population genetics of adaptation from standing genetic variation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">169</span> (<span class="ltx_text ltx_bib_number">4</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†2335‚Äì2352</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15716498" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/10.1534/genetics.104.036947" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS7.p5" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>,
<a href="#S1.p3" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>,
<a href="#S2.p2" title="2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib9" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">G. M. Hewitt (1996)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Some genetic consequences of ice ages, and their role in divergence and speciation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Biological Journal of the Linnean Society</span> <span class="ltx_text ltx_bib_volume">58</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†247‚Äì276</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 0024-4066</span>,
<a href="http://dx.doi.org/http://dx.doi.org/10.1006/bijl.1996.0035" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p2" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib87" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">R. E. Howes, M. Dewi, F. B. Piel, W. M. Monteiro, K. E. Battle, J. P. Messina, A. Sakuntabhai, A. W. Satyagraha, T. N. Williams, J. K. Baird and S. I. Hay (2013)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Spatial distribution of G6PD deficiency variants across malaria-endemic regions</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Malar. J.</span> <span class="ltx_text ltx_bib_volume">12</span>, <span class="ltx_text ltx_bib_pages"> pp.¬†418</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.F3" title="Figure¬†3 ‚Ä£ 1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">3</span></a>,
<a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>,
<a href="#S1.SS2.p4" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib90" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">R. E. Howes, F. B. Piel, A. P. Patil, O. A. Nyangiri, P. W. Gething, M. Dewi, M. M. Hogg, K. E. Battle, C. D. Padilla, J. K. Baird and S. I. Hay (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">G6PD deficiency prevalence and estimates of affected populations in malaria endemic countries: a geostatistical model-based map</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS Med.</span> <span class="ltx_text ltx_bib_volume">9</span> (<span class="ltx_text ltx_bib_number">11</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†e1001339</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib131" class="ltx_bibitem ltx_bib_book"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. Jagers (1975)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Branching processes with biological applications</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_series">Wiley Series in Probability and Statistics: Applied
Probability and Statistics Section Series</span>,  <span class="ltx_text ltx_bib_publisher">Wiley</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text isbn ltx_bib_external">ISBN 978-0-471-43652-2</span>,
<span class="ltx_text lccn ltx_bib_external">LCCN 74032296</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS7.p2" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>.
</span></li>
<li id="bib.bib32" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">C. Jeong and A. D. Rienzo (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Adaptations to local environments in modern human populations</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Current Opinion in Genetics &amp; Development</span> <span class="ltx_text ltx_bib_volume">29</span> (<span class="ltx_text ltx_bib_number">0</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1‚Äì8</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 0959-437X</span>,
<a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.gde.2014.06.011" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://www.sciencedirect.com/science/article/pii/S0959437X14000641" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p1" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib51" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">T. Karasov, P. W. Messer and D. A. Petrov (2010)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Evidence that adaptation in Drosophila is not limited by mutation at single sites</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS genetics</span> <span class="ltx_text ltx_bib_volume">6</span> (<span class="ltx_text ltx_bib_number">6</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†e1000924</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p2" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib63" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">A. N. Kolmogorov (1937)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">On the statistical theory of metal crystallization</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Izv. Akad. Nauk SSSR, Ser. Math</span> <span class="ltx_text ltx_bib_volume">1</span>, <span class="ltx_text ltx_bib_pages"> pp.¬†335‚Äì360</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px3.p2" title="Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib77" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">A. Kolmogorov, I. Petrovskii and N. Piscunov (1937)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">A study of the equation of diffusion with increase in the quantity of matter, and its application to a biological problem</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Byul. Moskovskogo Gos. Univ.</span> <span class="ltx_text ltx_bib_volume">1</span> (<span class="ltx_text ltx_bib_number">6</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1‚Äì25</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://books.google.com/books?id=ikN59GkYJKIC&amp;lpg=PP1&amp;dq=A.N.%20Kolmogorov%3A%20Selected%20Works&amp;client=firefox-a&amp;pg=PA242#v=onepage&amp;q=&amp;f=false" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px2.p1" title="Geographic spread of alleles ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib99" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">A. S. Kondrashov (2003)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Accumulation of Dobzhansky-Muller incompatibilities within a spatially structured population</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Evolution</span> <span class="ltx_text ltx_bib_volume">57</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†151‚Äì153</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS6.p4" title="1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.6</span></a>,
<a href="#S2.SS7.SSS0.Px4.p6" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib89" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">D. P. Kwiatkowski (2005)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">How malaria has affected the human genome and what human genetics can teach us about malaria</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Am. J. Hum. Genet.</span> <span class="ltx_text ltx_bib_volume">77</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†171‚Äì192</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib40" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">I. Lazaridis, N. Patterson, A. Mittnik, G. Renaud, S. Mallick, P. H. Sudmant, J. G. Schraiber, S. Castellano, K. Kirsanow and C. Economou (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Ancient human genomes suggest three ancestral populations for present-day europeans</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Nature</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px1.p1" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib182" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">E. M. Leffler, K. Bullaughey, D. R. Matute, W. K. Meyer, L. S√©gurel, A. Venkat, P. Andolfatto and M. Przeworski (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Revisiting an old riddle: What determines genetic diversity levels within species?</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS Biol</span> <span class="ltx_text ltx_bib_volume">10</span> (<span class="ltx_text ltx_bib_number">9</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†e1001388</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1371/journal.pbio.1001388" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://dx.doi.org/10.1371%2Fjournal.pbio.1001388" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px3.p1" title="How then do substitutions occur? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib75" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">D. A. Levin (1979)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The nature of plant species</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Science</span> <span class="ltx_text ltx_bib_volume">204</span> (<span class="ltx_text ltx_bib_number">4391</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†pp. 381‚Äì384</span> (<span class="ltx_text ltx_bib_language">English</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.jstor.org/stable/1748373" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 00368075</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib47" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">B. S. Lieberman and S. Dudgeon (1996)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">An evaluation of stabilizing selection as a mechanism for stasis</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Palaeogeography, Palaeoclimatology, Palaeoecology</span> <span class="ltx_text ltx_bib_volume">127</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†229‚Äì238</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p2" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib28" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">Q. Long, F. A. Rabanal, D. Meng, C. D. Huber, A. Farlow, A. Platzer, Q. Zhang, B. J. Vilhjalmsson, A. Korte, V. Nizhynska, V. Voronin, P. Korte, L. Sedman, T. Mandakova, M. A. Lysak, U. Seren, I. Hellmann and M. Nordborg (2013)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Massive genomic variation and strong selection in <span class="ltx_text ltx_font_italic">Arabidopsis thaliana</span> lines from Sweden</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Nat. Genet.</span> <span class="ltx_text ltx_bib_volume">45</span> (<span class="ltx_text ltx_bib_number">8</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†884‚Äì890</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p2" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib94" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">C. Louicharoen, E. Patin, R. Paul, I. Nuchprayoon, B. Witoonpanich, C. Peerapittayamongkol, I. Casademont, T. Sura, N. M. Laird, P. Singhasivanon, L. Quintana-Murci and A. Sakuntabhai (2009)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Positively selected G6PD-Mahidol mutation reduces <span class="ltx_text ltx_font_italic">Plasmodium vivax</span> density in Southeast Asians</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Science</span> <span class="ltx_text ltx_bib_volume">326</span> (<span class="ltx_text ltx_bib_number">5959</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1546‚Äì1549</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib6" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">Malaria Genomic Epidemiology Network (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Reappraisal of known malaria resistance loci in a large multicenter study</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Nat Genet</span> <span class="ltx_text ltx_bib_volume">46</span> (<span class="ltx_text ltx_bib_number">11</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1197‚Äì1204</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 1061-4036</span>,
<a href="http://dx.doi.org/10.1038/ng.3107" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>,
<a href="#S1.SS2.p3" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib7" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">A. Manjurano, N. Sepulveda, B. Nadjm, G. Mtove, H. Wangai, C. Maxwell, R. Olomi, H. Reyburn, E. M. Riley, C. J. Drakeley, T. G. Clark and MalariaGEN Consortium (2015)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">African glucose-6-phosphate dehydrogenase alleles associated with protection from severe malaria in heterozygous females in Tanzania</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS Genet</span> <span class="ltx_text ltx_bib_volume">11</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†e1004960</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1371/journal.pgen.1004960" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/10.1371/journal.pgen.1004960" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>,
<a href="#S1.SS2.p3" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib5" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">E. A. Martens and O. Hallatschek (2011)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Interfering waves of adaptation promote spatial mixing</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">189</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1045‚Äì1060</span> (<span class="ltx_text ltx_bib_language">en</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 0016-6731, 1943-2631</span>,
<a href="http://dx.doi.org/10.1534/genetics.111.130112" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p4" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib33" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">A. Martin and V. Orgogozo (2013)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The Loci of repeated evolution: a catalog of genetic hotspots of phenotypic variation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Evolution</span> <span class="ltx_text ltx_bib_volume">67</span> (<span class="ltx_text ltx_bib_number">5</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1235‚Äì1250</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p1" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib65" class="ltx_bibitem ltx_bib_inbook"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">E. Mayr (1954)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Evolution as a process</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p4" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib68" class="ltx_bibitem ltx_bib_book"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">E. Mayr (1963)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Animal species and evolution</span>.
</span>
<span class="ltx_bibblock"> <span class="ltx_text ltx_bib_publisher">Harvard University Press</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib53" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. W. Messer and D. A. Petrov (2013)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Population genomics of rapid adaptation by soft selective sweeps</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Trends Ecol. Evol. (Amst.)</span> <span class="ltx_text ltx_bib_volume">28</span> (<span class="ltx_text ltx_bib_number">11</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†659‚Äì669</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p3" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib88" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">A. Minucci, K. Moradkhani, M. J. Hwang, C. Zuppi, B. Giardina and E. Capoluongo (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Glucose-6-phosphate dehydrogenase (G6PD) mutations database: review of the ‚Äúold‚Äù and update of the new mutations</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Blood Cells Mol. Dis.</span> <span class="ltx_text ltx_bib_volume">48</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†154‚Äì165</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib100" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">C. L. Morjan and L. H. Rieseberg (2004)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">How species evolve collectively: implications of gene flow and selection for the spread of advantageous alleles</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Mol. Ecol.</span> <span class="ltx_text ltx_bib_volume">13</span> (<span class="ltx_text ltx_bib_number">6</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1341‚Äì1356</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib64" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. M√∏ller (1992)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Random Johnson-Mehl tessellations</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Advances in Applied Probability</span> <span class="ltx_text ltx_bib_volume">24</span> (<span class="ltx_text ltx_bib_number">4</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†814‚Äì844</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.jstor.org/stable/1427714" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px3.p2" title="Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib45" class="ltx_bibitem ltx_bib_inbook"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. M√∏ller (1994)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Lectures on random voronoi tessellations</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_series">Lecture Notes in Statistics</span>, Vol. <span class="ltx_text ltx_bib_volume"></span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px3.p3" title="Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib62" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. M√∏ller (1995)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Generation of Johnson-Mehl crystals and comparative analysis of models for random nucleation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Advances in Applied Probability</span> <span class="ltx_text ltx_bib_volume">27</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†367‚Äì383</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.jstor.org/stable/1427831" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px3.p2" title="Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib180" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">A. V. Nagaev (1971)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">A limit theorem for a supercritical branching process</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Mathematical notes of the Academy of Sciences of the
USSR</span> <span class="ltx_text ltx_bib_volume">9</span> (<span class="ltx_text ltx_bib_number">5</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†338‚Äì342</span> (<span class="ltx_text ltx_bib_language">English</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1007/BF01094363" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 0001-4346</span>,
<a href="http://dx.doi.org/10.1007/BF01094363" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS7.p2" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>.
</span></li>
<li id="bib.bib17" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">H. L. Norton, R. A. Kittles, E. Parra, P. McKeigue, X. Mao, K. Cheng, V. A. Canfield, D. G. Bradley, B. McEvoy and M. D. Shriver (2007)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Genetic evidence for the convergent evolution of light skin in Europeans and east Asians</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Mol Biol Evol</span> <span class="ltx_text ltx_bib_volume">24</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†710‚Äì722</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1093/molbev/msl203" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<span class="ltx_text ltx_bib_external">http://mbe.oxfordjournals.org/cgi/reprint/24/3/710.pdf</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px1.p1" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib41" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">I. Olalde, M. E. Allentoft, F. S√°nchez-Quinto, G. Santpere, C. W. Chiang, M. DeGiorgio, J. Prado-Martinez, J. A. Rodr√≠guez, S. Rasmussen and J. Quilez (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Derived immune and ancestral pigmentation alleles in a 7,000-year-old mesolithic european</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Nature</span> <span class="ltx_text ltx_bib_volume">507</span> (<span class="ltx_text ltx_bib_number">7491</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†225‚Äì228</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px1.p1" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib43" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">H. A. Orr and A. J. Betancourt (2001)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Haldane‚Äôs sieve and adaptation from the standing genetic variation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">157</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†875‚Äì884</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11157004" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px1.p2" title="Rates of origination of standing and new mutations ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>,
<a href="#S1.SS7.p5" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>,
<a href="#S1.p3" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib44" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">H. A. Orr (2005)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The probability of parallel evolution</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Evolution</span> <span class="ltx_text ltx_bib_volume">59</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†216‚Äì220</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15792240" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS6.p1" title="1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.6</span></a>,
<a href="#S1.p2" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib36" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">H. A. Orr (2000)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Adaptation and the cost of complexity</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Evolution</span> <span class="ltx_text ltx_bib_volume">54</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†13‚Äì20</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS6.p1" title="1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.6</span></a>.
</span></li>
<li id="bib.bib38" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">S. P. Otto (2004)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Two steps forward, one step back: the pleiotropic effects of favoured alleles</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Proc. Biol. Sci.</span> <span class="ltx_text ltx_bib_volume">271</span> (<span class="ltx_text ltx_bib_number">1540</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†705‚Äì714</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS6.p1" title="1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.6</span></a>.
</span></li>
<li id="bib.bib2" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. Otwinowski and J. Krug (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Clonal interference and Muller‚Äôs ratchet in spatial habitats</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Physical Biology</span> <span class="ltx_text ltx_bib_volume">11</span> (<span class="ltx_text ltx_bib_number">5</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†056003</span> (<span class="ltx_text ltx_bib_language">eng</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 1478-3975</span>,
<a href="http://dx.doi.org/10.1088/1478-3975/11/5/056003" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p4" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib12" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. Pamillo (1979)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Genic variation at sex-linked loci: quantification of regular selection models</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Hereditas</span> <span class="ltx_text ltx_bib_volume">91</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†129‚Äì133</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 1601-5223</span>,
<a href="http://dx.doi.org/10.1111/j.1601-5223.1979.tb01652.x" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/10.1111/j.1601-5223.1979.tb01652.x" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p3" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib23" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">N. Patterson, P. Moorjani, Y. Luo, S. Mallick, N. Rohland, Y. Zhan, T. Genschoreck, T. Webster and D. Reich (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Ancient admixture in human history</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">192</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1065‚Äì1093</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib71" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">M. W. Pennell, L. J. Harmon and J. C. Uyeda (2014a)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Is there room for punctuated equilibrium in macroevolution?</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Trends in Ecology &amp; Evolution</span> <span class="ltx_text ltx_bib_volume">29</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†23‚Äì32</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 0169-5347</span>,
<a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.tree.2013.07.004" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://www.sciencedirect.com/science/article/pii/S0169534713001997" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p3" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib69" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">M. W. Pennell, L. J. Harmon and J. C. Uyeda (2014b)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Speciation is unlikely to drive divergence rates</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Trends in Ecology &amp; Evolution</span> <span class="ltx_text ltx_bib_volume">29</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†72‚Äì73</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 0169-5347</span>,
<a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.tree.2013.12.010" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://www.sciencedirect.com/science/article/pii/S0169534713003054" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p3" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib115" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. S. Pennings and J. Hermisson (2006)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Soft sweeps III: the signature of positive selection from recurrent mutation</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS Genet</span> <span class="ltx_text ltx_bib_volume">2</span> (<span class="ltx_text ltx_bib_number">12</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†e186</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.plos.org/10.1371%2Fjournal.pgen.0020186" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/10.1371/journal.pgen.0020186" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p2" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>,
<a href="#S2.SS7.SSS0.Px1.p3" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib117" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. S. Pennings and J. Hermisson (2006)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Soft sweeps II - molecular population genetics of adaptation from recurrent mutation or migration</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Mol Biol Evol</span>, <span class="ltx_text ltx_bib_pages"> pp.¬†msj117</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://mbe.oxfordjournals.org/cgi/content/abstract/msj117v1" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/10.1093/molbev/msj117" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<span class="ltx_text ltx_bib_external">http://mbe.oxfordjournals.org/cgi/reprint/msj117v1.pdf</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS7.p1" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>,
<a href="#S1.SS7.p4" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>,
<a href="#S1.SS7.p5" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>,
<a href="#S1.p3" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>,
<a href="#S2.p2" title="2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib26" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. K. Pickrell, G. Coop, J. Novembre, S. Kudaravalli, J. Z. Li, D. Absher, B. S. Srinivasan, G. S. Barsh, R. M. Myers, M. W. Feldman and J. K. Pritchard (2009)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Signals of recent positive selection in a worldwide sample of human populations</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genome Res</span> <span class="ltx_text ltx_bib_volume">19</span> (<span class="ltx_text ltx_bib_number">5</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†826‚Äì837</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19307593" title="" class="ltx_ref ltx_bib_external">Link</a>,
<a href="http://dx.doi.org/10.1101/gr.087577.108" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p2" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib14" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. K. Pickrell and D. Reich (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Toward a new history and geography of human genes informed by ancient DNA</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Trends Genet.</span> <span class="ltx_text ltx_bib_volume">30</span> (<span class="ltx_text ltx_bib_number">9</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†377‚Äì389</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p4" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib1" class="ltx_bibitem ltx_bib_manual"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">R Core Team (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">R: A language and environment for statistical computing</span>.
</span>
<span class="ltx_bibblock"> <span class="ltx_text ltx_bib_publisher">R Foundation for Statistical Computing</span>, <span class="ltx_text ltx_bib_place">Vienna, Austria</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.R-project.org/" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSSx1.p1" title="Acknowledgements ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib120" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. Ralph and G. Coop (2010)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Parallel adaptation: one or many waves of advance of an advantageous allele?</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">186</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†647‚Äì668</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1534/genetics.110.119594" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<span class="ltx_text ltx_bib_external">http://www.genetics.org/cgi/reprint/186/2/647.pdf</span>,
<a href="http://www.genetics.org/cgi/content/abstract/186/2/647" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.F1" title="Figure¬†1 ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>,
<a href="#S1.SS1.SSS0.Px1.p1" title="Rates of origination of standing and new mutations ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>,
<a href="#S1.SS1.SSS0.Px2.p1" title="Geographic spread of alleles ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>,
<a href="#S1.SS1.SSS0.Px3.p2" title="Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>,
<a href="#S1.SS1.SSS0.Px3.p3" title="Putting it together. ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>,
<a href="#S1.SS1.p3" title="1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>,
<a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>,
<a href="#S1.SS3.p1" title="1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.3</span></a>,
<a href="#S1.SS3.p2" title="1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.3</span></a>,
<a href="#S1.SS3.p3" title="1.3 The geographic resolution of adaptation from new and standing variation ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.3</span></a>,
<a href="#S1.SS6.p4" title="1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.6</span></a>,
<a href="#S1.SS7.p1" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>,
<a href="#S1.p4" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>,
<a href="#S1.p5" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>,
<a href="#S2.SS7.SSS0.Px1.p1" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>,
<a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib76" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">P. L. Ralph and G. Coop (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Convergent evolution during local adaptation to patchy landscapes</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">bioRxiv/006940</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1101/006940" title="" class="ltx_ref doi ltx_bib_external">Document</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p4" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>,
<a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib85" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">L. H. Rieseberg and J. M. Burke (2001)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The biological reality of species: gene flow, selection, and collective evolution</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Taxon</span> <span class="ltx_text ltx_bib_volume">50</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†pp. 47‚Äì67</span> (<span class="ltx_text ltx_bib_language">English</span>).
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://www.jstor.org/stable/1224511" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text issn ltx_bib_external">ISSN 00400262</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib83" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">E. B. Rosenblum, B. A. Sarver, J. W. Brown, S. D. Roches, K. M. Hardwick, T. D. Hether, J. M. Eastman, M. W. Pennell and L. J. Harmon (2012)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Goldilocks Meets Santa Rosalia: An Ephemeral Speciation Model Explains Patterns of Diversification Across Time Scales</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Evol Biol</span> <span class="ltx_text ltx_bib_volume">39</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†255‚Äì261</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p2" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>,
<a href="#S2.SS7.SSS0.Px4.p3" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib95" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">C. Ruwende, S. C. Khoo, R. W. Snow, S. N. Yates, D. Kwiatkowski, S. Gupta, P. Warn, C. E. Allsopp, S. C. Gilbert and N. Peschu (1995)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Natural selection of hemi- and heterozygotes for G6PD deficiency in Africa by resistance to severe malaria</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Nature</span> <span class="ltx_text ltx_bib_volume">376</span> (<span class="ltx_text ltx_bib_number">6537</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†246‚Äì249</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>,
<a href="#S1.SS2.p3" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib96" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">M. A. Saunders, M. Slatkin, C. Garner, M. F. Hammer and M. W. Nachman (2005)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The extent of linkage disequilibrium caused by selection on G6PD in humans</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">171</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1219‚Äì1229</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib15" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">G. Sella, D. A. Petrov, M. Przeworski and P. Andolfatto (2009)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Pervasive natural selection in the Drosophila genome?</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">PLoS Genet.</span> <span class="ltx_text ltx_bib_volume">5</span> (<span class="ltx_text ltx_bib_number">6</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†e1000495</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p5" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib78" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">M. Slatkin and D. Charlesworth (1978)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The spatial distribution of transient alleles in a subdivided population: a simulation study</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">89</span> (<span class="ltx_text ltx_bib_number">4</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†793‚Äì810</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px2.p1" title="Geographic spread of alleles ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib79" class="ltx_bibitem ltx_bib_inproceedings"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">M. Slatkin (1976)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The rate of spread of an advantageous allele in a subdivided population</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_pages"> pp.¬†767‚Äì780</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS1.SSS0.Px2.p1" title="Geographic spread of alleles ‚Ä£ 1.1 Model description ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.1</span></a>.
</span></li>
<li id="bib.bib97" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">M. Slatkin (2008)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">A Bayesian method for jointly estimating allele age and selection intensity</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genet Res (Camb)</span> <span class="ltx_text ltx_bib_volume">90</span> (<span class="ltx_text ltx_bib_number">1</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†129‚Äì137</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib22" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">V. Sousa and J. Hey (2013)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Understanding the origin of species with genome-scale data: modelling gene flow</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Nat. Rev. Genet.</span> <span class="ltx_text ltx_bib_volume">14</span> (<span class="ltx_text ltx_bib_number">6</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†404‚Äì414</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px2.p1" title="Are species held together by widespread selective sweeps? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib161" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">D. L. Stern (2013)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The genetic causes of convergent evolution</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Nat Rev Genet</span> <span class="ltx_text ltx_bib_volume">14</span> (<span class="ltx_text ltx_bib_number">11</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†751‚Äì764</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1038/nrg3483" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://www.ncbi.nlm.nih.gov/pubmed/24105273" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.p1" title="1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1</span></a>.
</span></li>
<li id="bib.bib98" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">S. A. Tishkoff, R. Varkonyi, N. Cahinhinan, S. Abbes, G. Argyropoulos, G. Destro-Bisol, A. Drousiotou, B. Dangerfield, G. Lefranc, J. Loiselet, A. Piro, M. Stoneking, A. Tagarelli, G. Tagarelli, E. H. Touma, S. M. Williams and A. G. Clark (2001)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Haplotype diversity and linkage disequilibrium at human G6PD: recent origin of alleles that confer malarial resistance</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Science</span> <span class="ltx_text ltx_bib_volume">293</span> (<span class="ltx_text ltx_bib_number">5529</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†455‚Äì462</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1126/science.1061573" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<span class="ltx_text ltx_bib_external">http://www.sciencemag.org/cgi/reprint/293/5529/455.pdf</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS2.p2" title="1.2 G6PD example ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.2</span></a>.
</span></li>
<li id="bib.bib34" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">R. L. Unckless and H. A. Orr (2009)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Dobzhansky-Muller incompatibilities and adaptation to a shared environment</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Heredity (Edinb)</span> <span class="ltx_text ltx_bib_volume">102</span> (<span class="ltx_text ltx_bib_number">3</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†214‚Äì217</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS6.p1" title="1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.6</span></a>.
</span></li>
<li id="bib.bib70" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">C. Venditti and M. Pagel (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Plenty of room for punctuational change</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Trends in Ecology &amp; Evolution</span> <span class="ltx_text ltx_bib_volume">29</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†71‚Äì72</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><span class="ltx_text issn ltx_bib_external">ISSN 0169-5347</span>,
<a href="http://dx.doi.org/http://dx.doi.org/10.1016/j.tree.2013.12.002" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://www.sciencedirect.com/science/article/pii/S0169534713002930" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p3" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib35" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">J. J. Welch and D. Waxman (2003)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Modularity and the cost of complexity</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Evolution</span> <span class="ltx_text ltx_bib_volume">57</span> (<span class="ltx_text ltx_bib_number">8</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†1723‚Äì1734</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS6.p1" title="1.6 Multiple variant types ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.6</span></a>.
</span></li>
<li id="bib.bib39" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">S. Wilde, A. Timpson, K. Kirsanow, E. Kaiser, M. Kayser, M. Unterl√§nder, N. Hollfelder, I. D. Potekhina, W. Schier, M. G. Thomas and J. Burger (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Direct evidence for positive selection of skin, hair, and eye pigmentation in Europeans during the last 5,000 years</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Proceedings of the National Academy of Sciences</span> <span class="ltx_text ltx_bib_volume">111</span> (<span class="ltx_text ltx_bib_number">13</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†4832‚Äì4837</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1073/pnas.1316513111" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://www.pnas.org/content/111/13/4832.abstract" title="" class="ltx_ref ltx_bib_external">Link</a>,
<span class="ltx_text ltx_bib_external">http://www.pnas.org/content/111/13/4832.full.pdf+html</span></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px1.p1" title="The confusing signal of geographic convergent evolution. ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib52" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">B. A. Wilson, D. A. Petrov and P. W. Messer (2014)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Soft selective sweeps in complex demographic scenarios</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">198</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†669‚Äì684</span>.
</span>
<span class="ltx_bibblock">External Links: <span class="ltx_text ltx_bib_links"><a href="http://dx.doi.org/10.1534/genetics.114.165571" title="" class="ltx_ref doi ltx_bib_external">Document</a>,
<a href="http://www.ncbi.nlm.nih.gov/pubmed/25060100" title="" class="ltx_ref ltx_bib_external">Link</a></span>
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS7.p5" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>.
</span></li>
<li id="bib.bib66" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">S. Wright (1932)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">The roles of mutation, inbreeding, crossbreeding and selection in evolution.</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Proc. VI Int. Congr. Genet.</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S2.SS7.SSS0.Px4.p4" title="Speciation and extinction as phases of molecular evolution? ‚Ä£ 2 Discussion ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">2</span></a>.
</span></li>
<li id="bib.bib31" class="ltx_bibitem ltx_bib_article"><span class="ltx_bibtag ltx_bib_author-year ltx_role_refnum">S. Wright (1943)</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_title">Isolation by Distance</span>.
</span>
<span class="ltx_bibblock"><span class="ltx_text ltx_bib_journal">Genetics</span> <span class="ltx_text ltx_bib_volume">28</span> (<span class="ltx_text ltx_bib_number">2</span>), <span class="ltx_text ltx_bib_pages"> pp.¬†114‚Äì138</span>.
</span>
<span class="ltx_bibblock ltx_bib_cited">Cited by: <a href="#S1.SS7.p2" title="1.7 Local establishment and comparison to panmixia ‚Ä£ 1 Introduction ‚Ä£ The role of standing variation in geographic convergent adaptation" class="ltx_ref"><span class="ltx_text ltx_ref_tag">1.7</span></a>.
</span></li>
</ul>
</div>
  

</div>
</div>
<div class="ltx_page_footer">
<div class="ltx_page_logo">Generated  on Wed Apr 15 16:42:24 2015 by <a href="http://dlmf.nist.gov/LaTeXML/">LaTeXML <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAOCAYAAAD5YeaVAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9wKExQZLWTEaOUAAAAddEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIFRoZSBHSU1Q72QlbgAAAdpJREFUKM9tkL+L2nAARz9fPZNCKFapUn8kyI0e4iRHSR1Kb8ng0lJw6FYHFwv2LwhOpcWxTjeUunYqOmqd6hEoRDhtDWdA8ApRYsSUCDHNt5ul13vz4w0vWCgUnnEc975arX6ORqN3VqtVZbfbTQC4uEHANM3jSqXymFI6yWazP2KxWAXAL9zCUa1Wy2tXVxheKA9YNoR8Pt+aTqe4FVVVvz05O6MBhqUIBGk8Hn8HAOVy+T+XLJfLS4ZhTiRJgqIoVBRFIoric47jPnmeB1mW/9rr9ZpSSn3Lsmir1fJZlqWlUonKsvwWwD8ymc/nXwVBeLjf7xEKhdBut9Hr9WgmkyGEkJwsy5eHG5vN5g0AKIoCAEgkEkin0wQAfN9/cXPdheu6P33fBwB4ngcAcByHJpPJl+fn54mD3Gg0NrquXxeLRQAAwzAYj8cwTZPwPH9/sVg8PXweDAauqqr2cDjEer1GJBLBZDJBs9mE4zjwfZ85lAGg2+06hmGgXq+j3+/DsixYlgVN03a9Xu8jgCNCyIegIAgx13Vfd7vdu+FweG8YRkjXdWy329+dTgeSJD3ieZ7RNO0VAXAPwDEAO5VKndi2fWrb9jWl9Esul6PZbDY9Go1OZ7PZ9z/lyuD3OozU2wAAAABJRU5ErkJggg==" alt="[LOGO]"></img></a></div></div>
</div>
</body>
</html>
